--- libsm-1.2.6.orig/.gitlab-ci.yml
+++ libsm-1.2.6/.gitlab-ci.yml
@@ -0,0 +1,124 @@
+# vim: set expandtab shiftwidth=2 tabstop=8 textwidth=0 filetype=yaml:
+#
+# This CI uses the freedesktop.org ci-templates.
+# Please see the ci-templates documentation for details:
+# https://freedesktop.pages.freedesktop.org/ci-templates/
+
+.templates_sha: &template_sha 185ede0e9b9b1924b92306ab8b882a6294e92613 # see https://docs.gitlab.com/ee/ci/yaml/#includefile
+
+
+include:
+  # Debian container builder template
+  - project: 'freedesktop/ci-templates'
+    ref: *template_sha
+    file: '/templates/debian.yml'
+  - project: 'freedesktop/ci-templates'
+    ref: *template_sha
+    file: '/templates/ci-fairy.yml'
+  - template: Security/SAST.gitlab-ci.yml
+
+
+stages:
+  - prep             # prep work like rebuilding the container images if there is a change
+  - install libice
+  - build            # for actually building and testing things in a container
+  - test
+  - deploy
+
+
+variables:
+  FDO_UPSTREAM_REPO: 'xorg/lib/libSM'
+  FDO_DISTRIBUTION_VERSION: 'stable'
+  # The tag should be updated each time the list of packages is updated.
+  # Changing a tag forces the associated image to be rebuilt.
+  # Note: the tag has no meaning, we use a date format purely for readability
+  FDO_DISTRIBUTION_TAG:  '2023-10-13.0'
+  FDO_DISTRIBUTION_PACKAGES: 'git gcc pkgconf autoconf automake libtool make xutils-dev x11proto-dev xtrans-dev libbsd-dev uuid-dev xmlto xorg-sgml-doctools w3m xsltproc fop xz-utils'
+
+
+#
+# Verify that commit messages are as expected
+#
+check-commits:
+  extends:
+    - .fdo.ci-fairy
+  stage: prep
+  script:
+    - ci-fairy check-commits --junit-xml=results.xml
+  except:
+    - master@xorg/lib/libSM
+  variables:
+    GIT_DEPTH: 100
+  artifacts:
+    reports:
+      junit: results.xml
+
+#
+# Verify that the merge request has the allow-collaboration checkbox ticked
+#
+check-merge-request:
+  extends:
+    - .fdo.ci-fairy
+  stage: deploy
+  script:
+    - ci-fairy check-merge-request --require-allow-collaboration --junit-xml=results.xml
+  artifacts:
+    when: on_failure
+    reports:
+      junit: results.xml
+  allow_failure: true
+
+
+#
+# Build a container with the given tag and the packages pre-installed.
+# This only happens if/when the tag changes, otherwise the existing image is
+# re-used.
+#
+container-prep:
+  extends:
+    - .fdo.container-build@debian
+  stage: prep
+  variables:
+    GIT_STRATEGY: none
+
+
+# Install latest version of libICE, since debian doesn't package 1.1.0 yet
+libice-build:
+  extends:
+    - .fdo.distribution-image@debian
+  stage: install libice
+  script:
+    - export INSTDIR="$PWD/_inst"
+    - git clone --depth=1 https://gitlab.freedesktop.org/xorg/lib/libice
+    - pushd libice > /dev/null
+    - autoreconf -ivf
+    - mkdir _builddir
+    - pushd _builddir > /dev/null
+    - ../configure --disable-silent-rules --prefix="$INSTDIR"
+    - make
+    - make install
+    - popd > /dev/null
+    - popd > /dev/null
+  variables:
+  artifacts:
+    paths:
+      - _inst
+
+#
+# The default build, runs on the image built above.
+#
+build:
+  stage: build
+  extends:
+    - .fdo.distribution-image@debian
+  script:
+    - export INSTDIR="$PWD/_inst"
+    - export PKG_CONFIG_PATH=$(find $INSTDIR/ -name '*.pc' -printf "%h:")
+    - autoreconf -ivf
+    - mkdir _builddir
+    - pushd _builddir > /dev/null
+    - ../configure --disable-silent-rules --with-libuuid --enable-docs
+    - make
+    - make check
+    - make distcheck
+    - popd > /dev/null
--- libsm-1.2.6.orig/autogen.sh
+++ libsm-1.2.6/autogen.sh
@@ -0,0 +1,17 @@
+#! /bin/sh
+
+srcdir=`dirname "$0"`
+test -z "$srcdir" && srcdir=.
+
+ORIGDIR=`pwd`
+cd "$srcdir"
+
+autoreconf -v --install || exit 1
+cd "$ORIGDIR" || exit $?
+
+git config --local --get format.subjectPrefix >/dev/null 2>&1 ||
+    git config --local format.subjectPrefix "PATCH libSM"
+
+if test -z "$NOCONFIGURE"; then
+    exec "$srcdir"/configure "$@"
+fi
--- libsm-1.2.6.orig/debian/README.source
+++ libsm-1.2.6/debian/README.source
@@ -0,0 +1,49 @@
+------------------------------------------------------
+Quick Guide To Patching This Package For The Impatient
+------------------------------------------------------
+
+1. Make sure you have quilt installed
+2. Unpack the package as usual with "dpkg-source -x"
+3. Run the "patch" target in debian/rules
+4. Create a new patch with "quilt new" (see quilt(1))
+5. Edit all the files you want to include in the patch with "quilt edit" 
+   (see quilt(1)).
+6. Write the patch with "quilt refresh" (see quilt(1))
+7. Run the "clean" target in debian/rules
+
+Alternatively, instead of using quilt directly, you can drop the patch in to 
+debian/patches and add the name of the patch to debian/patches/series.
+
+------------------------------------
+Guide To The X Strike Force Packages
+------------------------------------
+
+The X Strike Force team maintains X packages in git repositories on
+git.debian.org in the pkg-xorg subdirectory. Most upstream packages
+are actually maintained in git repositories as well, so they often
+just need to be pulled into git.debian.org in a "upstream-*" branch.
+Otherwise, the upstream sources are manually installed in the Debian
+git repository.
+
+The .orig.tar.gz upstream source file could be generated using this
+"upstream-*" branch in the Debian git repository but it is actually
+copied from upstream tarballs directly.
+
+Due to X.org being highly modular, packaging all X.org applications
+as their own independent packages would have created too many Debian
+packages. For this reason, some X.org applications have been grouped
+into larger packages: xutils, xutils-dev, x11-apps, x11-session-utils,
+x11-utils, x11-xfs-utils, x11-xkb-utils, x11-xserver-utils.
+Most packages, including the X.org server itself and all libraries
+and drivers are, however maintained independently.
+
+The Debian packaging is added by creating the "debian-*" git branch
+which contains the aforementioned "upstream-*" branch plus the debian/
+repository files.
+When a patch has to be applied to the Debian package, two solutions
+are involved:
+* If the patch is available in one of the upstream branches, it
+  may be git'cherry-picked into the Debian repository. In this
+  case, it appears directly in the .diff.gz.
+* Otherwise, the patch is added to debian/patches/ which is managed
+  with quilt as documented in /usr/share/doc/quilt/README.source.
--- libsm-1.2.6.orig/debian/changelog
+++ libsm-1.2.6/debian/changelog
@@ -0,0 +1,257 @@
+libsm (2:1.2.6-1) unstable; urgency=medium
+
+  * New upstream release.
+  * control: Fix -dev to depend on x11proto-dev instead of -core-dev.
+  * watch: Update upstream vcs url.
+  * control: Migrate to debhelper-compat, bump to 13.
+  * control: Migrate to pkgconf.
+  * rules: Drop old dbgsym migration.
+
+ -- Timo Aaltonen <tjaalton@debian.org>  Tue, 25 Mar 2025 15:14:52 +0200
+
+libsm (2:1.2.4-1) unstable; urgency=medium
+
+  [ Emilio Pozuelo Monfort ]
+  * Move doc build-deps to Build-Depends-Indep. Thanks Samuel Thibault.
+    Closes: #1055365.
+
+  [ Julien Cristau ]
+  * Add Alan Coopersmith's key to d/upstream/signing-key.asc.
+  * New upstream release
+    + Bump libice-dev build-dep to >= 2:1.1.0
+    + Add uuid-dev to libsm-dev dependencies for uuid.pc
+
+ -- Julien Cristau <jcristau@debian.org>  Thu, 31 Oct 2024 19:14:59 +0100
+
+libsm (2:1.2.3-1) unstable; urgency=medium
+
+  * New upstream release.
+  * Let uscan check upstream tarball signatures.
+  * Update Vcs-* URLs to point to salsa.debian.org.
+  * Switch to dbgsym package.
+  * Bump debhelper compat to 11.
+  * Build-depend on x11proto-dev.
+  * Move URL from description to Homepage field.
+  * Set source format to 1.0.
+  * Switch URLs to https.
+  * Drop xsfbs from README.source.
+  * Bump standards version to 4.3.0.
+
+ -- Andreas Boll <aboll@debian.org>  Fri, 08 Feb 2019 22:32:40 +0100
+
+libsm (2:1.2.2-1) unstable; urgency=medium
+
+  * New upstream release.
+  * Remove Cyril from Uploaders.
+  * Rewrite debian/rules using dh.  Bump compat to 9, get rid of xsfbs.
+
+ -- Julien Cristau <jcristau@debian.org>  Sun, 13 Jul 2014 01:57:15 +0200
+
+libsm (2:1.2.1-2) unstable; urgency=low
+
+  * Add missing ${misc:Depends} to libsm-doc.
+  * Add missing Replaces/Breaks on libsm-dev to the doc package.  Thanks,
+    Sven Joachim!
+
+ -- Julien Cristau <jcristau@debian.org>  Mon, 09 Apr 2012 20:18:02 +0200
+
+libsm (2:1.2.1-1) unstable; urgency=low
+
+  * New upstream release.
+  * Drop 0001-sm.pc-don-t-export-a-dependency-on-ice.patch, applied upstream.
+  * Don't require (fake)root for debian/rules clean.
+  * Split the docs to a separate libsm-doc package
+    - mark libsm-dev Multi-Arch: same, and libsm-doc Multi-Arch: foreign
+    - don't build the docs in build-arch
+  * Bump build-deps on xutils-dev, xmlto and xorg-sgml-doctools.
+
+ -- Julien Cristau <jcristau@debian.org>  Mon, 09 Apr 2012 17:55:48 +0200
+
+libsm (2:1.2.0-2) unstable; urgency=low
+
+  * Team upload.
+
+  [ Steve Langasek ]
+  * Build for multiarch.
+
+  [ Julien Cristau ]
+  * Remove David Nusinow and Brice Goglin from Uploaders.
+  * Bump Standards-Version to 3.9.2.
+
+ -- Julien Cristau <jcristau@debian.org>  Sat, 11 Jun 2011 14:49:47 +0200
+
+libsm (2:1.2.0-1) unstable; urgency=low
+
+  [ Julien Cristau ]
+  * Rename the build directory to not include DEB_BUILD_GNU_TYPE for no
+    good reason.  Thanks, Colin Watson!
+  * Remove myself from Uploaders
+
+  [ Cyril Brulebois ]
+  * New upstream release.
+  * Bump xutils-dev build-dep for new macros.
+  * Add xmlto, xorg-sgml-doctools, and w3m build-dep for the doc.
+  * Pass --with-xmlto and --without-fop, we want html and txt only.
+  * Kill *.xml in the doc directory, no point in shipping them.
+  * Ship *.html, *.css, and *.txt from the doc directory.
+  * Switch from --list-missing to --fail-missing for additional safety.
+  * Add myself to Uploaders.
+
+ -- Cyril Brulebois <kibi@debian.org>  Mon, 08 Nov 2010 01:42:36 +0100
+
+libsm (2:1.1.1-1) unstable; urgency=low
+
+  * New upstream release.
+  * Move libsm6-dbg to 'debug' section.
+  * Drop x11-common (pre-)dependencies, this isn't needed anymore.
+  * Bump libice-dev build-dep to 2:1.0.5.
+  * Bump xutils-dev build-dep to 1:7.4+4 for new util-macros.
+  * Add README.source from xsfbs, bump Standards-Version to 3.8.3.
+  * Look for space-separated DEB_BUILD_OPTIONS.
+
+ -- Julien Cristau <jcristau@debian.org>  Tue, 25 Aug 2009 17:58:18 +0200
+
+libsm (2:1.1.0-2) unstable; urgency=low
+
+  * Upload to unstable.
+
+ -- Julien Cristau <jcristau@debian.org>  Mon, 16 Feb 2009 01:33:27 +0100
+
+libsm (2:1.1.0-1) experimental; urgency=low
+
+  [ Brice Goglin ]
+  * Use ${binary:Version} instead of the deprecated ${Source-Version}.
+
+  [ Julien Cristau ]
+  * New upstream release.
+    - generate client IDs using libuuid instead of gethostbyname()
+  * Switch to running autoreconf at build time; clean up in debian/rules
+    clean, and build-depend on automake, libtool and xutils-dev.
+  * Drop -1 debian revisions from build-deps.
+  * Stop handling nostrip explicitly in debian/rules (dh_strip does it
+    already), and allow parallel builds using sample code from policy.
+  * Add myself to Uploaders.
+  * Don't export a dependency on ice in sm.pc.
+
+ -- Julien Cristau <jcristau@debian.org>  Fri, 18 Jul 2008 16:16:47 +0200
+
+libsm (2:1.0.3-2) unstable; urgency=low
+
+  * Fix short descriptions of libsm-dev and libsm-dbg,
+    thanks Philippe Cloutier, closes: #432967.
+  * Add upstream URL to debian/copyright.
+  * Bump Standards-Version to 3.7.3 (no changes).
+  * Add myself to Uploaders, and remove Branden and Fabio with their
+    permission.
+  * Add Vcs-Browser field and drop the XS- prefix from Vcs-Git.
+  * Add a link to www.X.org and a reference to the libSM module in the long
+    description.
+
+ -- Brice Goglin <bgoglin@debian.org>  Fri, 13 Jun 2008 07:07:33 +0200
+
+libsm (2:1.0.3-1) unstable; urgency=low
+
+  [ Julien Cristau ]
+  * Move binary packages to the proper sections.
+  * New upstream release.
+
+  [ Timo Aaltonen ]
+  * Bump the epoch so that this can be synced to Ubuntu in the future.
+
+ -- Julien Cristau <jcristau@debian.org>  Mon, 14 May 2007 16:41:22 +0200
+
+libsm (1:1.0.2-2) unstable; urgency=low
+
+  * Upload to unstable.
+
+ -- Julien Cristau <jcristau@debian.org>  Wed, 11 Apr 2007 10:53:40 +0200
+
+libsm (1:1.0.2-1) experimental; urgency=low
+
+  * New upstream release.
+  * Drop obsolete CVS info from the descriptions, and add XS-Vcs-Git.
+  * Install the upstream changelog.
+
+ -- Julien Cristau <jcristau@debian.org>  Fri,  9 Feb 2007 17:37:39 +0100
+
+libsm (1:1.0.1-3) unstable; urgency=low
+
+  * Exclude libSM.la from dh_install.
+  * The library in debian/rules is libsm6 not libxau6! Closes: #383468.
+  * libsm-dev uses versioned depends on libice-dev from X11R7. 
+    Closes: #372129.
+
+ -- Drew Parsons <dparsons@debian.org>  Fri, 22 Sep 2006 21:33:23 +1000
+
+libsm (1:1.0.1-2) unstable; urgency=low
+
+  [ Andres Salomon ]
+  * Test for obj-$(DEB_BUILD_GNU_TYPE) before creating it during build;
+    idempotency fix.
+
+  [ Drew Parsons ]
+  * dbg package has priority extra.
+
+ -- David Nusinow <dnusinow@debian.org>  Tue, 29 Aug 2006 23:35:12 +0000
+
+libsm (1:1.0.1-1) experimental; urgency=low
+
+  * New upstream release
+  * Run dh_install with --list-missing
+  * Bump debhelper compat to 5
+  * Remove extra x11-common dep from -dev package
+  * Version x11-common pre-dep in -dev package to be 1:7.0.0 to match the rest
+    of Debian and shut lintian up
+  * Bump standards version to 3.7.2.0
+  * Don't look to install manpages for -dev package, there aren't any
+
+ -- David Nusinow <dnusinow@debian.org>  Mon,  3 Jul 2006 17:18:41 -0400
+
+libsm (1:1.0.0-4) unstable; urgency=low
+
+  * Reorder makeshlib command in rules file so that ldconfig is run
+    properly. Thanks Drew Parsons and Steve Langasek.
+
+ -- David Nusinow <dnusinow@debian.org>  Tue, 18 Apr 2006 21:49:55 -0400
+
+libsm (1:1.0.0-3) unstable; urgency=low
+
+  * Upload to unstable
+
+ -- David Nusinow <dnusinow@debian.org>  Thu, 23 Mar 2006 22:44:25 -0500
+
+libsm (1:1.0.0-2) experimental; urgency=low
+
+  * Provide versioned build depends for the X libs
+
+ -- David Nusinow <dnusinow@debian.org>  Thu, 23 Feb 2006 22:46:43 -0500
+
+libsm (1:1.0.0-1) UNRLEASED; urgency=low
+
+  * First upload to Debian
+
+ -- David Nusinow <dnusinow@debian.org>  Thu, 29 Dec 2005 20:47:53 -0500
+
+libsm (1:6.0.4-4) breezy; urgency=low
+
+  * Version the x11proto-core-dev build-dep to avoid _XOPEN_SOURCE.
+
+ -- Adam Conrad <adconrad@ubuntu.com>  Sun, 24 Jul 2005 10:42:00 +0000
+
+libsm (1:6.0.4-3) breezy; urgency=low
+
+  * Add libice-dev to libsm-dev Depends.
+
+ -- Daniel Stone <daniel.stone@ubuntu.com>  Thu, 21 Jul 2005 18:07:37 +1000
+
+libsm (1:6.0.4-2) breezy; urgency=low
+
+  * Fix Build-Depends-Indep vs B-D hilarity/stupidity.
+
+ -- Daniel Stone <daniel.stone@ubuntu.com>  Wed,  1 Jun 2005 00:34:43 +1000
+
+libsm (1:6.0.4-1) breezy; urgency=low
+
+  * First libsm release.
+
+ -- Daniel Stone <daniel.stone@ubuntu.com>  Mon, 16 May 2005 22:10:17 +1000
--- libsm-1.2.6.orig/debian/control
+++ libsm-1.2.6/debian/control
@@ -0,0 +1,73 @@
+Source: libsm
+Section: x11
+Priority: optional
+Maintainer: Debian X Strike Force <debian-x@lists.debian.org>
+Build-Depends:
+ debhelper-compat (= 13),
+ pkgconf,
+ x11proto-dev,
+ xtrans-dev,
+ libice-dev (>= 2:1.1.0),
+ uuid-dev,
+ xutils-dev (>= 1:7.6+2),
+ quilt,
+Build-Depends-Indep:
+ xmlto (>= 0.0.22), xorg-sgml-doctools (>= 1:1.8), w3m,
+Standards-Version: 4.3.0
+Homepage: https://www.x.org
+Vcs-Git: https://salsa.debian.org/xorg-team/lib/libsm.git
+Vcs-Browser: https://salsa.debian.org/xorg-team/lib/libsm
+
+Package: libsm6
+Section: libs
+Architecture: any
+Depends: ${shlibs:Depends}, ${misc:Depends}
+Pre-Depends: ${misc:Pre-Depends}
+Multi-Arch: same
+Description: X11 Session Management library
+ This package provides the main interface to the X11 Session Management
+ library, which allows for applications to both manage sessions, and make use
+ of session managers to save and restore their state for later use.
+ .
+ This module can be found at
+ git://anongit.freedesktop.org/git/xorg/lib/libSM
+
+Package: libsm-dev
+Section: libdevel
+Architecture: any
+Multi-Arch: same
+Depends:
+ ${shlibs:Depends},
+ ${misc:Depends},
+ libsm6 (= ${binary:Version}),
+ x11proto-dev,
+ libice-dev (>= 1:1.0.0-1),
+ uuid-dev,
+Suggests: libsm-doc
+Description: X11 Session Management library (development headers)
+ This package provides the main interface to the X11 Session Management
+ library, which allows for applications to both manage sessions, and make use
+ of session managers to save and restore their state for later use.
+ .
+ This package contains the development headers for the library found in libsm6.
+ Non-developers likely have little use for this package.
+ .
+ This module can be found at
+ git://anongit.freedesktop.org/git/xorg/lib/libSM
+
+Package: libsm-doc
+Section: doc
+Architecture: all
+Multi-Arch: foreign
+Depends:
+ ${misc:Depends},
+Replaces:
+ libsm-dev (<< 2:1.2.1),
+Breaks:
+ libsm-dev (<< 2:1.2.1),
+Description: documentation for the X Session Management protocol and library
+ The X Session Management protocol facilitates the management of groups of
+ client applications by a session manager.
+ .
+ This package contains documentation for the session management protocol
+ itself and for the associated library, libSM.
--- libsm-1.2.6.orig/debian/copyright
+++ libsm-1.2.6/debian/copyright
@@ -0,0 +1,24 @@
+This package was downloaded from
+https://xorg.freedesktop.org/releases/individual/lib/
+
+Copyright 1993, 1998  The Open Group
+
+Permission to use, copy, modify, distribute, and sell this software and its
+documentation for any purpose is hereby granted without fee, provided that
+the above copyright notice appear in all copies and that both that
+copyright notice and this permission notice appear in supporting
+documentation.
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+OPEN GROUP BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN
+AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+Except as contained in this notice, the name of The Open Group shall not be
+used in advertising or otherwise to promote the sale, use or other dealings
+in this Software without prior written authorization from The Open Group.
--- libsm-1.2.6.orig/debian/libsm-dev.install
+++ libsm-1.2.6/debian/libsm-dev.install
@@ -0,0 +1,4 @@
+usr/include/X11/*
+usr/lib/*/libSM.a
+usr/lib/*/libSM.so
+usr/lib/*/pkgconfig/sm.pc
--- libsm-1.2.6.orig/debian/libsm-doc.install
+++ libsm-1.2.6/debian/libsm-doc.install
@@ -0,0 +1 @@
+usr/share/doc/libsm-dev
--- libsm-1.2.6.orig/debian/libsm6.install
+++ libsm-1.2.6/debian/libsm6.install
@@ -0,0 +1 @@
+usr/lib/*/libSM.so.6*
--- libsm-1.2.6.orig/debian/not-installed
+++ libsm-1.2.6/debian/not-installed
@@ -0,0 +1 @@
+usr/lib/*/libSM.la
--- libsm-1.2.6.orig/debian/patches/series
+++ libsm-1.2.6/debian/patches/series
@@ -0,0 +1 @@
+# placeholder
--- libsm-1.2.6.orig/debian/rules
+++ libsm-1.2.6/debian/rules
@@ -0,0 +1,23 @@
+#!/usr/bin/make -f
+# debian/rules for the Debian libxau package.
+# Copyright © 2004 Scott James Remnant <scott@netsplit.com>
+# Copyright © 2005 Daniel Stone <daniel@fooishbar.org>
+# Copyright © 2005 David Nusinow <dnusinow@debian.org>
+
+# Uncomment this to turn on verbose mode.
+#export DH_VERBOSE=1
+
+# set this to the name of the main shlib's binary package
+PACKAGE = libsm6
+
+%:
+	dh $@ --with quilt --builddirectory=build/
+
+override_dh_auto_configure-arch: docflags = --disable-docs
+override_dh_auto_configure-indep: docflags = --enable-docs --with-xmlto
+override_dh_auto_configure-arch override_dh_auto_configure-indep:
+	dh_auto_configure -- \
+		--with-libuuid \
+		--without-fop \
+		--docdir=\$${datadir}/doc/libsm-dev \
+		$(docflags)
--- libsm-1.2.6.orig/debian/source/format
+++ libsm-1.2.6/debian/source/format
@@ -0,0 +1 @@
+1.0
--- libsm-1.2.6.orig/debian/upstream/signing-key.asc
+++ libsm-1.2.6/debian/upstream/signing-key.asc
@@ -0,0 +1,125 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+
+mQINBFeKY50BEADAX0lod3IVceb/IWJn3kTAcO2P7PWlcBiyUDaq5b2kFkliKleZ
+ec4LoCHakQBlkRBMPNwOOxvADNk3tLQjBDpbYr6lQIrN+AxMGkXBhJ82T3bsDvlj
+3Z1wRJ1zVA7eMIktsk0FAoJxV1y7e3sBKcP0eTlXqXvR2djhi+FW+ueJDAJIFSkb
+uFirgwtX5t8nt8jCmIl75KNUKOakoENY3hLWtr16W8fO1JGkEhghI2mXcz664KTd
+MPZp6JH0/8UHTHzmATOCTqNxoDtMTi2l5059Lh/nhmso9moTYqyKmaJP2rnZUr62
+97sRMG4WcxaYfWpPyO3MCmDyGeh4sW0OC06PpED3i9xMzf/kMkMdY4ZIFcLRcPtf
+LIJhw+lc/GE1Rqe961IB5xCgnZezB7ZIL+ZlOAMwKGkq7lLbcZr2QZn84lpABKF0
+AvxECoJ4etmIcdbDVmsw18AhA3u9sr98hS5IXDyeos3Xwz6Abml8aPrhqhkKvo+J
+Kcq9FNYHg0RRlos0TqocjDzGnUjEYrmIopLcwIu2SnsNSJTygZGtqrpT+2sGEqvm
+k6Oyk95QCa580zqldvxe3CG0vrAfPvoG7irllM68TS4JcqqDHTq6eupUv9ZdIzXf
+eyTHa5cytGahgVtUcui1lzqcCBkqwN8TKl+0wCcEnxRasHJy3A2Gp+AG3wARAQAB
+tCJNYXR0aGlldSBIZXJyYiA8bWF0dGhpZXVAaGVycmIuZXU+iQI+BBMBAgAoAhsD
+BgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgAUCW0nLtwUJB4HPDwAKCRBoc5PuN9Eo
++HrjD/sF7vcimyb4XgxxQx2ThtXjoDdbxoaCnnCiNC9gfO4/OU31xFhXNpjMkBXA
+gUAtjBayCwx8iYP60eIrzJA7yxzdhhUAHv4V9Y/2lUzw/prjDNV4eO65tIHu79ws
+C59BV9i6p9pV0fQ7dOKowlSh2W/NiO2ydgk0Q1PuSoN8jJBKJxou7LDL2wmJlXrx
+b/qyPAS1EMri6E6sgnb/bODFCUSoa592dnU8TA5FvPMU3p0viWUY5w/cbRWrst12
+W6rlney1pV2e6jO++riTMghFtxMD4hqPvZLpCrdoJ4CXilGWCSvs5DoPYFcB2Opn
+wYpnSfW6sKTGbCztL1GKXUf6gnQcLBfkw8RpL9YWMIgWMbFD9JLNpvMMGOrAxQZ1
+Cz+Dw+wYV4ukFG1hLyDWiQsAMHAdSFIGOngyxi2zu7Aow3xIUGFxxiZ6LgOQASks
+115C9I6vqYJ8zpyh3YbSEn4pq1yhVAFeAiXCjFZGMrKN75+W6hKqFTj2kdQftzCg
+zV1VyUT8Yl9tCNzihyE1aHbIdUbmLzaU7Nf+xaP3bD94hpk5a3DkwUBnIMUMbiEU
+ieBdF5/u6rzv3eOvSfJDuBqlyUQEWSsv52arcqeVwAAgDD8Z/XMZPUxoPsXMop0Q
+j1qYY+GjeBP0/EWKbqBBPTZsMOy6cP5YZZln22gqZc1Ok0AJwrQnTWF0dGhpZXUg
+SGVycmIgPG1hdHRoaWV1LmhlcnJiQGxhYXMuZnI+iQI+BBMBAgAoAhsDBgsJCAcD
+AgYVCAIJCgsEFgIDAQIeAQIXgAUCW0nLsgUJB4HPDwAKCRBoc5PuN9Eo+P0KD/0c
+ux1U/r3KtFFM50tuwf8zrxuG/dfQbPnMBfY0WvPMrDUMRPmpylsAU6tgsO0D2OR7
+Fulc7Mu08qq9ddQiJhpog4P5i1+b/edbkR6jUr0prb3J1GBw3tx34x5OnKUS0WOY
+aXkTNDVm3Xnwm6ILdqOvEPuDizMIFdgEQc1TAcT4x3whvPz0VVE+UrKHcYHBw5ua
+H/UC7mzdhHqyuMdVcEJQoO8I+x7a/mNGrbyG8Qdx2Onclwk1950Dzq1fPOPet/JX
+9m0MUsZ3tF2BlXccdzNoMShNkbagUyEoKEQOL5I6RkrxJmi+BPsBsQjKkCA80ULJ
+xNk0iWLXICj1JBRn7/E1UhBxOcaYws3OZM5UjbFUpE+HOm1d0mY4rWFb9heGEwOP
+gj/1lag/gHY6Wm2uK3UCFVF9P7Ag5dgTdJapfoCsW0Fq6cQ1Y0fA/SAsigksDgxw
+W6mI0SZDcJFpvImCe6Odq0/+XCgqiTiooxtXzP97sOk+uz66st/oF0Q9vNw+ykNg
+C0lI2e8ypfRnhs1ef3YJvhnxo/eTFJncsP1cpjLeKRMeh4T6psZoyzGZOhsfM7sz
+7TcMqYZAYUyWZLbSSl/6dbI/R9g91DeZcxiQZJ1T1mBJOT7cXDGjPqsCYlIbCVrW
+qulJ/M/ysffiZLs4+dnv2saNfgVTpyGn0tzPcqE7YrkCDQRXimOdARAA4otssvZm
+sKg+g0bVyJHhn/YOHLYMih+Xf07xJHyalH0UCGnGdHZwl0B97G950SwQ7yVXtGa9
+CAPe97clE6dPD6jaumQ13BHavXM+ThgjCe8V56ayYcdzqFkxlCx0Uocoa63G0/cE
+TiOqeqhNZs8JY+D7l83jCa4lU/1pLusbkCpCQ7d5/FFLz7QSihzJWp+UTsjbNik5
+spaseEMGFRKUcB3SZ/l1dTgc0wBQ1hlvLX+h4/sG0iUs1pVpo5ORC+bUfWRokl96
+uj5QZz5rY21FaNSP1rB1HKHNkwhxifBCHQMhYGTXvD7GH+JNyF2TdRmo7eBCfAPJ
+aP3mX9t2SkCipdSsUs+Uuyib9MLA71ApW90AGiRm6HtOCxR0c3+qQRNIdFVm8mnM
+hCxXRexf6Z2wZdXXy6uY0LVRgI0o31NPJPk8l2Hnb/kHGxjyUFzEWh65J/eA368d
+4m8uF+Rr7WWlpQjwgWHU12kGThEVFFBFh2gmeIjYZdDDVhCi2mQ6lGSV2Pt7pZYL
+/PPChWLBqrVBkIUQ0GV22nRYvGdaIv2LVPu8PggbPs/wwh35nJ3rUQyJF55CFV5y
+WIWAWXfRYTKG9jkt+ncjZLEBxDO26zzO/MjIVPZxGyYryXEOgr6xp38xbyX9FpjL
+KBaIueLWEyphVjBb1uUpDGx+UDYe9vbJjPUAEQEAAYkCJQQYAQIADwIbDAUCW0nL
+2gUJB4HPNwAKCRBoc5PuN9Eo+ITIEACXjr4+9VROULXikv1m5awC2ciSMkUm0pXa
+hdurqcWvjK31rvVn6Q0K8opt/1nHZgsN1hYSjP+xPuATfCazp82mJyCkIAESqfMU
+CZa5fdiq0+OMRmOq/F5yKf/2TYTF6DuUnee58J9xks9+f0v1lnT7tj7KH6nLHhXo
+TCWbKGyxCccFbOOtPUJnh9yfIKgHXJeg9yLWBTqjPOA5BZ825kjRAq+11lhW05CS
+BtygxG848Y3ntsJ5XqmyhZkWAQIi0hi3Y7AkPd/XxhAFmC6819ek3d2H3/u9S46D
+DFvxrTwkzJzcF3MVc6Ft+wXFWEeZDgzXy68yzp9vF5OfOIpLohSCCo9G3dhvCX2h
+BLCkV9gA5x5y3MqeYQ6UQ3AMR97C0pzyY6umic0+Zio4P8rtZJ+sOnJnXHh97X1X
+2gdP+2bt95D2ODmbSZpMArDpSCj/JljDy0sxcTiISN3RdKYxFDmVuaVIC3rGEU66
+69NTW1u4WdcxHUCuq2VVFlvhfHPXa6pQdF/cPb/ziq/OPluK0SZHW2wodYUVJGn4
+b7qt5AEGijxuTiSCjaxIqyvKHoWFZt7oxxz2RDhRH/8csGSTjHXyUFq/HV4UsA+m
+vKjfECIS0gP6WutPELItLo6LSwXl9OxmoflxBHewTw0QOtEM1dS8Mo1KAn52b3QW
+DGYA9maD2w==
+=/1wA
+-----END PGP PUBLIC KEY BLOCK-----
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+
+mQGiBEab+moRBACDH5yKqS3wcc5bdxY7PBNuwKvF5TKMfagmSvuRDtZjjIIWaA/n
+Z1KboV9Gq5g7kP7+Kfu+Qgd8u65eVsWwmPW10fXvj3aCU53glx2EdGdrHcgiyH2g
+EQfPiyBw+trIppWFRV0IDXSLMA1FNC92t2nSG/VFHaPTVwcgkIRSfcXDvwCglGdE
+a6f4uLqoNHP+m4yYnzapFuMD/R4+2AJDAvEWKDdYCGZzlawjAmmWyXrmT7/C/mx9
+8qUR473l4buXjHgDkkXXlHqdzil1vK85PhrKzNJDCCmlHUJNz+QwiAMOLwpD+kwV
+Pb57RG7y+a5JQ5+jtVw4RlUxZIk/wj2An9YBO3A5vR7PdjM32ZJCN2+aM4dYfNzQ
+xQKTA/47icvBaBVTl9rztjg2pd2Aqpc1P/GsIYLGj7XjnnJvGAENBHSH1QjpZMJG
+CTS9oJ+B0/wrIr+pA+MdFgYAb6ojMQJOO6UChjWWSGjMFcs/CeXhxlLBido3DtAE
+TbNTwO6OEfAvdosvTdhJFnwvZlJ+zZGGy5CrF2Fd9PUe9tmASbQoQWxhbiBDb29w
+ZXJzbWl0aCA8YWxhbmNAZnJlZWRlc2t0b3Aub3JnPoh/BBMRCgA/AhsDBgsJCAcD
+AgYVCAIJCgsEFgIDAQIeAQIXgBYhBEoZPAbTXnxnD6TvC6L7nggfLRMOBQJkQs2e
+BQknDNS0AAoJEKL7nggfLRMOUgAAoIyNXuU5kOotpZrKvPrjQATbsPClAJ9wkh/n
+3IAobV6aDLb1qOt5RT3Pp7QrQWxhbiBDb29wZXJzbWl0aCA8YWxhbi5jb29wZXJz
+bWl0aEBzdW4uY29tPohmBBMRAgAmAhsDBgsJCAcDAgQVAggDBBYCAwECHgECF4AF
+AlF50WIFCRRD2HgACgkQovueCB8tEw7qxQCfSOXeH2tjg4nXcu7uewpyXKEOk6oA
+oIWBOZoMZG4NHGWguD2N+2uftru+tC5BbGFuIENvb3BlcnNtaXRoIDxhbGFuLmNv
+b3BlcnNtaXRoQG9yYWNsZS5jb20+iIIEExEKAEICGwMGCwkIBwMCBhUIAgkKCwQW
+AgMBAh4BAheAAhkBFiEEShk8BtNefGcPpO8LovueCB8tEw4FAmRCzZ4FCScM1LQA
+CgkQovueCB8tEw7V6wCdHgVUPrQGkNj9smnlELVVeE5WWjgAoImCxB5V4YvEOUy6
+ECZtwKQ6Np+puQINBEab+m8QCACaeWlJYaOa1IaUuopO7KbAEBT0bsnDyjDbtCeB
+dL/zqD89PObKFfMrS4+B6ghpNmWpodY0oJYmcLpPZu1e1oMcfpwSWsKVg1/3iizm
+xK3w8EOHXzhb421y0DJjZ0DWlBiFizz95fBrgThzKutAbvkz4Ietu71J7S+UREBE
+Kk1yM1gulNdELA25S8CL3TlUnRhecZfzdZsl80ir67GcFraBXPBvb/YXAo4bTdbJ
+vW+zo3OdYmV73HrXLz8uUQOgFFt8hBMWnnVHS57yBfbIxWCHyVhVWIdgIGr5xq3u
+nxfjVdr9838d8EU+x/f/5uGCA2CZoGYMkHHF3oE8HxgSKt6LAAMGB/9RYjFDDBEu
+SJc9pw6iHfgwlhAgFOsILyBUXF62VsQ8Qfma55kwl6AA9MXnaXzpER6fF01XP/TF
+PsEPpWUcdjkOjCVtFxrhbx5veCmRf/B81gNgIe1OJ9Pt7C6ZOs1nHmats2TRdMlU
+48A/fPsTStUPbD/GjinbYPMjB94YrCkeq3Vvpn1+atEPBqJdoWbBR/6siIZz84Vw
+5J7jofBOYjJM63o1V1BJ/vmM49t14z+fUjmy851ZMsSPpASPar1RD3xQQ8m4x1qi
+3IEXAqVwM44sxIvv3d9Yg2thKyR1eEeNqG3nBaoj56A7Yl9yIj+8/X5sTOGlj2WU
+ar6vtYbecJJ/iEkEGBECAAkFAkab+m8CGwwACgkQovueCB8tEw6C6ACgh2EacP13
+8hLp8NWC/7jGc5e/KiEAnRVXFzW6g/N6lYVuC8u4cgy/KNGiuQINBFF51SEBEACh
+6YzpmNOep9LbGaFvauXoLDHJebSgvLopq4TtOZ56vuf+tP2HnQvvlfXPW/9/1+zt
+vC8gZALQYA9MSWUg0NA4U8ygKegM40LQbyOvLopxsMUo0/qEKTzMaoQNSNJM6BAw
+nRKlfh7FTgBHWwIv7T/W23Uw88FXPDFg64mpgVKnOd0W9NDH38veN+VIE0uIAITM
+ADYcul2ZjjAYSEzs8RSVBA0wybS2xNG9Yz0UEuzR0IGpNYp5rrzzHN8ALqBHEgkr
+dcbqW2LAaFzoyucAv3hNSyHWaEcltjpE9eNA6g2fM30SA/OtIZ4q52IG9PUzzbY/
+d6bnVAFLek/frXCzeP68YcRzsLHuuihhzuG0Na2epn3KOkmHfVetHC+oQtnjHnMs
+WZYjLte5Y+xF/JcVB/qJiIB3OxxqMlEelflgetHMVo+b9Oc3Xi3mdmiwXDxYcZXw
+xEpvH+Hs3SUg0tUGZgtD+Yd0SX4YX5nQa/JB86+9ddQ9fzu9QL9VVeRcifWMVEsa
+eCD7syuyW24FR3AOj716w81lLJZmYDwA6Sh+IU5hvzXQqoRF3Ln72OrOcjZLAfF9
+0Xjb7kksnxp0c95L7citF8wuykV9j6riyOgPCiaax8GRqhAdTQxvG9ommJ4ITo8n
+yGBFUBd5xn2gcL+JawjxKGUlGivH/zUGYzVPDUMk5wARAQABiQKFBBgRCgAmAhsC
+FiEEShk8BtNefGcPpO8LovueCB8tEw4FAmRCzYoFCRwu+ekCKQkQovueCB8tEw7B
+XSAEGQEKAAYFAlF51SEACgkQz98UiCjGQqfW5g//dOdJHt23cdMyz5VADaE7u+L0
+E+eX9GtHF4J649eXsui59EtbHh2nXdGhd5SqQ8FDi9GCEKaQ4S31n/YBLEBCkj7R
+0IMikW2o78/JxDovB8+aL606hgmafNVx1aIshIglrl8Xlu3sjeAvG48W6YjdL2mf
+rIDHjIVwOZsMihbOJvST6Q3upHdnmjDtM5HCQmI5NEXDWYj6IZuhJnnrDWwNsyYV
+4KPoUBxAcqIyCeZbVssuWWnHPXX8VavVq98vpVynfGzGYpJbDj19C/utMjKGI5dc
+vbVaucA7X/oktxrxS6SBDhuIaAE94ZHlbxqfyHfETI/La2Z/ALDAtYdhJR2gSkTH
+yKSW1QqYlulSfB//lnna44mmTuRONbDNgb0FGSvtsBMZ80iHDqPgUfS60kxCfFrs
+SGfTFU+X4QAzpTtUJEcr+J4HULDeMfwOgghVfmKxFXWfud8xDaCXuywLTtVgMCZp
+4P7MAyuJlaxsFTu+c1Vly94grk4UMtALLMqCXSosA490gLTSdg3HSwxt2Q/LJdy4
+27ZIMvjGXIruns8U/OmL9dVgWu3bJHsL68Skx8Ts63qTN9QXM/PB+8VwOaC7PJ+g
+6t40DleOmdsS8cN31yf5KB8rsL4un4u1yrMJfpnSblPMu5wJi3kjoA+Dd5ZFqx9n
+Ti4wBjfVYGCPsleq59IkbACfe7lUvmRzyej/KOK2nO+1k7WIGGYAn3ZGvxh7+DJN
+THy/zBUuNZ0nWWb8
+=KE+F
+-----END PGP PUBLIC KEY BLOCK-----
--- libsm-1.2.6.orig/debian/watch
+++ libsm-1.2.6/debian/watch
@@ -0,0 +1,4 @@
+#git=https://gitlab.freedesktop.org/xorg/lib/libSM
+version=3
+opts=pgpsigurlmangle=s/$/.sig/ \
+https://xorg.freedesktop.org/releases/individual/lib/ libSM-(.*)\.tar\.gz
