--- libxtst-1.2.5.orig/.gitlab-ci.yml
+++ libxtst-1.2.5/.gitlab-ci.yml
@@ -0,0 +1,99 @@
+# vim: set expandtab shiftwidth=2 tabstop=8 textwidth=0 filetype=yaml:
+#
+# This CI uses the freedesktop.org ci-templates.
+# Please see the ci-templates documentation for details:
+# https://freedesktop.pages.freedesktop.org/ci-templates/
+
+.templates_sha: &template_sha 185ede0e9b9b1924b92306ab8b882a6294e92613 # see https://docs.gitlab.com/ee/ci/yaml/#includefile
+
+
+include:
+  # Debian container builder template
+  - project: 'freedesktop/ci-templates'
+    ref: *template_sha
+    file: '/templates/debian.yml'
+  - project: 'freedesktop/ci-templates'
+    ref: *template_sha
+    file: '/templates/ci-fairy.yml'
+  - template: Security/SAST.gitlab-ci.yml
+
+
+stages:
+  - prep             # prep work like rebuilding the container images if there is a change
+  - build            # for actually building and testing things in a container
+  - test
+  - deploy
+
+
+variables:
+  FDO_UPSTREAM_REPO: 'xorg/lib/libXtst'
+  FDO_DISTRIBUTION_VERSION: 'stable'
+  # The tag should be updated each time the list of packages is updated.
+  # Changing a tag forces the associated image to be rebuilt.
+  # Note: the tag has no meaning, we use a date format purely for readability
+  FDO_DISTRIBUTION_TAG:  '2023-10-13.0'
+  FDO_DISTRIBUTION_PACKAGES: 'git gcc pkgconf autoconf automake libtool make xutils-dev x11proto-dev libx11-dev libxext-dev libxi-dev xmlto xorg-sgml-doctools w3m xsltproc fop xz-utils'
+
+
+#
+# Verify that commit messages are as expected
+#
+check-commits:
+  extends:
+    - .fdo.ci-fairy
+  stage: prep
+  script:
+    - ci-fairy check-commits --junit-xml=results.xml
+  except:
+    - master@xorg/lib/libXtst
+  variables:
+    GIT_DEPTH: 100
+  artifacts:
+    reports:
+      junit: results.xml
+
+#
+# Verify that the merge request has the allow-collaboration checkbox ticked
+#
+check-merge-request:
+  extends:
+    - .fdo.ci-fairy
+  stage: deploy
+  script:
+    - ci-fairy check-merge-request --require-allow-collaboration --junit-xml=results.xml
+  artifacts:
+    when: on_failure
+    reports:
+      junit: results.xml
+  allow_failure: true
+
+
+#
+# Build a container with the given tag and the packages pre-installed.
+# This only happens if/when the tag changes, otherwise the existing image is
+# re-used.
+#
+container-prep:
+  extends:
+    - .fdo.container-build@debian
+  stage: prep
+  variables:
+    GIT_STRATEGY: none
+
+
+#
+# The default build, runs on the image built above.
+#
+build:
+  stage: build
+  extends:
+    - .fdo.distribution-image@debian
+  script:
+    - autoreconf -ivf
+    - mkdir _builddir
+    - pushd _builddir > /dev/null
+    - ../configure --disable-silent-rules --enable-specs
+    - make
+    - make check
+    - make distcheck
+    - popd > /dev/null
--- libxtst-1.2.5.orig/ChangeLog
+++ libxtst-1.2.5/ChangeLog
@@ -1,318 +1,3 @@
-commit abf3ff74b97c4231d2324d66318bfa1d22e44913
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Thu Aug 1 17:25:18 2024 -0700
-
-    libXtst 1.2.5
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit 4e4eb10495cc0d24b02c4cc82d86f3977f3cc95f
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Sun Jul 21 10:05:36 2024 -0700
-
-    parse_reply_call_callback: avoid NULL dereference if reply is missing data
-    
-    Clears up 7 -Wanalyzer-null-dereference warnings from gcc 14.1
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-    Part-of: <https://gitlab.freedesktop.org/xorg/lib/libxtst/-/merge_requests/7>
-
-commit 3f05df5a6c5140dc3d44f35b9fb635cca8b682c1
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Sat Jul 20 17:03:59 2024 -0700
-
-    XRecordFreeState: avoid NULL dereference when called in error path
-    
-    If the client_info pointer is NULL (for instance, if we decided
-    the number of entries would cause an integer overflow), then
-    don't attempt to walk it to free the entries.
-    
-    Found by gcc 14.1:
-    
-    XRecord.c:513:31: warning: dereference of NULL ‘0’ [CWE-476]
-     [-Wanalyzer-null-dereference]
-      513 |         if (state->client_info[i]->ranges) {
-          |             ~~~~~~~~~~~~~~~~~~^~~
-    [...]
-        |  452 |         ret->client_info = client_inf;
-        |      |         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-        |      |                          |
-        |      |                          (10) ‘client_inf’ is NULL
-    [...]
-        |  457 |            XRecordFreeState(ret);
-        |      |            ^~~~~~~~~~~~~~~~~~~~~
-        |      |            |
-        |      |            (14) ...to here
-        |      |            (15) calling ‘XRecordFreeState’ from ‘XRecordGetContext’
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-    Part-of: <https://gitlab.freedesktop.org/xorg/lib/libxtst/-/merge_requests/7>
-
-commit eb8370d5e4cf3d68f05b679d3679f695542eddb1
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Sat Jul 20 16:35:48 2024 -0700
-
-    XRecordGetContext: Avoid double free in error path
-    
-    XRecordFreeState() will handle the free for us, so let it.
-    
-    Found by gcc 14.1:
-    
-    XRecord.c:514:31: warning: use after ‘free’ of ‘*state.client_info + i * 8’
-     [CWE-416] [-Wanalyzer-use-after-free]
-      514 |         if (state->client_info[i]->ranges) {
-          |             ~~~~~~~~~~~~~~~~~~^~~
-    [...]
-        |  455 |            free(client_inf);
-        |      |            ~~~~~~~~~~~~~~~~
-        |      |            |
-        |      |            (18) freed here
-    
-    Fixes: e7e04b7 ("integer overflow in XRecordGetContext() [CVE-2013-2063]")
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-    Part-of: <https://gitlab.freedesktop.org/xorg/lib/libxtst/-/merge_requests/7>
-
-commit c542acce77be47156e53b74f5f9ca7156b3de7f5
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Fri Oct 13 13:53:39 2023 -0700
-
-    gitlab CI: add xz-utils to container for "make distcheck"
-    
-    Also update to latest ci-templates to support new Debian "stable" release
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit b0d55c9d98269aeefb521f1f2ce464288f176326
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Wed Mar 8 16:05:18 2023 -0800
-
-    configure: Use LT_INIT from libtool 2 instead of deprecated AC_PROG_LIBTOOL
-    
-    AC_PROG_LIBTOOL was replaced by LT_INIT in libtool 2 in 2008,
-    so it's time to rely on it.
-    
-    Clears autoconf warnings:
-    
-    configure.ac:34: warning: The macro `AC_PROG_LIBTOOL' is obsolete.
-    configure.ac:34: You should run autoupdate.
-    aclocal.m4:3465: AC_PROG_LIBTOOL is expanded from...
-    configure.ac:34: the top level
-    
-    libtoolize: Consider adding 'AC_CONFIG_MACRO_DIRS([m4])' to configure.ac,
-    libtoolize: and rerunning libtoolize and aclocal.
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit a6ea11a5b35e838879152e8141c9c68813994848
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Sat Feb 25 09:33:45 2023 -0800
-
-    Remove "All rights reserved" from Oracle copyright notices
-    
-    Oracle no longer includes this term in our copyright & license notices.
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit 1e8215d8b4c85343da4301cd022639ae5f36756c
-Author: Christos Zoulas <christos@NetBSD.org>
-Date:   Wed Sep 28 09:46:26 2022 +0200
-
-    Coverity CID 1373522: Fix memory leak
-    
-    Signed-off-by: Thomas Klausner <wiz@gatalith.at>
-
-commit 99b89c3bcb0ebb0b6dd86bfdc9d276715eaea889
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Mon Sep 26 18:07:24 2022 -0700
-
-    libXtst 1.2.4
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit 14a44d0e3e6c3d9e757e2fdd143587efe532f1e9
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Sat Aug 20 10:39:35 2022 -0700
-
-    Remove obsolete casts from Xmalloc() and Xcalloc() calls
-    
-    Not needed in C89 and later
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit afc015217221acd8667a96111b147f3a742a2c4e
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Thu Aug 18 19:12:42 2022 -0700
-
-    Variable scope reductions as suggested by cppcheck
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit ff3456b04e1b42fd77e0db061bf7a563108b5868
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Thu Aug 18 19:02:35 2022 -0700
-
-    Resolve -Wsign-compare warnings
-    
-    XRecord.c: In function ‘XRecordFreeState’:
-    XRecord.c:515:15: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
-         for(i=0; i<state->nclients; i++) {
-                   ^
-    XRecord.c: In function ‘parse_reply_call_callback’:
-    XRecord.c:752:25: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
-       if (current_index + 4 > rep->length << 2)
-                             ^
-    XRecord.c:759:28: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
-          if (current_index + 1 > rep->length << 2)
-                                ^
-    XRecord.c:763:25: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
-       if (current_index + 8 > rep->length << 2)
-                             ^
-    XRecord.c:777:25: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
-       if (current_index + 4 > rep->length << 2)
-                             ^
-    XRecord.c:785:25: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
-       if (current_index + 4 > rep->length << 2)
-                             ^
-    XRecord.c:792:28: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
-          if (current_index + 4 > rep->length<<2)
-                                ^
-    XRecord.c:797:25: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
-       if (current_index + 8 > rep->length << 2)
-                             ^
-    XRecord.c:810:28: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
-          if (current_index + 8 > rep->length << 2)
-                                ^
-    XRecord.c:818:25: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
-       if (current_index + 4 > rep->length << 2)
-                             ^
-    XRecord.c:824:31: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
-          } else if (current_index < rep->length << 2)
-                                   ^
-    XRecord.c:830:24: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
-          if (current_index < rep->length << 2)
-                            ^
-    XRecord.c:859:27: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
-         } while (current_index<rep->length<<2);
-                               ^
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit f8a2329d8a24c0901d945986232267c02f080fc4
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Thu Aug 18 18:52:40 2022 -0700
-
-    send_axes: Mark switch statement fallthrough as intentional
-    
-    Quiets gcc warnings:
-    XTest.c: In function ‘send_axes’:
-    XTest.c:274:19: warning: this statement may fall through [-Wimplicit-fallthrough=]
-          ev.valuator5 = *(axes+5);
-          ~~~~~~~~~~~~~^~~~~~~~~~~
-    XTest.c:275:2: note: here
-      case 5:
-      ^~~~
-    XTest.c:276:19: warning: this statement may fall through [-Wimplicit-fallthrough=]
-          ev.valuator4 = *(axes+4);
-          ~~~~~~~~~~~~~^~~~~~~~~~~
-    XTest.c:277:2: note: here
-      case 4:
-      ^~~~
-    XTest.c:278:19: warning: this statement may fall through [-Wimplicit-fallthrough=]
-          ev.valuator3 = *(axes+3);
-          ~~~~~~~~~~~~~^~~~~~~~~~~
-    XTest.c:279:2: note: here
-      case 3:
-      ^~~~
-    XTest.c:280:19: warning: this statement may fall through [-Wimplicit-fallthrough=]
-          ev.valuator2 = *(axes+2);
-          ~~~~~~~~~~~~~^~~~~~~~~~~
-    XTest.c:281:2: note: here
-      case 2:
-      ^~~~
-    XTest.c:282:19: warning: this statement may fall through [-Wimplicit-fallthrough=]
-          ev.valuator1 = *(axes+1);
-          ~~~~~~~~~~~~~^~~~~~~~~~~
-    XTest.c:283:2: note: here
-      case 1:
-      ^~~~
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit 80c35fd74d99039949be2522f18f4040e2f6eec3
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Sun Jul 17 18:23:10 2022 -0700
-
-    gitlab CI: add a basic build test
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit 0077e7828bd26869d7a3e9ec6158ab09bf646ef3
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Sun Jul 17 18:21:31 2022 -0700
-
-    Fix spelling/wording issues
-    
-    Found by using:
-        codespell --builtin clear,rare,usage,informal,code,names
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit 63dc5cc8fe89807c7945949d1d86da44ce69687a
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Sun Jul 17 18:20:38 2022 -0700
-
-    Build xz tarballs instead of bzip2
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit 769598f20386845f5839fbc68dfed1f4245197be
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Fri Dec 7 19:54:52 2018 -0800
-
-    Update configure.ac bug URL for gitlab migration
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit 9d20b76d717add6401d66f272656d3cf763d7130
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Mon Nov 19 22:46:44 2018 -0800
-
-    Update README for gitlab migration
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit 7e083f2e618ce063d1ab2885671a0c5ca8472772
-Author: Mihail Konev <k.mvc@ya.ru>
-Date:   Thu Jan 26 13:52:49 2017 +1000
-
-    autogen: add default patch prefix
-    
-    Signed-off-by: Mihail Konev <k.mvc@ya.ru>
-
-commit c051091cecad4c22dab362f3d3b7463ad2a52754
-Author: Emil Velikov <emil.l.velikov@gmail.com>
-Date:   Mon Mar 9 12:00:52 2015 +0000
-
-    autogen.sh: use quoted string variables
-    
-    Place quotes around the $srcdir, $ORIGDIR and $0 variables to prevent
-    fall-outs, when they contain space.
-    
-    Signed-off-by: Emil Velikov <emil.l.velikov@gmail.com>
-    Reviewed-by: Peter Hutterer <peter.hutterer@who-t.net>
-    Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
-
-commit a4994d5f8bc949eabe2c9d8a1a7af4bfcb68df9c
-Author: Peter Hutterer <peter.hutterer@who-t.net>
-Date:   Tue Jan 24 10:32:07 2017 +1000
-
-    autogen.sh: use exec instead of waiting for configure to finish
-    
-    Syncs the invocation of configure with the one from the server.
-    
-    Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
-    Reviewed-by: Emil Velikov <emil.velikov@collabora.com>
-
 commit 9f5621a410f18149d4c76b02daa7f1a98b4a2c16
 Author: Matthieu Herrb <matthieu.herrb@laas.fr>
 Date:   Tue Oct 4 21:28:17 2016 +0200
--- libxtst-1.2.5.orig/autogen.sh
+++ libxtst-1.2.5/autogen.sh
@@ -0,0 +1,17 @@
+#! /bin/sh
+
+srcdir=`dirname "$0"`
+test -z "$srcdir" && srcdir=.
+
+ORIGDIR=`pwd`
+cd "$srcdir"
+
+autoreconf -v --install || exit 1
+cd "$ORIGDIR" || exit $?
+
+git config --local --get format.subjectPrefix >/dev/null 2>&1 ||
+    git config --local format.subjectPrefix "PATCH libXtst"
+
+if test -z "$NOCONFIGURE"; then
+    exec "$srcdir"/configure "$@"
+fi
--- libxtst-1.2.5.orig/debian/changelog
+++ libxtst-1.2.5/debian/changelog
@@ -0,0 +1,265 @@
+libxtst (2:1.2.5-1) unstable; urgency=medium
+
+  [ Timo Aaltonen ]
+  * New upstream release.
+  * control: Migrate to x11proto-dev.
+  * upstream: Update signing-key.asc.
+  * control, rules: Migrate to debhelper-compat 13.
+  * control: Migrate to pkgconf.
+  * control: Drop duplicat xmlto build-dep.
+  * control: Drop obsolete constraints.
+  * control: Bump policy to 4.7.0.
+
+  [ Debian Janitor ]
+  * Replace XC-Package-Type with Package-Type.
+  * Update Vcs-* headers from URL redirect.
+  * Use canonical URL in Vcs-Git.
+
+  [ Emilio Pozuelo Monfort ]
+  * Move doc build-deps to Build-Depends-Indep. Thanks Samuel Thibault.
+    Closes: #1055384.
+
+ -- Timo Aaltonen <tjaalton@debian.org>  Wed, 22 Jan 2025 09:19:45 +0200
+
+libxtst (2:1.2.3-1) unstable; urgency=low
+
+  [ Andreas Boll ]
+  * New upstream release.
+    - Fixes CVE-2016-7951 and CVE-2016-7952 (Closes: #840444).
+  * Bump libx11-dev build-dep to 2:1.6.0 per configure.ac.
+  * Let uscan verify tarball signatures.
+  * Remove Cyril from Uploaders.
+  * Update a bunch of URLs in packaging to https.
+  * Fix lintian error: pre-depends-directly-on-multiarch-support.
+
+  [ Julien Cristau ]
+  * Bump debhelper build-dep to 8.1.3 for ${misc:Pre-Depends}.
+
+  [ Emilio Pozuelo Monfort ]
+  * Bump debhelper compat to 10.
+  * Switch from xsfbs to dh.
+  * Switch to -dbgsym packages.
+  * Bump Standards-Version to 3.9.8, no changes.
+  * Drop automake and libtool build-dependencies, debhelper takes care
+    of that now.
+  * Drop unneeded override for dh_installchangelogs, it already installs
+    the upstream ChangeLog automatically.
+
+ -- Emilio Pozuelo Monfort <pochu@debian.org>  Tue, 06 Dec 2016 01:38:08 +0100
+
+libxtst (2:1.2.2-1) unstable; urgency=low
+
+  * New upstream release.
+  * debian/copyright: add copyright notices and licenses from the manpages and
+    specs.  Thanks, Ansgar!
+  * Bump debhelper compat level to 7.
+  * Use dpkg-buildflags.
+  * Disable silent build rules.
+  * Use dh_prep instead of dh_clean -k.
+
+ -- Julien Cristau <jcristau@debian.org>  Mon, 12 Aug 2013 20:33:03 +0200
+
+libxtst (2:1.2.1-1+deb7u1) wheezy-security; urgency=high
+
+  * integer overflow in XRecordGetContext() [CVE-2013-2063]
+
+ -- Julien Cristau <jcristau@debian.org>  Tue, 14 May 2013 20:12:48 +0200
+
+libxtst (2:1.2.1-1) unstable; urgency=low
+
+  * New upstream release.
+  * Split documentation to a separate package.  Mark libxtst-dev Multi-Arch:
+    same.
+  * Don't build the documentation from the build-arch target.
+  * Bump build-deps on xutils-dev, xmlto and xorg-sgml-doctools per
+    configure.ac.
+
+ -- Julien Cristau <jcristau@debian.org>  Mon, 23 Apr 2012 22:55:46 +0200
+
+libxtst (2:1.2.0-4) unstable; urgency=low
+
+  [ Julien Cristau ]
+  * Build for multiarch (closes: #639348).  Thanks, Peter Samuelson!
+  * Don't use dh_testroot in clean.
+
+  [ Steve Langasek ]
+  * Don't mark libxtst-dev Multi-Arch: same, because its documentation isn't
+    identical across architectures.
+
+ -- Steve Langasek <vorlon@debian.org>  Fri, 21 Oct 2011 20:17:23 -0700
+
+libxtst (2:1.2.0-3) unstable; urgency=low
+
+  * Upload to unstable.
+
+ -- Cyril Brulebois <kibi@debian.org>  Wed, 20 Jul 2011 23:10:53 +0200
+
+libxtst (2:1.2.0-2) experimental; urgency=low
+
+  * Add libxtst6-udeb, thanks to Thibaut GIRKA (Closes: #592504). This is
+    a prerequisite for on-screen keyboard support.
+  * Update Uploaders list. Thanks, David, Andres, and Brice!
+
+ -- Cyril Brulebois <kibi@debian.org>  Thu, 24 Feb 2011 16:47:44 +0100
+
+libxtst (2:1.2.0-1) unstable; urgency=low
+
+  * New upstream release.
+  * Bump xutils-dev build-dep for new macros.
+  * Update debian/copyright from upstream COPYING.
+  * Add xmlto, xorg-sgml-doctools, and w3m build-dep for the doc.
+  * Pass --with-xmlto and --without-fop (we want html and txt only).
+  * Pass --docdir=/usr/share/doc/libxtst-dev and add this directory to
+    the libxtst-dev.install file.
+  * Switch from --list-missing to --fail-missing for additional safety.
+  * Exclude libXtst.la from dh_install accordingly.
+  * Add myself to Uploaders.
+
+ -- Cyril Brulebois <kibi@debian.org>  Fri, 19 Nov 2010 14:52:30 +0100
+
+libxtst (2:1.1.0-3) unstable; urgency=low
+
+  [ Julien Cristau ]
+  * Rename the build directory to not include DEB_BUILD_GNU_TYPE for no
+    good reason.  Thanks, Colin Watson!
+
+  [ Brice Goglin ]
+  * Fix short description, thanks Nick Shaforostoff, closes: #565234.
+
+ -- Julien Cristau <jcristau@debian.org>  Wed, 23 Jun 2010 12:34:43 +0100
+
+libxtst (2:1.1.0-2) unstable; urgency=low
+
+  * Upload to unstable.
+
+ -- Julien Cristau <jcristau@debian.org>  Wed, 06 Jan 2010 13:02:50 +0000
+
+libxtst (2:1.1.0-1) experimental; urgency=low
+
+  [ Timo Aaltonen ]
+  * New upstream release.
+  * Bump the build-dep on xutils-dev (>= 1:7.5~1).
+  * Bump the build-dep on libxext-dev (>= 1:1.0.99.4).
+
+ -- Julien Cristau <jcristau@debian.org>  Thu, 26 Nov 2009 17:57:54 +0100
+
+libxtst (2:1.0.99.2-2) experimental; urgency=low
+
+  * Brown paper bag upload.
+  * Actually install the headers.
+
+ -- Julien Cristau <jcristau@debian.org>  Tue, 29 Sep 2009 17:16:59 +0200
+
+libxtst (2:1.0.99.2-1) experimental; urgency=low
+
+  [ Brice Goglin ]
+  * Bump Standards-Version to 3.7.3 (no changes).
+  * Drop the XS- prefix from Vcs-Git and Vcs-Browser fields in debian/control.
+  * Add a link to www.X.org and a reference to the upstream module
+    in the long description.
+  * Install the upstream ChangeLog.
+  * Add README.source, bump Standards-Version to 3.8.2.
+  * Use updated xsfbs, closes: #538589.
+  * Move -dbg package to section debug.
+
+  [ Julien Cristau ]
+  * Run autoreconf at build time.
+  * Parse space-separated DEB_BUILD_OPTIONS, handle parallel=N.
+  * Drop pre-dependency on x11-common from libxtst-dev.  This was needed for
+    upgrades from sarge.
+  * Bump Standards-Version to 3.8.3.
+  * New upstream release candidate
+    - record.h and XTest.h moved here from recordproto and xextproto
+  * Bump {Build-,}Depends, add Replaces on x11proto-xext-dev and
+    x11proto-record-dev.
+  * Drop -1 debian revision from Build-Depends [lintian].
+
+ -- Julien Cristau <jcristau@debian.org>  Tue, 29 Sep 2009 15:53:21 +0200
+
+libxtst (2:1.0.3-1) unstable; urgency=low
+
+  [ Julien Cristau ]
+  * Sync package sections with the override.
+  * Bump epoch so Ubuntu can sync this package.
+
+  [ Brice Goglin ]
+  * New upstream release.
+  * Add myself to Uploaders, and remove Branden and Fabio with their
+    permission.
+  * Add upstream URL to debian/copyright.
+  * Update 01_build_manpages.diff since its first half has been merged
+    upstream, and rename it to 01_fix_date_in_manpages.diff to match
+    the remaining part.
+
+ -- Brice Goglin <bgoglin@debian.org>  Sat, 25 Aug 2007 11:58:00 +0200
+
+libxtst (1:1.0.2-1) unstable; urgency=low
+
+  [ Drew Parsons ]
+  * dbg package has priority extra.
+
+  [ Julien Cristau ]
+  * Add XS-Vcs-* headers, and remove outdated CVS info.
+  * New upstream release.
+  * Build and install the new manpage, and add a new patch to make that work.
+    Accordingly, add build-deps on quilt and xmlto.
+  * Use ${binary:Version} instead of ${Source-Version} in debian/control.
+  * Update debian/copyright from COPYING.
+
+ -- Julien Cristau <jcristau@debian.org>  Thu, 07 Jun 2007 17:54:08 +0200
+
+libxtst (1:1.0.1-5) unstable; urgency=low
+
+  * Revert .la file addition.
+
+ -- Andres Salomon <dilinger@debian.org>  Mon, 24 Jul 2006 11:20:31 -0400
+
+libxtst (1:1.0.1-4) unstable; urgency=low
+
+  * Test for obj-$(DEB_BUILD_GNU_TYPE) before creating it during build;
+    idempotency fix.
+  * Run dh_install w/ --list-missing.
+  * Drop duplicate x11-common dep in -dev package.
+  * Bump standards version to 3.7.2.0.
+  * Version x11-common pre-dep in -dev package to 1:7.0.0 to match
+    the rest of Debian.
+  * Bump debhelper compat to 5.
+  * Fix dh_strip call to skip the -dbg package.
+  * Autoreconf w/ an updated xutils-dev so that manpages have the 'x'
+    extension dropped.
+  * Include .la files in -dev package.
+  * Don't attempt to install (non-existent) usr/include/X11/* stuff.
+  * Add a debian/copyright file.
+
+ -- Andres Salomon <dilinger@debian.org>  Mon, 24 Jul 2006 05:13:32 -0400
+
+libxtst (1:1.0.1-3) unstable; urgency=low
+
+  * Reorder makeshlib command in rules file so that ldconfig is run
+    properly. Thanks Drew Parsons and Steve Langasek.
+
+ -- David Nusinow <dnusinow@debian.org>  Tue, 18 Apr 2006 21:50:01 -0400
+
+libxtst (1:1.0.1-2) unstable; urgency=low
+
+  * Upload to unstable
+
+ -- David Nusinow <dnusinow@debian.org>  Thu, 23 Mar 2006 22:45:21 -0500
+
+libxtst (1:1.0.1-1) experimental; urgency=low
+
+  * First modular upload to Debian
+
+ -- David Nusinow <dnusinow@debian.org>  Thu, 29 Dec 2005 20:55:45 -0500
+
+libxtst (1:1.13.0-2) breezy; urgency=low
+
+  * Bump Build-Deps on libx11-dev and libxext-dev to avoid _XOPEN_SOURCE.
+
+ -- Daniel Stone <daniel.stone@ubuntu.com>  Sat, 23 Jul 2005 00:49:26 +1000
+
+libxtst (1:1.13.0-1) breezy; urgency=low
+
+  * First libxtst release.
+
+ -- Daniel Stone <daniel.stone@ubuntu.com>  Mon, 16 May 2005 22:10:17 +1000
--- libxtst-1.2.5.orig/debian/control
+++ libxtst-1.2.5/debian/control
@@ -0,0 +1,101 @@
+Source: libxtst
+Section: x11
+Priority: optional
+Maintainer: Debian X Strike Force <debian-x@lists.debian.org>
+Build-Depends:
+ debhelper-compat (= 13),
+ libx11-dev,
+ libxext-dev,
+ libxi-dev,
+ x11proto-dev,
+ pkgconf,
+ xutils-dev,
+ quilt,
+Build-Depends-Indep:
+ xmlto,
+ xorg-sgml-doctools,
+ w3m,
+Standards-Version: 4.7.0
+Vcs-Git: https://salsa.debian.org/xorg-team/lib/libxtst.git
+Vcs-Browser: https://salsa.debian.org/xorg-team/lib/libxtst
+
+Package: libxtst6
+Section: libs
+Architecture: any
+Multi-Arch: same
+Pre-Depends: ${misc:Pre-Depends}
+Depends: ${shlibs:Depends}, ${misc:Depends}, x11-common
+Description: X11 Testing -- Record extension library
+ libXtst provides an X Window System client interface to the Record
+ extension to the X protocol.
+ .
+ The Record extension allows X clients to synthesise input events, which
+ is useful for automated testing.
+ .
+ More information about X.Org can be found at:
+ <URL:https://www.X.org>
+ .
+ This module can be found at
+ git://anongit.freedesktop.org/git/xorg/lib/libXtst
+
+Package: libxtst6-udeb
+Package-Type: udeb
+Section: debian-installer
+Architecture: any
+Depends: ${shlibs:Depends}, ${misc:Depends}
+Description: X11 Testing -- Record extension library
+ This is a udeb, or a microdeb, for the debian-installer.
+
+Package: libxtst-dev
+Section: libdevel
+Architecture: any
+Multi-Arch: same
+Depends:
+ ${shlibs:Depends},
+ ${misc:Depends},
+ libxtst6 (= ${binary:Version}),
+ libx11-dev,
+ libxext-dev,
+ libxi-dev,
+ x11proto-dev,
+Description: X11 Record extension library (development headers)
+ libXtst provides an X Window System client interface to the Record
+ extension to the X protocol.
+ .
+ The Record extension allows X clients to synthesise input events, which
+ is useful for automated testing.
+ .
+ This package contains the development headers for the library found in
+ libxtst6.  Non-developers likely have little use for this package.
+ .
+ More information about X.Org can be found at:
+ <URL:https://www.X.org>
+ .
+ This module can be found at
+ git://anongit.freedesktop.org/git/xorg/lib/libXtst
+
+Package: libxtst-doc
+Section: doc
+Architecture: all
+Multi-Arch: foreign
+Depends:
+ ${misc:Depends},
+Replaces:
+ libxtst-dev (<< 2:1.2.1),
+Breaks:
+ libxtst-dev (<< 2:1.2.1),
+Description: X11 Record extension library (documentation)
+ libXtst provides an X Window System client interface to the Record
+ extension to the X protocol.
+ .
+ The Record extension allows X clients to synthesise input events, which
+ is useful for automated testing.
+ .
+ This package contains the API documentation for the X Record and XTEST
+ extension libraries.  Non-developers likely have little use for this package.
+ .
+ More information about X.Org can be found at:
+ <URL:https://www.X.org>
+ .
+ This module can be found at
+ git://anongit.freedesktop.org/git/xorg/lib/libXtst
--- libxtst-1.2.5.orig/debian/copyright
+++ libxtst-1.2.5/debian/copyright
@@ -0,0 +1,119 @@
+This package was downloaded from
+https://xorg.freedesktop.org/releases/individual/lib/
+
+Copyright 1990, 1991 by UniSoft Group Limited
+Copyright 1992, 1993, 1995, 1998  The Open Group
+
+Permission to use, copy, modify, distribute, and sell this software and its
+documentation for any purpose is hereby granted without fee, provided that
+the above copyright notice appear in all copies and that both that
+copyright notice and this permission notice appear in supporting
+documentation.
+
+The above copyright notice and this permission notice shall be included
+in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+IN NO EVENT SHALL THE OPEN GROUP BE LIABLE FOR ANY CLAIM, DAMAGES OR
+OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+OTHER DEALINGS IN THE SOFTWARE.
+
+Except as contained in this notice, the name of The Open Group shall
+not be used in advertising or otherwise to promote the sale, use or
+other dealings in this Software without prior written authorization
+from The Open Group.
+
+***************************************************************************
+
+Copyright 1995 Network Computing Devices
+
+Permission to use, copy, modify, distribute, and sell this software and
+its documentation for any purpose is hereby granted without fee, provided
+that the above copyright notice appear in all copies and that both that
+copyright notice and this permission notice appear in supporting
+documentation, and that the name of Network Computing Devices
+not be used in advertising or publicity pertaining to distribution
+of the software without specific, written prior permission.
+
+NETWORK COMPUTING DEVICES DISCLAIMs ALL WARRANTIES WITH REGARD TO
+THIS SOFTWARE, INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY
+AND FITNESS, IN NO EVENT SHALL NETWORK COMPUTING DEVICES BE LIABLE
+FOR ANY SPECIAL, INDIRECT OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN
+AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
+OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+***************************************************************************
+
+Copyright 2005  Red Hat, Inc.
+
+Permission to use, copy, modify, distribute, and sell this software and its
+documentation for any purpose is hereby granted without fee, provided that
+the above copyright notice appear in all copies and that both that
+copyright notice and this permission notice appear in supporting
+documentation, and that the name of Red Hat not be used in
+advertising or publicity pertaining to distribution of the software without
+specific, written prior permission.  Red Hat makes no
+representations about the suitability of this software for any purpose.  It
+is provided "as is" without express or implied warranty.
+
+RED HAT DISCLAIMS ALL WARRANTIES WITH REGARD TO THIS SOFTWARE,
+INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS, IN NO
+EVENT SHALL RED HAT BE LIABLE FOR ANY SPECIAL, INDIRECT OR
+CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE,
+DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER
+TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
+PERFORMANCE OF THIS SOFTWARE.
+
+***************************************************************************
+
+Copyright © 1992 by UniSoft Group Ltd.
+
+Permission to use, copy, modify, and distribute this documentation for
+any purpose and without fee is hereby granted, provided that the above
+copyright notice and this permission notice appear in all copies.
+UniSoft makes no representations about the suitability for any purpose of
+the information in this document.  This documentation is provided "as is"
+without express or implied warranty.
+
+***************************************************************************
+
+Copyright © 1992, 1994, 1995 X Consortium
+
+Permission is hereby granted, free of charge, to any person obtaining a
+copy of this software and associated documentation files (the "Software"),
+to deal in the Software without restriction, including without limitation
+the rights to use, copy, modify, merge, publish, distribute, sublicense,
+and/or sell copies of the Software, and to permit persons to whom the
+Software is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
+THE X CONSORTIUM BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF
+OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+Except as contained in this notice, the name of the X Consortium shall not
+be used in advertising or otherwise to promote the sale, use or other
+dealings in this Software without prior written authorization from the
+X Consortium.
+
+***************************************************************************
+
+Copyright 1994 Network Computing Devices, Inc.
+
+Permission to use, copy, modify, distribute, and sell this
+documentation for any purpose is hereby granted without fee,
+provided that the above copyright notice and this permission
+notice appear in all copies.  Network Computing Devices, Inc.
+makes no representations about the suitability for any purpose
+of the information in this document.  This documentation is
+provided "as is" without express or implied warranty.
--- libxtst-1.2.5.orig/debian/libxtst-dev.install
+++ libxtst-1.2.5/debian/libxtst-dev.install
@@ -0,0 +1,5 @@
+usr/lib/*/libXtst.a
+usr/lib/*/libXtst.so
+usr/lib/*/pkgconfig/xtst.pc
+usr/share/man/man3
+usr/include/X11/extensions/
--- libxtst-1.2.5.orig/debian/libxtst-doc.install
+++ libxtst-1.2.5/debian/libxtst-doc.install
@@ -0,0 +1 @@
+usr/share/doc/libxtst-dev/
--- libxtst-1.2.5.orig/debian/libxtst6-udeb.install
+++ libxtst-1.2.5/debian/libxtst6-udeb.install
@@ -0,0 +1 @@
+usr/lib/*/libXtst.so.6* usr/lib
--- libxtst-1.2.5.orig/debian/libxtst6.install
+++ libxtst-1.2.5/debian/libxtst6.install
@@ -0,0 +1 @@
+usr/lib/*/libXtst.so.6*
--- libxtst-1.2.5.orig/debian/patches/01_fix_date_in_manpages.diff
+++ libxtst-1.2.5/debian/patches/01_fix_date_in_manpages.diff
@@ -0,0 +1,14 @@
+Make sure the manpage footer contains the date, not the date within
+<pubdate></pubdate>.
+
+--- a/man/XTest.xml
++++ b/man/XTest.xml
+@@ -43,7 +43,7 @@ X Consortium.
+ <!-- $Xorg: xtestlib.ms,v 1.3 2000/08/17 19:42:37 cpqbld Exp $ -->
+  <refentryinfo>
+   <productname>__vendorversion__</productname>
+-  <pubdate>6 June 2007</pubdate>
++  <date>6 June 2007</date>
+    <author><firstname>Kieron</firstname><surname>Drake</surname>
+    <affiliation><orgname>UniSoft Ltd.</orgname></affiliation>
+    </author>
--- libxtst-1.2.5.orig/debian/patches/series
+++ libxtst-1.2.5/debian/patches/series
@@ -0,0 +1 @@
+01_fix_date_in_manpages.diff
--- libxtst-1.2.5.orig/debian/rules
+++ libxtst-1.2.5/debian/rules
@@ -0,0 +1,20 @@
+#!/usr/bin/make -f
+
+# set this to the name of the main shlib's binary package
+PACKAGE = libxtst6
+
+%:
+	dh $@ --with quilt --builddirectory=build/
+
+override_dh_auto_configure-indep: docflags = --enable-specs --with-xmlto --without-fop
+override_dh_auto_configure-arch: docflags = --disable-specs
+override_dh_auto_configure-arch override_dh_auto_configure-indep:
+	dh_auto_configure -- $(docflags) \
+		--docdir=\$${prefix}/share/doc/libxtst-dev
+
+override_dh_install:
+	find debian/tmp/ -name '*.la' -exec rm '{}' ';'
+	dh_install
+
+override_dh_makeshlibs:
+	dh_makeshlibs -a --add-udeb=$(PACKAGE)-udeb
--- libxtst-1.2.5.orig/debian/upstream/signing-key.asc
+++ libxtst-1.2.5/debian/upstream/signing-key.asc
@@ -0,0 +1,125 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+
+mQINBFeKY50BEADAX0lod3IVceb/IWJn3kTAcO2P7PWlcBiyUDaq5b2kFkliKleZ
+ec4LoCHakQBlkRBMPNwOOxvADNk3tLQjBDpbYr6lQIrN+AxMGkXBhJ82T3bsDvlj
+3Z1wRJ1zVA7eMIktsk0FAoJxV1y7e3sBKcP0eTlXqXvR2djhi+FW+ueJDAJIFSkb
+uFirgwtX5t8nt8jCmIl75KNUKOakoENY3hLWtr16W8fO1JGkEhghI2mXcz664KTd
+MPZp6JH0/8UHTHzmATOCTqNxoDtMTi2l5059Lh/nhmso9moTYqyKmaJP2rnZUr62
+97sRMG4WcxaYfWpPyO3MCmDyGeh4sW0OC06PpED3i9xMzf/kMkMdY4ZIFcLRcPtf
+LIJhw+lc/GE1Rqe961IB5xCgnZezB7ZIL+ZlOAMwKGkq7lLbcZr2QZn84lpABKF0
+AvxECoJ4etmIcdbDVmsw18AhA3u9sr98hS5IXDyeos3Xwz6Abml8aPrhqhkKvo+J
+Kcq9FNYHg0RRlos0TqocjDzGnUjEYrmIopLcwIu2SnsNSJTygZGtqrpT+2sGEqvm
+k6Oyk95QCa580zqldvxe3CG0vrAfPvoG7irllM68TS4JcqqDHTq6eupUv9ZdIzXf
+eyTHa5cytGahgVtUcui1lzqcCBkqwN8TKl+0wCcEnxRasHJy3A2Gp+AG3wARAQAB
+tCJNYXR0aGlldSBIZXJyYiA8bWF0dGhpZXVAaGVycmIuZXU+iQI+BBMBAgAoBQJX
+imOdAhsDBQkDwmcABgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgAAKCRBoc5PuN9Eo
++PF1EACldzZPNYaC9H5E9sMn9pMsJTucBYVUy74Aw6MWAiAzRpxb9DmySmC2oEYW
+JJkwDTwv6M0Na0ed6zD79GKtAalORz2GppZpS7uoINClElWoM5TCYph6linyv9Wj
+OTlcbpX0Jqw0tdHNI2UOEjvBP3vW9kVYpEhfnHET8Ncp55j1hzoqxOhGIBE/67zc
+cLAenONAvA3YN3tHTGaOaFv+vuCFRJx9FpKbGHmdUPd3MtLqtaA4EQvDvDEholEI
+eWrjmdXJibSet6Amc5AIdFaQevZiADjjMh8MINw/6OEy9OB4s+z1RzgOrHgLiIZm
+dlP6WrNjXQwl2gmNPhctGaSHM+j2+3gckNGlI4LQYxNtKvI4iv/CoHDYmwgrcrZO
+TwFHfqt0LwqjpsU203Hw609oWYcxLeGZdITBjDz20UcfsmKQDqrBq3P1FuC5GBW3
+5bEa3wAhyE+/WKhJ94bXiHmpKsp50va3bEe17uQcYd8+E8L53aR7XP87qaHx//Mu
++OQa5Wc2d1OFHf1Mi62nbzr7pws/Mf7OSf/tnhRthuwtlfYnsUVo8usUKL/xStqo
+Ul4kc/Q81AlyaZfr7dbxsQWm2q3ksLaMaAxnk0p+kMXVzXZ9GKNOgUOJdbahORs5
+RU2f44xzfNavb63u3McADtaXskl+KHB4uDbGbGESVhm5PULk37QnTWF0dGhpZXUg
+SGVycmIgPG1hdHRoaWV1LmhlcnJiQGxhYXMuZnI+iQI+BBMBAgAoBQJXlJ63AhsD
+BQkDwmcABgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgAAKCRBoc5PuN9Eo+PKID/wM
+II+2d11clp1X7eZgkxkAHUhI2W3NSesuFnjkkQRKQoVMokDdeSOkBhMJuWoFfbZk
+jYs2VHU9029rDqcoDSqGwo2IffvrXXJ4SjOTjlvXS1lr/H2VdWRbq8ImnDwSsoiD
+dWB3dZyqzf7ABKZ7ccA+NMSs6NxeEN/0+0sTJ386Zp480ByNX0uPqYSq5lX/VEke
+nI8r02u2ZfuykhGkT0sM013VprfYLa+6HvF+QT9KfP220mqRbonaDkYvCxwjCMzd
+rUmvyqw3VsooUpg/W/PmDNeShSuOxebaGnFyGTNvTarElCBdynFD01dqOecOqfY8
+gy+PJ1aF1qjmf+RQD/SZq+gvgyXqyBhJy7zgJnzzNWzDlUIw0ZOLyZxzFR7lRV79
+2mrGgczlQr5rLAgBy2pgwsCmP7nFx50r4ft2juugnQixoOBU/YfhBplM76EROaCc
+MTs5nPEqzJ9p4SNkPcK8AroR2Ka3+f7t+XOoHpx/XhJOBYlPaUmoFkWKr0Y8BWWh
+1nJxyFKrSNbwUgam8ypZzwzbI1vDiX8Ol6NpEeOLwzFNT0pyTdC9UN93M1VIyKWC
+1vaeMogUREKT6SmDjRn3fISktZ0IGVf2AnFMhtgZ46TJO4BZgDdZAjTkZc/lP0yF
+Nl6MpGwnaymmL50ckT77OdlfIcXFwvNPFwWlFPlcyrkCDQRXimOdARAA4otssvZm
+sKg+g0bVyJHhn/YOHLYMih+Xf07xJHyalH0UCGnGdHZwl0B97G950SwQ7yVXtGa9
+CAPe97clE6dPD6jaumQ13BHavXM+ThgjCe8V56ayYcdzqFkxlCx0Uocoa63G0/cE
+TiOqeqhNZs8JY+D7l83jCa4lU/1pLusbkCpCQ7d5/FFLz7QSihzJWp+UTsjbNik5
+spaseEMGFRKUcB3SZ/l1dTgc0wBQ1hlvLX+h4/sG0iUs1pVpo5ORC+bUfWRokl96
+uj5QZz5rY21FaNSP1rB1HKHNkwhxifBCHQMhYGTXvD7GH+JNyF2TdRmo7eBCfAPJ
+aP3mX9t2SkCipdSsUs+Uuyib9MLA71ApW90AGiRm6HtOCxR0c3+qQRNIdFVm8mnM
+hCxXRexf6Z2wZdXXy6uY0LVRgI0o31NPJPk8l2Hnb/kHGxjyUFzEWh65J/eA368d
+4m8uF+Rr7WWlpQjwgWHU12kGThEVFFBFh2gmeIjYZdDDVhCi2mQ6lGSV2Pt7pZYL
+/PPChWLBqrVBkIUQ0GV22nRYvGdaIv2LVPu8PggbPs/wwh35nJ3rUQyJF55CFV5y
+WIWAWXfRYTKG9jkt+ncjZLEBxDO26zzO/MjIVPZxGyYryXEOgr6xp38xbyX9FpjL
+KBaIueLWEyphVjBb1uUpDGx+UDYe9vbJjPUAEQEAAYkCJQQYAQIADwUCV4pjnQIb
+DAUJA8JnAAAKCRBoc5PuN9Eo+D8dEACa60Q3ta6BWyHG0SOgfYGHE15LodACVHNI
+N6Ou+JtmLarMW/AvPclNC25mxZV0ywLbun4CnJ9qYbt/Kx7djn48mrNa0rKN8Q+V
+K5RvQA1kD890yzwu5jH6r5BQ8VBcfsPvsvatgbquzFn+NNiH9U4xRf/9BSY2Zk3G
+yA15xG0T9zoklOMg8MWbeRaJPkDELyaHPWerbO7rebynePENSFPz3o3g+K9WcCM2
+xkEL571SmT4z3Mp/p0pwemWBCP2WoKCnSjAGiiHpCFru3SlZhRIvNJyK5jeS/IU6
+d5qeTBse6TXzp6Q4xkzACIN66P5SG/YY3/ONbfs6wB3lIkvVC9n7jEXjMK1T0fK8
+9DBDjzvAkJcKLLuIljjkMhRWSCED74sn+MlaWm0xMeo276EnaVILNcrHecSr8+eX
+pVXSWEJ1+ErzZladJC+CrqUm0QljPV8Smtmk9MvOLHZ4qL4bI4Hu7MywuGNrLSol
+qO0pAT1AjaYTRuH2MhZ6mJe/EtSl0EHXEkcDteE4jbYj3lwVhA1c/So0CdayImmD
+/0tdqUfekw4va8PpbQ0wroL0XUvf3wl6HOhFhahWSqqb1fVr2slVttkaMb8M4MPt
+Ka2m4qiiuGYivPIAVapSEA4DYc+krVqVXV/yDd3T7XcNtnClVo+rmOn5WiGq24am
+79+hF4bWyw==
+=WW1Z
+-----END PGP PUBLIC KEY BLOCK-----
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+
+mQGiBEab+moRBACDH5yKqS3wcc5bdxY7PBNuwKvF5TKMfagmSvuRDtZjjIIWaA/n
+Z1KboV9Gq5g7kP7+Kfu+Qgd8u65eVsWwmPW10fXvj3aCU53glx2EdGdrHcgiyH2g
+EQfPiyBw+trIppWFRV0IDXSLMA1FNC92t2nSG/VFHaPTVwcgkIRSfcXDvwCglGdE
+a6f4uLqoNHP+m4yYnzapFuMD/R4+2AJDAvEWKDdYCGZzlawjAmmWyXrmT7/C/mx9
+8qUR473l4buXjHgDkkXXlHqdzil1vK85PhrKzNJDCCmlHUJNz+QwiAMOLwpD+kwV
+Pb57RG7y+a5JQ5+jtVw4RlUxZIk/wj2An9YBO3A5vR7PdjM32ZJCN2+aM4dYfNzQ
+xQKTA/47icvBaBVTl9rztjg2pd2Aqpc1P/GsIYLGj7XjnnJvGAENBHSH1QjpZMJG
+CTS9oJ+B0/wrIr+pA+MdFgYAb6ojMQJOO6UChjWWSGjMFcs/CeXhxlLBido3DtAE
+TbNTwO6OEfAvdosvTdhJFnwvZlJ+zZGGy5CrF2Fd9PUe9tmASbQuQWxhbiBDb29w
+ZXJzbWl0aCA8YWxhbi5jb29wZXJzbWl0aEBvcmFjbGUuY29tPoiCBBMRCgBCAhsD
+BgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgAIZARYhBEoZPAbTXnxnD6TvC6L7nggf
+LRMOBQJkQs2eBQknDNS0AAoJEKL7nggfLRMO1esAnR4FVD60BpDY/bJp5RC1VXhO
+Vlo4AKCJgsQeVeGLxDlMuhAmbcCkOjafqbQoQWxhbiBDb29wZXJzbWl0aCA8YWxh
+bmNAZnJlZWRlc2t0b3Aub3JnPoh/BBMRCgA/AhsDBgsJCAcDAgYVCAIJCgsEFgID
+AQIeAQIXgBYhBEoZPAbTXnxnD6TvC6L7nggfLRMOBQJkQs2eBQknDNS0AAoJEKL7
+nggfLRMOUgAAoIyNXuU5kOotpZrKvPrjQATbsPClAJ9wkh/n3IAobV6aDLb1qOt5
+RT3Pp7QrQWxhbiBDb29wZXJzbWl0aCA8YWxhbi5jb29wZXJzbWl0aEBzdW4uY29t
+Poh9BBMRCgA9AhsDBgsJCAcDAgQVAggDBBYCAwECHgECF4AWIQRKGTwG0158Zw+k
+7wui+54IHy0TDgUCZELNngUJJwzUtAAKCRCi+54IHy0TDunHAKCKZyiF/IwgoqYQ
+r4SjpuMD64swlgCfdkI2v8kaaFwSrEp3AeYvcT2psPS5Ag0EUXnVIQEQAKHpjOmY
+056n0tsZoW9q5egsMcl5tKC8uimrhO05nnq+5/60/YedC++V9c9b/3/X7O28LyBk
+AtBgD0xJZSDQ0DhTzKAp6AzjQtBvI68uinGwxSjT+oQpPMxqhA1I0kzoEDCdEqV+
+HsVOAEdbAi/tP9bbdTDzwVc8MWDriamBUqc53Rb00Mffy9435UgTS4gAhMwANhy6
+XZmOMBhITOzxFJUEDTDJtLbE0b1jPRQS7NHQgak1inmuvPMc3wAuoEcSCSt1xupb
+YsBoXOjK5wC/eE1LIdZoRyW2OkT140DqDZ8zfRID860hnirnYgb09TPNtj93pudU
+AUt6T9+tcLN4/rxhxHOwse66KGHO4bQ1rZ6mfco6SYd9V60cL6hC2eMecyxZliMu
+17lj7EX8lxUH+omIgHc7HGoyUR6V+WB60cxWj5v05zdeLeZ2aLBcPFhxlfDESm8f
+4ezdJSDS1QZmC0P5h3RJfhhfmdBr8kHzr7111D1/O71Av1VV5FyJ9YxUSxp4IPuz
+K7JbbgVHcA6PvXrDzWUslmZgPADpKH4hTmG/NdCqhEXcufvY6s5yNksB8X3ReNvu
+SSyfGnRz3kvtyK0XzC7KRX2PquLI6A8KJprHwZGqEB1NDG8b2iaYnghOjyfIYEVQ
+F3nGfaBwv4lrCPEoZSUaK8f/NQZjNU8NQyTnABEBAAGJAoUEGBEKACYCGwIWIQRK
+GTwG0158Zw+k7wui+54IHy0TDgUCZELNigUJHC756QIpwV0gBBkBCgAGBQJRedUh
+AAoJEM/fFIgoxkKn1uYP/3TnSR7dt3HTMs+VQA2hO7vi9BPnl/RrRxeCeuPXl7Lo
+ufRLWx4dp13RoXeUqkPBQ4vRghCmkOEt9Z/2ASxAQpI+0dCDIpFtqO/PycQ6LwfP
+mi+tOoYJmnzVcdWiLISIJa5fF5bt7I3gLxuPFumI3S9pn6yAx4yFcDmbDIoWzib0
+k+kN7qR3Z5ow7TORwkJiOTRFw1mI+iGboSZ56w1sDbMmFeCj6FAcQHKiMgnmW1bL
+Lllpxz11/FWr1avfL6Vcp3xsxmKSWw49fQv7rTIyhiOXXL21WrnAO1/6JLca8Uuk
+gQ4biGgBPeGR5W8an8h3xEyPy2tmfwCwwLWHYSUdoEpEx8ikltUKmJbpUnwf/5Z5
+2uOJpk7kTjWwzYG9BRkr7bATGfNIhw6j4FH0utJMQnxa7Ehn0xVPl+EAM6U7VCRH
+K/ieB1Cw3jH8DoIIVX5isRV1n7nfMQ2gl7ssC07VYDAmaeD+zAMriZWsbBU7vnNV
+ZcveIK5OFDLQCyzKgl0qLAOPdIC00nYNx0sMbdkPyyXcuNu2SDL4xlyK7p7PFPzp
+i/XVYFrt2yR7C+vEpMfE7Ot6kzfUFzPzwfvFcDmguzyfoOreNA5XjpnbEvHDd9cn
++SgfK7C+Lp+LtcqzCX6Z0m5TzLucCYt5I6APg3eWRasfZ04uMAY31WBgj7JXqufS
+CRCi+54IHy0TDiRsAJ97uVS+ZHPJ6P8o4rac77WTtYgYZgCfdka/GHv4Mk1MfL/M
+FS41nSdZZvy5Ag0ERpv6bxAIAJp5aUlho5rUhpS6ik7spsAQFPRuycPKMNu0J4F0
+v/OoPz085soV8ytLj4HqCGk2Zamh1jSgliZwuk9m7V7Wgxx+nBJawpWDX/eKLObE
+rfDwQ4dfOFvjbXLQMmNnQNaUGIWLPP3l8GuBOHMq60Bu+TPgh627vUntL5REQEQq
+TXIzWC6U10QsDblLwIvdOVSdGF5xl/N1myXzSKvrsZwWtoFc8G9v9hcCjhtN1sm9
+b7Ojc51iZXvcetcvPy5RA6AUW3yEExaedUdLnvIF9sjFYIfJWFVYh2AgavnGre6f
+F+NV2v3zfx3wRT7H9//m4YIDYJmgZgyQccXegTwfGBIq3osAAwYH/1FiMUMMES5I
+lz2nDqId+DCWECAU6wgvIFRcXrZWxDxB+ZrnmTCXoAD0xedpfOkRHp8XTVc/9MU+
+wQ+lZRx2OQ6MJW0XGuFvHm94KZF/8HzWA2Ah7U4n0+3sLpk6zWceZq2zZNF0yVTj
+wD98+xNK1Q9sP8aOKdtg8yMH3hisKR6rdW+mfX5q0Q8Gol2hZsFH/qyIhnPzhXDk
+nuOh8E5iMkzrejVXUEn++Yzj23XjP59SObLznVkyxI+kBI9qvVEPfFBDybjHWqLc
+gRcCpXAzjizEi+/d31iDa2ErJHV4R42obecFqiPnoDtiX3IiP7z9fmxM4aWPZZRq
+vq+1ht5wkn+ISQQYEQIACQUCRpv6bwIbDAAKCRCi+54IHy0TDoLoAKCHYRpw/Xfy
+Eunw1YL/uMZzl78qIQCdFVcXNbqD83qVhW4Ly7hyDL8o0aI=
+=awvS
+-----END PGP PUBLIC KEY BLOCK-----
--- libxtst-1.2.5.orig/debian/watch
+++ libxtst-1.2.5/debian/watch
@@ -0,0 +1,4 @@
+#git=git://anongit.freedesktop.org/xorg/lib/libXtst
+version=3
+opts=pgpsigurlmangle=s/$/.sig/ \
+https://xorg.freedesktop.org/releases/individual/lib/ libXtst-(.*)\.tar\.gz
