--- libxi-1.8.2.orig/.gitlab-ci.yml
+++ libxi-1.8.2/.gitlab-ci.yml
@@ -0,0 +1,105 @@
+# vim: set expandtab shiftwidth=2 tabstop=8 textwidth=0 filetype=yaml:
+
+.templates_sha: &template_sha 185ede0e9b9b1924b92306ab8b882a6294e92613 # see https://docs.gitlab.com/ee/ci/yaml/#includefile
+
+include:
+  - project: 'freedesktop/ci-templates'
+    ref: *template_sha
+    file: '/templates/debian.yml'
+  - project: 'freedesktop/ci-templates'
+    ref: *template_sha
+    file: '/templates/ci-fairy.yml'
+  - template: Security/SAST.gitlab-ci.yml
+
+stages:
+  - prep
+  - install xorgproto
+  - build
+  - test
+  - deploy
+
+variables:
+  FDO_DISTRIBUTION_TAG: '2023-10-13.0'  # change this to rebuild the images
+  FDO_UPSTREAM_REPO: 'xorg/lib/libxi'
+  FDO_DISTRIBUTION_VERSION: 'stable'
+  FDO_DISTRIBUTION_PACKAGES: 'git ca-certificates meson g++ make xutils-dev autoconf automake libtool pkg-config xmlto xorg-sgml-doctools w3m xsltproc libx11-dev libxext-dev libxfixes-dev'
+
+  MESON_BUILDDIR: 'build dir'
+  NINJA_ARGS: ''
+  MESON_ARGS: ''
+
+#
+# Verify that commit messages are as expected
+#
+check-commits:
+  extends:
+    - .fdo.ci-fairy
+  stage: prep
+  script:
+    - ci-fairy check-commits --junit-xml=results.xml
+  except:
+    - master@xorg/proto/xcbproto
+  variables:
+    GIT_DEPTH: 100
+  artifacts:
+    reports:
+      junit: results.xml
+  allow_failure: true
+
+#
+# Verify that the merge request has the allow-collaboration checkbox ticked
+#
+check-merge-request:
+  extends:
+    - .fdo.ci-fairy
+  stage: deploy
+  script:
+    - ci-fairy check-merge-request --require-allow-collaboration --junit-xml=results.xml
+  artifacts:
+    when: on_failure
+    reports:
+      junit: results.xml
+  allow_failure: true
+
+
+#
+# Build a container with the given tag and the packages pre-installed.
+# This only happens if/when the tag changes, otherwise the existing image is
+# re-used.
+#
+container-prep:
+  extends:
+    - .fdo.container-build@debian
+  stage: prep
+  variables:
+    GIT_STRATEGY: none
+
+xorgproto-build:
+  extends:
+    - .fdo.distribution-image@debian
+  stage: install xorgproto
+  script:
+    - export INSTDIR="$PWD/_inst"
+    - git clone --depth=1 https://gitlab.freedesktop.org/xorg/proto/xorgproto --branch=xorgproto-2021.4.99.2
+    - pushd xorgproto > /dev/null
+    - meson -Dprefix="$INSTDIR" _build
+    - ninja -j${FDO_CI_CONCURRENT:-4} -C _build install
+    - popd > /dev/null
+  variables:
+  artifacts:
+    paths:
+      - _inst
+
+build:
+  extends:
+    - .fdo.distribution-image@debian
+  stage: build
+  script:
+    - export INSTDIR="$PWD/_inst"
+    - export PKG_CONFIG_PATH=$(find $INSTDIR/ -name '*.pc' -printf "%h:")
+    - autoreconf -fi
+    - mkdir build
+    - pushd build > /dev/null
+    - ../configure
+    - make all -j${FDO_CI_CONCURRENT:-4}
+    - popd > /dev/null
--- libxi-1.8.2.orig/ChangeLog
+++ libxi-1.8.2/ChangeLog
@@ -1,434 +1,3 @@
-commit 3c2d5aedd7708c75710b39268631762de904c304
-Author: Peter Hutterer <peter.hutterer@who-t.net>
-Date:   Thu Sep 5 11:15:04 2024 +1000
-
-    libXi 1.8.2
-    
-    Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
-
-commit a5af26c654b4da5fd5eaaeaa2173c734ab9e63ec
-Author: Peter Hutterer <peter.hutterer@who-t.net>
-Date:   Tue Aug 27 11:01:06 2024 +1000
-
-    XGetFeedbackControl: skip over unknown feedback controls
-    
-    If the server sends an unknown feedback control class we would allocate
-    the amount of *protocol* bytes for the client representation but then
-    later mess up the actual composition of the classes due to
-    Feedback->length never being initialized.
-    
-    So while in theory it looks like this:
-       |-------------- allocated size -----------------|
-       [led feedback] [ ?unknown? ] [kbd feedback][....]
-    with unknown being the size of the class on the protocol,
-    in practice it looks like this:
-       |-------------- allocated size -----------------|
-       [led feedback] [ ?unknown?                ] [kbd feedback][...]
-    with unknown being the size of whatever was in the Feeback->length
-    pointed to at the time. The content of unknown is never initialized.
-    
-    Fix this by making unknown classes disappear so the above becomes:
-       |------- allocated size ---------|
-       [led feedback][kbd feedback][....]
-    
-    Closes #16
-    
-    Part-of: <https://gitlab.freedesktop.org/xorg/lib/libxi/-/merge_requests/18>
-
-commit dabce4dbe5eab35076e31fd0a628cbe11b858fa1
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Mon Aug 26 11:49:49 2024 -0700
-
-    wireTo*Event: check for malloc() failure
-    
-    Fixes 6 -Wanalyzer-possible-null-dereference and
-    1 -Wanalyzer-null-dereference and warnings from gcc 14.1
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-    Part-of: <https://gitlab.freedesktop.org/xorg/lib/libxi/-/merge_requests/17>
-
-commit 56b11459f833df8f324587847534a548b070da94
-Author: kyomawolf <kyomawolf@kyomawolf.eu>
-Date:   Thu Oct 5 21:52:36 2023 +0200
-
-    fixed wrong type in man page
-    
-    Part-of: <https://gitlab.freedesktop.org/xorg/lib/libxi/-/merge_requests/14>
-
-commit 57ce2e8ba4c1f1bc5792e14bc7082f3ebd0915be
-Author: Peter Hutterer <peter.hutterer@who-t.net>
-Date:   Mon Jan 29 11:27:11 2024 +1000
-
-    Allow XIFreeDeviceInfo on a null pointer
-
-commit a340bc0424a73d9fc4badbb7aee5284877635ca9
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Fri Oct 13 11:17:43 2023 -0700
-
-    gitlab CI: Update to latest CI templates
-    
-    Current template fails to rebuild the CI container with the current
-    Debian stable release (a new release since the last update)
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit 09f3eb570fe79bfc0c430b6059d7b4acaf371c24
-Author: Matthieu Herrb <matthieu@herrb.eu>
-Date:   Sat Aug 5 10:32:03 2023 +0000
-
-    Fix double alignment on 32bits machines with strict constraints
-
-commit 3a7503ec7703f10de17c622ea22b7bff736cea74
-Author: Peter Hutterer <peter.hutterer@who-t.net>
-Date:   Thu May 4 14:42:12 2023 +1000
-
-    libXi 1.8.1
-    
-    Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
-
-commit 0f1541e1e5bd57686edf95c3532544556ed370bb
-Author: Peter Hutterer <peter.hutterer@who-t.net>
-Date:   Fri Apr 28 11:12:47 2023 +1000
-
-    Initialize a few stack vars to zero
-    
-    Modified version of the patch from #4
-    
-    Fixes #4
-    
-    Reviewed-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit 826215af0cc46b19555063b8894de6781d4c5993
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Sat Mar 4 10:44:45 2023 -0800
-
-    configure: Use LT_INIT from libtool 2 instead of deprecated AC_PROG_LIBTOOL
-    
-    AC_PROG_LIBTOOL was replaced by LT_INIT in libtool 2 in 2008,
-    so it's time to rely on it.
-    
-    Clears autoconf warnings:
-    
-    configure.ac:13: warning: The macro `AC_PROG_LIBTOOL' is obsolete.
-    configure.ac:13: You should run autoupdate.
-    aclocal.m4:3465: AC_PROG_LIBTOOL is expanded from...
-    configure.ac:13: the top level
-    
-    libtoolize: Consider adding 'AC_CONFIG_MACRO_DIRS([m4])' to configure.ac,
-    libtoolize: and rerunning libtoolize and aclocal.
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit 08431d0684f9a1edf199f6c6060d2bef1ac78399
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Thu Aug 25 15:39:36 2022 -0700
-
-    XInput_find_display: Don't dereference NULL dpyinfo
-    
-    Unlikely, but could happen if calloc fails in XextAddDisplay()
-    
-    Relies on XextHasExtension(i) macro from <X11/extensions/extutil.h>
-    which checks for ((i) && ((i)->codes))
-    
-    Fixes: #15
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit bca3474a8622fde5815260461784282f78a4efb5
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Sat Jul 23 15:29:23 2022 -0700
-
-    gitlab CI: enable gitlab's builtin static analysis
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit 87202ce9851c5fb61038829abd5b1feb42334a35
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Sat Jul 23 15:28:42 2022 -0700
-
-    gitlab CI: enable commit & merge request checks
-    
-    Uses ci-fairy from freedesktop/ci-templates
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit 0fb16bad9c851c11d171ff601bf8adae9ffc6fdb
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Sun Jul 17 13:39:34 2022 -0700
-
-    Fix spelling/wording issues
-    
-    Found by using:
-        codespell --builtin clear,rare,informal,code,names
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit ed12a34ed79d3751fc35194d93b7ad370a1a8b8a
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Sun Jul 17 13:34:52 2022 -0700
-
-    Build xz tarballs instead of bzip2
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit f24d7f43ab4d97203e60677a3d42e11dbc80c8b4
-Author: Povilas Kanapickas <povilas@radix.lt>
-Date:   Wed Sep 15 19:28:03 2021 +0300
-
-    libXi 1.8
-    
-    Signed-off-by: Povilas Kanapickas <povilas@radix.lt>
-
-commit dd9a260b29c50be90e92048b0e067456ff18675e
-Author: Povilas Kanapickas <povilas@radix.lt>
-Date:   Wed Sep 15 19:27:50 2021 +0300
-
-    Bump the requirement of inputproto to 2.3.99.1
-
-commit a761ae5119b1460068d6f62219c4e2705ef20814
-Author: Povilas Kanapickas <povilas@radix.lt>
-Date:   Sat Sep 4 00:03:50 2021 +0300
-
-    man: Document XIAllowEvents
-
-commit 42ee1f0b0660c5d6aaefbb55ef76ea97ad57d4f6
-Author: Povilas Kanapickas <povilas@radix.lt>
-Date:   Tue Jun 1 14:43:39 2021 +0300
-
-    libXi 1.7.99.2
-
-commit a7343d53509e66d92e91238e9f4ecc2bf154e045
-Author: Povilas Kanapickas <povilas@radix.lt>
-Date:   Sun May 30 20:47:38 2021 +0300
-
-    gitlab-ci: Setup gitlab build
-
-commit dd3df0b61499fa231a5a82e14d17bf37ce917d1d
-Author: Povilas Kanapickas <povilas@radix.lt>
-Date:   Sun May 30 19:56:18 2021 +0300
-
-    Fix compile error that slipped through
-    
-    Lesson for the future: make sure the private environment (e.g
-    convenience scripts) don't add changes where they are not expected. In
-    my case I'm building and installing Debian packages straigth away. And
-    one of the extra commits for Debian support had a change that had to go
-    to the libXi repository.
-    
-    The development snapshot 1.7.99.1 has thus not been released (only a
-    commit with the version bump was pushed).
-    
-    Signed-off-by: Povilas Kanapickas <povilas@radix.lt>
-
-commit f036a6399a0bb1808dc82f7a91a1ae5a7e57691e
-Author: Povilas Kanapickas <povilas@radix.lt>
-Date:   Sun May 30 19:38:07 2021 +0300
-
-    libXi 1.7.99.1
-
-commit 4b2f0f0c0674915645881d9b73e9b8d4cf1a315f
-Author: Povilas Kanapickas <povilas@radix.lt>
-Date:   Sat Oct 10 02:53:42 2020 +0300
-
-    Update the list of supported XI versions
-
-commit ed1788b8c3a78f6dc523487e46e202a5f667407c
-Author: Povilas Kanapickas <povilas@radix.lt>
-Date:   Sat Oct 10 02:53:41 2020 +0300
-
-    Add support for XI 2.4 gesture events
-
-commit 8b4ea802fa3ba769dc8335f0e72c6f0cb7c9c35e
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Sun Oct 4 13:48:06 2020 -0700
-
-    man pages: fix more typos
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit 6affddac55f03bff63b1e37e475d8e0d0bb84e85
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Sun Oct 4 13:21:24 2020 -0700
-
-    man pages: Add more missing spaces.
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit cff134328cb31df5e366729aca6c4131a1e3304f
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Sun Oct 4 13:15:13 2020 -0700
-
-    man/XGrabDeviceButton.txt: Fix bad .P -> paragraph break conversion
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit 0944a10d6e22b02f229b4bbaab60aac170ef1bba
-Author: Samanta Navarro <ferivoz@riseup.net>
-Date:   Sun Oct 4 18:55:06 2020 +0000
-
-    Fix typos in manual pages
-
-commit e05da79a98775edb91ffe3de3b413f156bc4d6e7
-Author: Matt Turner <mattst88@gmail.com>
-Date:   Wed Jun 19 12:42:33 2019 -0400
-
-    libXi 1.7.10
-    
-    Signed-off-by: Matt Turner <mattst88@gmail.com>
-
-commit dd5cedb7cd631cf4aeaadfd9cb993f0086857dbd
-Author: Matt Turner <mattst88@gmail.com>
-Date:   Wed Jun 19 12:40:46 2019 -0400
-
-    Replace open-coded FP3232_TO_DOUBLE
-    
-    Signed-off-by: Matt Turner <mattst88@gmail.com>
-
-commit a42ee93c10bf9a63c22e0642d6e51877fba8dae5
-Author: Alexander Bersenev <bay@hackerdom.ru>
-Date:   Mon Jun 17 22:56:43 2019 +0500
-
-    Fix the FIXME in XIValuatorClass case of copy_classes function in XExtInt.c
-    
-    In addition fix FP3232_TO_DOUBLE macro to correctly compute the fractional
-    part.
-    
-    This fixes glitchy scrolling in Qt applications when the application was
-    just activated or was scrolled in the backgroud. Qt uses XIQueryDevice
-    call to synchronize internal scroll location with an actual one.
-    
-    Bug: https://gitlab.freedesktop.org/xorg/lib/libxi/issues/10
-
-commit 3e38d823a162c21492ac243924325730d4ea61bf
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Fri Dec 7 19:43:13 2018 -0800
-
-    Update configure.ac bug URL for gitlab migration
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit e020870b1e322e3458c1c18a456661f1951cab70
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Mon Nov 19 22:23:32 2018 -0800
-
-    Update README for gitlab migration
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit e7654506717f7ff08eae7ab4c0db3fc1c2d5b7ed
-Author: Raphaël Droz <raphael.droz+floss@gmail.com>
-Date:   Sat Oct 27 22:17:31 2018 +0000
-
-    Update XIChangeHierarchy.txt
-
-commit 60ef7bab67aca3dd7c83e8726feb3c46869a74c2
-Author: Jeff Smith <whydoubt@gmail.com>
-Date:   Thu Feb 1 01:59:56 2018 -0600
-
-    _XIPassiveGrabDevice needs to set time value
-    
-    When setting up a XIPassiveGrabDevice request, the time field is not
-    being set, leading to improper data being passed 'over the wire'.
-    
-    Accept a time value into _XIPassiveGrabDevice and use it to set the time
-    field in the request.  Since the the functions calling
-    _XIPassiveGrabDevice are part of the API, and they do not accept time
-    values, they can just pass CurrentTime.
-    
-    Signed-off-by: Jeff Smith <whydoubt@gmail.com>
-    Reviewed-by: Peter Hutterer <peter.hutterer@who-t.net>
-    Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
-
-commit a1c639255b7a9270fea52de3f186eb2fca9de01a
-Author: Peter Hutterer <peter.hutterer@who-t.net>
-Date:   Wed Jun 14 11:15:01 2017 +1000
-
-    man: add a bunch of missing spaces
-    
-    Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
-
-commit 8ffaac5710a9a983227454d8473604daae3f172a
-Author: Mihail Konev <k.mvc@ya.ru>
-Date:   Thu Jan 26 13:52:49 2017 +1000
-
-    autogen: add default patch prefix
-    
-    Signed-off-by: Mihail Konev <k.mvc@ya.ru>
-
-commit c6c386431e1fa6b45ff966191e68bee1150c5e87
-Author: Emil Velikov <emil.l.velikov@gmail.com>
-Date:   Mon Mar 9 12:00:52 2015 +0000
-
-    autogen.sh: use quoted string variables
-    
-    Place quotes around the $srcdir, $ORIGDIR and $0 variables to prevent
-    fall-outs, when they contain space.
-    
-    Signed-off-by: Emil Velikov <emil.l.velikov@gmail.com>
-    Reviewed-by: Peter Hutterer <peter.hutterer@who-t.net>
-    Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
-
-commit c7584066d5b06cd2bb9d99df43b741cf565820ea
-Author: Peter Hutterer <peter.hutterer@who-t.net>
-Date:   Tue Jan 24 10:32:07 2017 +1000
-
-    autogen.sh: use exec instead of waiting for configure to finish
-    
-    Syncs the invocation of configure with the one from the server.
-    
-    Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
-    Reviewed-by: Emil Velikov <emil.velikov@collabora.com>
-
-commit b87c312590877800b4e18a17a012b47839f5c06b
-Author: Peter Hutterer <peter.hutterer@who-t.net>
-Date:   Mon Jan 23 13:44:58 2017 +1000
-
-    libXi 1.7.9
-    
-    Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
-
-commit 557b60798a9da49386f1034b133838332735de22
-Author: Emilio Pozuelo Monfort <pochu@debian.org>
-Date:   Tue Dec 27 17:24:10 2016 +0100
-
-    Fix possible free of uninitialized pointer
-    
-    If the _XReply() call fails, we'll try to free an uninitialized
-    pointer.
-    
-    https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=849026
-    
-    Reported-by: Thomas Walker <thwalker3@gmail.com>
-    Signed-off-by: Emilio Pozuelo Monfort <pochu@debian.org>
-    Reviewed-by: Julien Cristau <jcristau@debian.org>
-    Tested-by: Thomas Walker <thwalker3@gmail.com>
-    Signed-off-by: Julien Cristau <jcristau@debian.org>
-
-commit 4c5c8d6246debc06a56120cff71bfdf1877884d0
-Author: Emilio Pozuelo Monfort <pochu@debian.org>
-Date:   Tue Oct 25 21:31:19 2016 +0200
-
-    Check that allocating a buffer succeeded
-    
-    Since we are going to write into the buffer, we should make sure the
-    allocation didn't fail.
-    
-    Reported-by: Julien Cristau <jcristau@debian.org>
-    Signed-off-by: Emilio Pozuelo Monfort <pochu@debian.org>
-    Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
-
-commit 7ac03c6c1907a39b5b42b17ad331295b8c85154d
-Author: Emilio Pozuelo Monfort <pochu@debian.org>
-Date:   Tue Oct 25 21:31:18 2016 +0200
-
-    Plug a memory leak
-    
-    Introduced in commit 19a9cd6.
-    
-    Reported-by: Julien Cristau <jcristau@debian.org>
-    Signed-off-by: Emilio Pozuelo Monfort <pochu@debian.org>
-    Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
-
 commit 1bdeb431c3cc9eec7e12fdd29a83237f2f228865
 Author: Peter Hutterer <peter.hutterer@who-t.net>
 Date:   Tue Oct 25 12:43:44 2016 +1000
@@ -1144,7 +713,7 @@
     Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
 
 commit af9f26510d87eee71f1cd688d7dcfbf173c13943
-Merge: 31c6cf9f6fbc 9e8a55dfcb3d
+Merge: 31c6cf9 9e8a55d
 Author: Peter Hutterer <peter.hutterer@who-t.net>
 Date:   Mon Dec 17 14:34:53 2012 +1000
 
@@ -1462,14 +1031,14 @@
     Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
 
 commit 47f3d887a7731f4ab68a69ed31dbe7acf6d13270
-Merge: 82a631263ef4 e73e2fe95dab
+Merge: 82a6312 e73e2fe
 Author: Peter Hutterer <peter.hutterer@who-t.net>
 Date:   Wed Dec 21 15:40:37 2011 +1000
 
     Merge branch 'multitouch'
     
     Conflicts:
-            configure.ac
+    	configure.ac
 
 commit 82a631263ef4e6f46c1f33748089db8bf603b095
 Author: Peter Hutterer <peter.hutterer@who-t.net>
@@ -1667,14 +1236,14 @@
     Reviewed-by: Gaetan Nadon <memsize@videotron.ca>
 
 commit 69f2580fd47cfe24105892e9ffb79cd046954b7d
-Merge: fc5d47895c13 3ed1b4632052
+Merge: fc5d478 3ed1b46
 Author: Peter Hutterer <peter.hutterer@who-t.net>
 Date:   Sat Sep 17 06:41:30 2011 +1000
 
     Merge branch 'smooth-scrolling'
     
     Conflicts:
-            src/XExtInt.c
+    	src/XExtInt.c
 
 commit 3ed1b463205295a5ebba7d570a4cb8dfade51a38
 Author: Peter Hutterer <peter.hutterer@who-t.net>
@@ -3483,14 +3052,14 @@
     Fix building if asciidoc or xmlto is not installed.
 
 commit 5975cb9e6f89bf71b1740cee951fc583d7b7cfc1
-Merge: 8ea9ebcedcb9 279a8c3a6466
+Merge: 8ea9ebc 279a8c3
 Author: Peter Hutterer <peter.hutterer@who-t.net>
 Date:   Thu Jun 4 15:49:07 2009 +1000
 
     Merge branch 'master' into xi2
     
     Conflicts:
-            man/XI.xml
+    	man/XI.xml
 
 commit 8ea9ebcedcb99480a5774c7755329a2ca8fda07b
 Author: Benjamin Close <Benjamin.Close@clearchain.com>
@@ -4300,7 +3869,7 @@
     Ansify the XI 2 functions.
 
 commit 50990b8deb8b8484e9e02e313387ea4d2ddbd242
-Merge: 26dde5aaf4bb 145947ed6032
+Merge: 26dde5a 145947e
 Author: Peter Hutterer <peter@cs.unisa.edu.au>
 Date:   Thu May 22 12:48:40 2008 +0930
 
@@ -4328,16 +3897,16 @@
     
     src/XExtInt.c:xinput_extension_hooks field event_to_wire of type
       XExtensionHooks that has the prototype     Status (*event_to_wire)(
-                  Display*                  /* display */,
-                  XEvent*                   /* re */,
-                  xEvent*                   /* event */
+    	      Display*			/* display */,
+                  XEvent*			/* re */,
+                  xEvent*			/* event */
       );
       does not match Status
       _XiEventToWire(
-        register Display     *dpy,          /* pointer to display structure */
-        register XEvent      *re,           /* pointer to client event */
-        register xEvent     **event,        /* wire protocol event */
-        register int         *count);
+        register Display	 *dpy,		/* pointer to display structure */
+        register XEvent	 *re,		/* pointer to client event */
+        register xEvent	**event,	/* wire protocol event */
+        register int	 *count);
     
     Signed-off-by: Peter Hutterer <peter@cs.unisa.edu.au>
 
@@ -4348,7 +3917,7 @@
     Fix up AM_CFLAGS, pulled in the system-includes before the local ones.
 
 commit d68d321c0692848dd5eb67a3713676498c1c646b
-Merge: 7d0977bc02ce 3e99cc281509
+Merge: 7d0977b 3e99cc2
 Author: Peter Hutterer <peter@cs.unisa.edu.au>
 Date:   Mon Apr 28 14:35:11 2008 +0930
 
@@ -4356,9 +3925,9 @@
     
     Conflicts:
     
-            .gitignore
-            man/XGrabDeviceKey.man
-            man/XListInputDevices.man
+    	.gitignore
+    	man/XGrabDeviceKey.man
+    	man/XListInputDevices.man
 
 commit 7d0977bc02ce4f29c0ed335fcdcce4ed7c328259
 Author: Peter Hutterer <peter@cs.unisa.edu.au>
@@ -4888,14 +4457,14 @@
     Add *.o to .gitignore.
 
 commit 6e08a76fd8e356f98d2d6913f0c1b1401090768d
-Merge: ba84e84575b4 360ceacb2a3c
+Merge: ba84e84 360ceac
 Author: Daniel Stone <daniel@fooishbar.org>
 Date:   Sun Sep 24 20:02:24 2006 +0300
 
     Merge branch 'master' of git+ssh://git.freedesktop.org/git/xorg/lib/libXi
 
 commit ba84e84575b4167b0c6298e86a860b0741e2d2a3
-Merge: 93d3c2c45ccd 576c5794cf4b
+Merge: 93d3c2c 576c579
 Author: Daniel Stone <daniel@fooishbar.org>
 Date:   Sun Sep 24 19:58:56 2006 +0300
 
--- libxi-1.8.2.orig/autogen.sh
+++ libxi-1.8.2/autogen.sh
@@ -0,0 +1,17 @@
+#! /bin/sh
+
+srcdir=`dirname "$0"`
+test -z "$srcdir" && srcdir=.
+
+ORIGDIR=`pwd`
+cd "$srcdir"
+
+autoreconf -v --install || exit 1
+cd "$ORIGDIR" || exit $?
+
+git config --local --get format.subjectPrefix >/dev/null 2>&1 ||
+    git config --local format.subjectPrefix "PATCH libXi"
+
+if test -z "$NOCONFIGURE"; then
+    exec "$srcdir"/configure "$@"
+fi
--- libxi-1.8.2.orig/debian/README.source
+++ libxi-1.8.2/debian/README.source
@@ -0,0 +1,49 @@
+------------------------------------------------------
+Quick Guide To Patching This Package For The Impatient
+------------------------------------------------------
+
+1. Make sure you have quilt installed
+2. Unpack the package as usual with "dpkg-source -x"
+3. Run the "patch" target in debian/rules
+4. Create a new patch with "quilt new" (see quilt(1))
+5. Edit all the files you want to include in the patch with "quilt edit" 
+   (see quilt(1)).
+6. Write the patch with "quilt refresh" (see quilt(1))
+7. Run the "clean" target in debian/rules
+
+Alternatively, instead of using quilt directly, you can drop the patch in to 
+debian/patches and add the name of the patch to debian/patches/series.
+
+------------------------------------
+Guide To The X Strike Force Packages
+------------------------------------
+
+The X Strike Force team maintains X packages in git repositories on
+salsa.debian.org in the xorg-team group. Most upstream packages
+are actually maintained in git repositories as well, so they often
+just need to be pulled into salsa.debian.org in a "upstream-*" branch.
+Otherwise, the upstream sources are manually installed in the Debian
+git repository.
+
+The .orig.tar.gz upstream source file could be generated this
+"upstream-*" branch in the Debian git repository but it is actually
+copied from upstream tarballs directly.
+
+Due to X.org being highly modular, packaging all X.org applications
+as their own independent packages would have created too many Debian
+packages. For this reason, some X.org applications have been grouped
+into larger packages: xutils, xutils-dev, x11-apps, x11-session-utils,
+x11-utils, x11-xfs-utils, x11-xkb-utils, x11-xserver-utils.
+Most packages, including the X.org server itself and all libraries
+and drivers are, however maintained independently.
+
+The Debian packaging is added by creating the "debian-*" git branch
+which contains the aforementioned "upstream-*" branch plus the debian/
+repository files.
+When a patch has to be applied to the Debian package, two solutions
+are involved:
+* If the patch is available in one of the upstream branches, it
+  may be git'cherry-picked into the Debian repository. In this
+  case, it appears directly in the .diff.gz.
+* Otherwise, the patch is added to debian/patches/ which is managed
+  with quilt as documented in /usr/share/doc/quilt/README.source.
--- libxi-1.8.2.orig/debian/changelog
+++ libxi-1.8.2/debian/changelog
@@ -0,0 +1,546 @@
+libxi (2:1.8.2-1) unstable; urgency=medium
+
+  * New upstream release.
+
+ -- Timo Aaltonen <tjaalton@debian.org>  Wed, 25 Sep 2024 10:48:59 +0300
+
+libxi (2:1.8.1-1) unstable; urgency=medium
+
+  [ Timo Aaltonen ]
+  * New upstream release.
+  * control: Drop obsolete Replaces.
+  * control: Migrate to pkgconf.
+
+  [ Debian Janitor ]
+  * Remove constraints unnecessary since buster (oldstable):
+    + Build-Depends: Drop versioned constraint on asciidoc, libx11-dev,
+      libxext-dev, libxfixes-dev, xorg-sgml-doctools and xutils-dev.
+    + libxi-dev: Drop versioned constraint on xorg-sgml-doctools in Depends.
+
+ -- Timo Aaltonen <tjaalton@debian.org>  Thu, 15 Feb 2024 11:12:02 +0200
+
+libxi (2:1.8-1) unstable; urgency=medium
+
+  [ Timo Aaltonen ]
+  * New upstream release.
+  * control: Migrate to x11proto-dev.
+  * control: Bump x11proto-dev dependencies.
+  * Update README.source for salsa.
+  * Add upstream metadata, drop git repo from d/watch.
+  * Update symbols.
+
+  [ Simon McVittie ]
+  * d/control: Fix Vcs-Git URL. (Closes: #990229)
+
+ -- Timo Aaltonen <tjaalton@debian.org>  Thu, 23 Sep 2021 05:01:30 +0300
+
+libxi (2:1.7.10-1) unstable; urgency=medium
+
+  * New upstream release.
+  * Migrate to debhelper-compat, bump to 12.
+  * Update signing-key.asc.
+  * control: Bump policy to 4.5.0.
+  * control: Update VCS urls.
+
+ -- Timo Aaltonen <tjaalton@debian.org>  Tue, 02 Jun 2020 14:38:30 +0300
+
+libxi (2:1.7.9-1) unstable; urgency=medium
+
+  * New upstream release.
+
+ -- Emilio Pozuelo Monfort <pochu@debian.org>  Mon, 23 Jan 2017 21:22:47 +0100
+
+libxi (2:1.7.8-2) unstable; urgency=medium
+
+  * Cherry-pick upstream commit 557b6079, don't free an uninitialized
+    buffer. Closes: #849026.
+
+ -- Emilio Pozuelo Monfort <pochu@debian.org>  Thu, 29 Dec 2016 11:22:29 +0100
+
+libxi (2:1.7.8-1) unstable; urgency=medium
+
+  [ Andreas Boll ]
+  * New upstream release.
+    - Fixes CVE-2016-7945 and CVE-2016-7946 (Closes: #840440).
+  * Update d/upstream/signing-key.asc with Matthieu Herrb's key.
+  * Update a bunch of URLs in packaging to https.
+  * Bump Standards-Version to 3.9.8, no changes needed.
+
+  [ Emilio Pozuelo Monfort ]
+  * Acknowledge 2:1.7.6-1.1 NMU.
+  * Cherry-pick upstream commit 7ac03c6c to plug a memory leak in the
+    security fix.
+  * Cherry-pick upstream commit 4c5c8d62, check a buffer was allocated
+    before writing to it.
+  * Bump debhelper compat to 10.
+    + debhelper now calls dh-autoreconf automatically.
+    + debhelper now enables --parallel by default.
+  * Switch to -dbgsym packages.
+  * debhelper passes --disable-silent-rules to configure for us.
+  * Drop explicit build target. dh $@ can handle it.
+
+ -- Emilio Pozuelo Monfort <pochu@debian.org>  Tue, 06 Dec 2016 00:50:24 +0100
+
+libxi (2:1.7.6-1.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Mark libxi-dev as Multi-Arch: same. Closes: #689068.
+  * Explicitly specify documentation's anchor ids instead of generating
+    non-reproducible ones during the build (Francois Gouget).
+
+ -- Matthias Klose <doko@debian.org>  Thu, 03 Nov 2016 14:52:54 +0100
+
+libxi (2:1.7.6-1) unstable; urgency=medium
+
+  * New upstream release.
+  * Remove Drew and Cyril from Uploaders.
+  * Fix package descriptions: input hotplugging has been implemented years ago
+    (closes: #782987).  Thanks, Paul Wise!
+
+ -- Julien Cristau <jcristau@debian.org>  Sun, 24 Jan 2016 14:15:17 +0100
+
+libxi (2:1.7.5-1) unstable; urgency=medium
+
+  [ Andreas Boll ]
+  * New upstream release.
+  * Switch to dh with the autoreconf and quilt addons.
+  * Fix lintian error: pre-depends-directly-on-multiarch-support libxi6.
+  * Let uscan verify tarball signatures.
+  * Fix Vcs-* fields.
+  * Remove obsolete XC- prefix from Package-Type field.
+  * Add upstream url.
+  * Bump standards version to 3.9.6.
+
+  [ Julien Cristau ]
+  * Delete obsolete debian/xsfbs.
+
+ -- Julien Cristau <jcristau@debian.org>  Tue, 13 Oct 2015 12:29:20 +0200
+
+libxi (2:1.7.4-1) unstable; urgency=low
+
+  * New upstream release.
+
+ -- Maarten Lankhorst <maarten.lankhorst@ubuntu.com>  Mon, 21 Jul 2014 09:57:26 +0200
+
+libxi (2:1.7.2-1) unstable; urgency=low
+
+  * New upstream release.
+
+ -- Julien Cristau <jcristau@debian.org>  Mon, 12 Aug 2013 18:46:14 +0200
+
+libxi (2:1.7.1.901-1) unstable; urgency=low
+
+  * New upstream release candidate.
+
+  [ Timo Aaltonen ]
+  * control: Bump policy to 3.9.4, no changes.
+  * control: Bump x11proto-input-dev build-dep to 2.2.99.1.
+  * libxi6.symbols: Added new symbols.
+  * rules: Bump shlibs.
+  * control: Add libfixes-dev to build-deps and libxi-dev Depends.
+
+  [ Julien Cristau ]
+  * Use dpkg-buildflags.
+  * Disable silent build rules.
+  * Bump debhelper compat level to 7.
+  * Simplify installing manpages.
+  * Fix clean rule for config.h.in.
+  * Use dh_prep instead of dh_clean -k.
+
+ -- Julien Cristau <jcristau@debian.org>  Sun, 30 Jun 2013 15:51:02 +0200
+
+libxi (2:1.6.1-1) unstable; urgency=low
+
+  [ Julien Cristau ]
+  * Don't require (fake)root for debian/rules clean.
+
+  [ Cyril Brulebois ]
+  * New upstream release.
+
+ -- Cyril Brulebois <kibi@debian.org>  Sun, 20 May 2012 17:00:03 +0200
+
+libxi (2:1.6.0-1) unstable; urgency=low
+
+  [ Chase Douglas ]
+  * New upstream release
+    - Fix bus error on MIPS N32 (closes: #636920)
+  * Bump build deps on x11proto-input-dev
+  * Bump lintian standards version to 3.9.3
+
+  [ Julien Cristau ]
+  * Pull some more changes from upstream master:
+    - Fix wrong button label and mask copy on OS X (fixes regression in the
+      patch for bug#636920)
+    - Move version comparison into a helper function
+    - Set the RawEvent sourceid (#34240)
+  * Upload to unstable.
+
+ -- Julien Cristau <jcristau@debian.org>  Mon, 16 Apr 2012 19:32:32 +0200
+
+libxi (2:1.5.99.2-1) experimental; urgency=low
+
+  * New upstream release candidate:
+    - With XI 2.2 support, needed for multitouch.
+  * Fix changelog entry for 2:1.4.99.1-1:
+    - With XI 2.1 support, no multitouch yet.
+  * Bump x11proto-input-dev and libx11-dev build-deps.
+  * Update libxi6.symbols with new symbols and bump shlibs.
+
+ -- Cyril Brulebois <kibi@debian.org>  Thu, 22 Dec 2011 14:19:35 +0100
+
+libxi (2:1.5.0-1) experimental; urgency=low
+
+  * New upstream release.
+
+ -- Cyril Brulebois <kibi@debian.org>  Wed, 21 Dec 2011 10:12:07 +0100
+
+libxi (2:1.4.99.1-1) experimental; urgency=low
+
+  * New upstream release candidate:
+    - With XI 2.1 support.
+  * Update build dependencies accordingly:
+    - Bump xutils-dev, x11proto-input-dev, xorg-sgml-doctools.
+    - Add xsltproc.
+  * Update libxi-dev.install: specs/ and docs/ have been merged.
+  * Don't install *.db yet.
+  * Add -c4 to dh_makeshlibs.
+
+ -- Cyril Brulebois <kibi@debian.org>  Thu, 15 Dec 2011 15:05:46 +0100
+
+libxi (2:1.4.3-3) unstable; urgency=low
+
+  * Team upload.
+  * Don't ship debug symbols for the udeb in libxi6-dbg.
+  * Add build-{arch,indep} debian/rules targets.
+
+ -- Julien Cristau <jcristau@debian.org>  Sun, 12 Jun 2011 00:20:23 +0200
+
+libxi (2:1.4.3-2) unstable; urgency=low
+
+  * Team upload.
+
+  [ Steve Langasek ]
+  * Build for multiarch.
+
+  [ Julien Cristau ]
+  * Remove David Nusinow from Uploaders.
+
+ -- Julien Cristau <jcristau@debian.org>  Sat, 11 Jun 2011 16:14:32 +0200
+
+libxi (2:1.4.3-1) unstable; urgency=low
+
+  * New upstream release.
+  * Add a comment to the series file so that it isn't eaten by
+    dpkg-source.
+  * Bump Standards-Version (no changes).
+
+ -- Cyril Brulebois <kibi@debian.org>  Thu, 09 Jun 2011 12:08:59 +0200
+
+libxi (2:1.4.2-1) unstable; urgency=low
+
+  [ Julien Cristau ]
+  * Bump Standards-Version to 3.9.1 (no changes).
+
+  [ Cyril Brulebois ]
+  * New upstream release.
+
+ -- Cyril Brulebois <kibi@debian.org>  Sun, 27 Mar 2011 06:36:41 +0200
+
+libxi (2:1.4.1-1) unstable; urgency=low
+
+  * New upstream release.
+  * Use xorg-sgml-doctool's copy of xorg.css.
+  * Upload to unstable.
+
+ -- Julien Cristau <jcristau@debian.org>  Tue, 08 Feb 2011 00:04:32 +0100
+
+libxi (2:1.4.0-1) experimental; urgency=low
+
+  * New upstream release.
+  * Update debian/copyright from upstream COPYING.
+  * Bump xutils-dev build-dep for new macros.
+  * Pass --with-xmlto and --without-fop to configure.
+  * Add xmlto, xorg-sgml-doctools, and w3m build-dep accordingly.
+  * Switch dh_install from --list-missing to --fail-missing for added
+    safety.
+  * Get rid of *.xml files before running dh_install.
+  * Ship *.html, *.txt, *.css in libxi-dev.
+
+ -- Cyril Brulebois <kibi@debian.org>  Fri, 14 Jan 2011 06:22:37 +0100
+
+libxi (2:1.3-6) unstable; urgency=medium
+
+  * WireToEvent: Set display member of all events as well (cherry-pick from
+    upstream).
+
+ -- Julien Cristau <jcristau@debian.org>  Fri, 17 Dec 2010 17:00:44 +0100
+
+libxi (2:1.3-5) unstable; urgency=low
+
+  * Cherry-picked fixes from upstream:
+    - man: fix types for XGetSelectedExtensionEvents
+    - Initialize extension with the right number of events (obsoletes our
+      quilt patch)
+    - Don't unlock the Display twice
+    - Fix usage of uninitialized value
+    - Fix memory leak in XIGetSelectedEvents
+    - Always unlock display correctly
+    - man: improve readability of XAllowDeviceEvents
+    - man: fix typo in XIQueryDevice doc (closes: #598964)
+    - Fix typo when converting raw events from the wire
+    - man: XGetDeviceProperty(3) has no parameter 'pending'
+
+ -- Julien Cristau <jcristau@debian.org>  Thu, 18 Nov 2010 17:31:04 +0100
+
+libxi (2:1.3-4) unstable; urgency=low
+
+  * Add udeb needed for the graphical installer: libxi6-udeb.
+  * Version/Bump some B-D to make sure the udeb gets its dependencies on
+    the (recently-added) udebs rather than on the libraries:
+     - libx11-dev
+     - libxext-dev
+  * Add myself to Uploaders.
+
+ -- Cyril Brulebois <kibi@debian.org>  Thu, 11 Mar 2010 22:49:49 +0100
+
+libxi (2:1.3-3) unstable; urgency=low
+
+  [ Julien Cristau ]
+  * Rename the build directory to not include DEB_BUILD_GNU_TYPE for no
+    good reason.  Thanks, Colin Watson!
+  * Remove myself from Uploaders
+  * Update patch from 2:1.3-2 to not crash if an app (like, say, anything
+    using Qt3) calls libXi functions but the extension isn't available
+    (closes: #568323).  Thanks, Roman Mamedov!
+
+  [ Brice Goglin ]
+  * Remove Jamey Sharp and Josh Triplett from Uploaders, closes: #568276.
+
+  [ Cyril Brulebois ]
+  * Bump Standards-Version from 3.8.3 to 3.8.4 (no changes needed).
+  * Upload to unstable.
+
+ -- Cyril Brulebois <kibi@debian.org>  Wed, 03 Mar 2010 22:14:43 +0100
+
+libxi (2:1.3-2) unstable; urgency=low
+
+  * Initialize extension with the right number of events.  This should fix
+    issues when libXi announces more events than the X server will actually
+    send for that extension, and thus smashes Xlib's event vector (closes:
+    #515734, #515946).  Patch by Peter Hutterer, analysis by Nathan Kidd.
+  * Upload to unstable.
+
+ -- Julien Cristau <jcristau@debian.org>  Wed, 06 Jan 2010 16:23:44 +0000
+
+libxi (2:1.3-1) experimental; urgency=low
+
+  * Bump Standards-Version to 3.8.3.
+  * Add build-deps on xmlto and asciidoc to build the manpages.
+  * New upstream release.
+
+ -- Julien Cristau <jcristau@debian.org>  Sat, 21 Nov 2009 18:39:28 +0100
+
+libxi (2:1.2.99.4-1) experimental; urgency=low
+
+  [ Timo Aaltonen ]
+  * Add README.source from xsfbs.  Bump Standards-Version to 3.8.1.
+
+  [ Julien Cristau ]
+  * Move libxi6-dbg to section 'debug'.
+  * Drop libxi-dev pre-dependency on x11-common, only needed for upgrades from
+    sarge.
+  * New upstream release candidate.
+  * XInput.h moved here from x11proto-input-dev; add Replaces.
+  * Also remove config.h.in in clean.
+  * Bump shlibs and update symbols.
+
+ -- Julien Cristau <jcristau@debian.org>  Tue, 29 Sep 2009 14:24:21 +0200
+
+libxi (2:1.2.1-2) unstable; urgency=low
+
+  * Fix typo in debian/rules, which made us build with make -j if
+    DEB_BUILD_OPTIONS didn't ask for it.  Thanks to Samuel Thibault for
+    reporting the problem!
+
+ -- Julien Cristau <jcristau@debian.org>  Mon, 16 Mar 2009 15:27:45 +0100
+
+libxi (2:1.2.1-1) unstable; urgency=low
+
+  * New upstream release.
+
+ -- Julien Cristau <jcristau@debian.org>  Thu, 12 Mar 2009 16:00:29 +0100
+
+libxi (2:1.2.0-2) unstable; urgency=low
+
+  * Add symbols file.
+  * Upload to unstable.
+
+ -- Julien Cristau <jcristau@debian.org>  Mon, 16 Feb 2009 19:53:01 +0100
+
+libxi (2:1.2.0-1) experimental; urgency=low
+
+  [ Timo Aaltonen ]
+  * New upstream release.
+  * debian/control:
+    - Bump x11proto-input-dev build-dep to 1.5.
+  * debian/rules:
+    - Bump the shlibs for libxi6.
+
+  [ Julien Cristau ]
+  * Drop -1 debian revisions from build-deps.
+  * Bump Standards-Version to 3.7.3.
+  * Drop the XS- prefix from Vcs-* control fields.
+  * libxi6{,-dbg} don't need to depend on x11-common.
+  * Wrap build-deps in debian/control.
+  * Build-depend on automake and libtool, run autoreconf on build, and allow
+    parallel builds.
+  * Bump libxi-dev's dependency on x11proto-input-dev to 1.5 as well.
+
+  [ Brice Goglin ]
+  * Add a link to www.X.org in the long description.
+
+ -- Julien Cristau <jcristau@debian.org>  Fri, 19 Dec 2008 15:55:22 +0100
+
+libxi (2:1.1.4-1) unstable; urgency=low
+
+  * New upstream release.
+
+ -- Julien Cristau <jcristau@debian.org>  Mon, 17 Nov 2008 03:16:48 +0100
+
+libxi (2:1.1.3-1) unstable; urgency=low
+
+  * Add upstream URL to debian/copyright.
+  * New upstream release.
+
+ -- Julien Cristau <jcristau@debian.org>  Thu, 06 Sep 2007 01:45:30 +0200
+
+libxi (2:1.1.2-1) unstable; urgency=low
+
+  * New upstream release
+    + fixes locking bugs (closes: #429147).
+  * Add myself to Uploaders and remove Branden and Fabio with their
+    permission.
+  * Use ${binary:Version} instead of ${Source-Version}.
+  * Add a XS-Vcs-Browser header.
+
+ -- Julien Cristau <jcristau@debian.org>  Wed, 01 Aug 2007 05:07:05 +0200
+
+libxi (2:1.1.1-1) unstable; urgency=low
+
+  * New upstream release.
+
+ -- Julien Cristau <jcristau@debian.org>  Fri, 06 Jul 2007 03:52:46 +0200
+
+libxi (2:1.1.0-2) experimental; urgency=low
+
+  * Pull fixes from upstream git.
+    + remove patch 00_upstream_002_XiCheckExtInit_unlockdisplay.diff applied
+      upstream.
+  * Bump epoch to 2 so ubuntu can sync this package.
+
+ -- Julien Cristau <jcristau@debian.org>  Mon,  5 Mar 2007 17:24:20 +0100
+
+libxi (1:1.1.0-1) experimental; urgency=low
+
+  * Adjust Section values to what the override says.
+  * Update location of upstream repository from CVS to git in the long
+    descriptions.
+  * New upstream release.
+    + Drop patches applied upstream
+    + Bump build-dependency on x11proto-input-dev to >= 1.4.
+  * Install the upstream changelog.
+  * Add XS-Vcs-Git header in debian/control.
+
+ -- Julien Cristau <jcristau@debian.org>  Fri,  9 Feb 2007 12:34:51 +0100
+
+libxi (1:1.0.1-4) unstable; urgency=low
+
+  * libXi: Various Display locking correctness issues (Closes: #400442)
+  * Add Josh and myself to Uploaders.
+
+ -- Jamey Sharp <sharpone@debian.org>  Mon, 27 Nov 2006 15:07:34 -0800
+
+libxi (1:1.0.1-3) unstable; urgency=low
+
+  * Bring X11R7.1 into unstable.  Still closes #377204.
+  * dbg package has priority extra.
+
+ -- Drew Parsons <dparsons@debian.org>  Sat, 26 Aug 2006 20:46:35 +1000
+
+libxi (1:1.0.1-2) experimental; urgency=low
+
+  * Patch 10_manpage_section.diff sets the man page section according
+    to the definition given by xutils macros.  Set Build-Depends:
+    xutils-dev to prove the point.  Install man pages using
+    dh_installman.  Closes: #377204.
+
+ -- Drew Parsons <dparsons@debian.org>  Fri, 11 Aug 2006 15:03:27 +1000
+
+libxi (1:1.0.1-1) experimental; urgency=low
+
+  * New upstream version (X11R7.1).
+    - applies patch 01_manpage_location.diff.
+  * libxi provides no header files, so don't try to install usr/include/X11/*
+  * Exclude .la from dh_install.
+
+ -- Drew Parsons <dparsons@debian.org>  Thu, 10 Aug 2006 03:00:53 +1000
+
+libxi (1:1.0.0-6) UNRELEASED; urgency=low
+
+  * Test for obj-$(DEB_BUILD_GNU_TYPE) before creating it during build;
+    idempotency fix.
+  * Run dh_install w/ --list-missing.
+  * Drop duplicate x11-common dep in -dev package.
+  * Bump standards version to 3.7.2.0.
+  * Version x11-common pre-dep in -dev package to 1:7.0.0 to match
+    the rest of Debian.
+  * Bump debhelper compat to 5.
+  * Fix dh_strip call to skip the -dbg package.
+
+ -- Andres Salomon <dilinger@debian.org>  Mon, 17 Jul 2006 01:21:03 -0400
+
+libxi (1:1.0.0-5) unstable; urgency=low
+
+  * Reorder makeshlib command in rules file so that ldconfig is run
+    properly. Thanks Drew Parsons and Steve Langasek.
+  * Add quilt to build-depends
+
+ -- David Nusinow <dnusinow@debian.org>  Wed, 19 Apr 2006 00:36:48 -0400
+
+libxi (1:1.0.0-4) unstable; urgency=low
+
+  * Remove references to xss in -dev and -dbg package descriptions. Thanks
+    Josh Triplett. (closes: #362059)
+
+ -- David Nusinow <dnusinow@debian.org>  Tue, 11 Apr 2006 20:26:26 -0400
+
+libxi (1:1.0.0-3) unstable; urgency=low
+
+  * Upload to unstable
+
+ -- David Nusinow <dnusinow@debian.org>  Thu, 23 Mar 2006 22:44:42 -0500
+
+libxi (1:1.0.0-2) experimental; urgency=low
+
+  * Backport manpage location fix
+
+ -- David Nusinow <dnusinow@debian.org>  Thu,  9 Mar 2006 23:22:36 -0500
+
+libxi (1:1.0.0-1) experimental; urgency=low
+
+  * First modular upload to Debian
+  * Install manpages in libxi-dev. Thanks Julien Cristau.
+
+ -- David Nusinow <dnusinow@debian.org>  Thu, 29 Dec 2005 20:53:15 -0500
+
+libxi (1:1.3.0-2) breezy; urgency=low
+
+  * Bump libx11-dev and libxext-dev Build-Deps to avoid _XOPEN_SOURCE.
+
+ -- Daniel Stone <daniel.stone@ubuntu.com>  Sat, 23 Jul 2005 00:28:53 +1000
+
+libxi (1:1.3.0-1) breezy; urgency=low
+
+  * First libxi release.
+
+ -- Daniel Stone <daniel.stone@ubuntu.com>  Mon, 16 May 2005 22:10:17 +1000
--- libxi-1.8.2.orig/debian/control
+++ libxi-1.8.2/debian/control
@@ -0,0 +1,81 @@
+Source: libxi
+Section: x11
+Priority: optional
+Maintainer: Debian X Strike Force <debian-x@lists.debian.org>
+Build-Depends:
+ debhelper-compat (= 12),
+ quilt,
+ x11proto-dev (>= 2021.5),
+ libx11-dev,
+ libxext-dev,
+ libxfixes-dev,
+ xmlto,
+ asciidoc,
+ pkgconf,
+ quilt,
+ xutils-dev,
+# specs
+ xmlto,
+ xorg-sgml-doctools,
+ xsltproc,
+ w3m,
+Standards-Version: 4.5.0
+Vcs-Git: https://salsa.debian.org/xorg-team/lib/libxi.git
+Vcs-Browser: https://salsa.debian.org/xorg-team/lib/libxi
+Homepage: https://www.x.org/
+
+Package: libxi6
+Section: libs
+Architecture: any
+Depends: ${shlibs:Depends}, ${misc:Depends}
+Pre-Depends: ${misc:Pre-Depends}
+Multi-Arch: same
+Description: X11 Input extension library
+ libXi provides an X Window System client interface to the XINPUT
+ extension to the X protocol.
+ .
+ The Input extension allows setup and configuration of multiple input devices,
+ and hotplugging of input devices (to be added and removed on the fly).
+ .
+ More information about X.Org can be found at:
+ <URL:https://www.X.org>
+ .
+ This module can be found at
+ git://anongit.freedesktop.org/git/xorg/lib/libXi
+
+Package: libxi6-udeb
+Package-Type: udeb
+Section: debian-installer
+Architecture: any
+Depends: ${shlibs:Depends}, ${misc:Depends}
+Description: X11 Input extension library
+ This is a udeb, or a microdeb, for the debian-installer.
+
+Package: libxi-dev
+Section: libdevel
+Architecture: any
+Multi-Arch: same
+Depends:
+ ${shlibs:Depends},
+ ${misc:Depends},
+ libxi6 (= ${binary:Version}),
+ libx11-dev,
+ libxext-dev,
+ libxfixes-dev,
+ x11proto-dev (>= 2021.5),
+ xorg-sgml-doctools,
+Description: X11 Input extension library (development headers)
+ libXi provides an X Window System client interface to the XINPUT
+ extension to the X protocol.
+ .
+ The Input extension allows setup and configuration of multiple input devices,
+ and hotplugging of input devices (to be added and removed on the fly).
+ .
+ This package contains the development headers for the library found in
+ libxi6.  Non-developers likely have little use for this package.
+ .
+ More information about X.Org can be found at:
+ <URL:https://www.X.org>
+ .
+ This module can be found at
+ git://anongit.freedesktop.org/git/xorg/lib/libXi
--- libxi-1.8.2.orig/debian/copyright
+++ libxi-1.8.2/debian/copyright
@@ -0,0 +1,87 @@
+This package was downloaded from
+https://xorg.freedesktop.org/releases/individual/lib/
+
+Copyright 1989, 1998  The Open Group
+
+Permission to use, copy, modify, distribute, and sell this software and its
+documentation for any purpose is hereby granted without fee, provided that
+the above copyright notice appear in all copies and that both that
+copyright notice and this permission notice appear in supporting
+documentation.
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+OPEN GROUP BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN
+AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+Except as contained in this notice, the name of The Open Group shall not be
+used in advertising or otherwise to promote the sale, use or other dealings
+in this Software without prior written authorization from The Open Group.
+
+Copyright 1989 by Hewlett-Packard Company, Palo Alto, California.
+
+			All Rights Reserved
+
+Permission to use, copy, modify, and distribute this software and its
+documentation for any purpose and without fee is hereby granted,
+provided that the above copyright notice appear in all copies and that
+both that copyright notice and this permission notice appear in
+supporting documentation, and that the name of Hewlett-Packard not be
+used in advertising or publicity pertaining to distribution of the
+software without specific, written prior permission.
+
+HEWLETT-PACKARD DISCLAIMS ALL WARRANTIES WITH REGARD TO THIS SOFTWARE,
+INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS, IN NO
+EVENT SHALL HEWLETT-PACKARD BE LIABLE FOR ANY SPECIAL, INDIRECT OR
+CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF
+USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR
+OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
+PERFORMANCE OF THIS SOFTWARE.
+
+Copyright 2008 Peter Hutterer
+
+Permission to use, copy, modify, distribute, and sell this software and its
+documentation for any purpose is hereby granted without fee, provided that
+the above copyright notice appear in all copies and that both that
+copyright notice and this permission notice appear in supporting
+documentation.
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+AUTHOR BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN
+AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+Except as contained in this notice, the name of the author shall not be
+used in advertising or otherwise to promote the sale, use or other dealings
+in this Software without prior written authorization from the author.
+
+Copyright © 2009 Red Hat, Inc.
+
+Permission is hereby granted, free of charge, to any person obtaining a
+copy of this software and associated documentation files (the "Software"),
+to deal in the Software without restriction, including without limitation
+the rights to use, copy, modify, merge, publish, distribute, sublicense,
+and/or sell copies of the Software, and to permit persons to whom the
+Software is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice (including the next
+paragraph) shall be included in all copies or substantial portions of the
+Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
+THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+DEALINGS IN THE SOFTWARE.
--- libxi-1.8.2.orig/debian/libxi-dev.install
+++ libxi-1.8.2/debian/libxi-dev.install
@@ -0,0 +1,7 @@
+usr/lib/*/libXi.a
+usr/lib/*/libXi.so
+usr/lib/*/pkgconfig/xi.pc
+usr/include/X11/extensions/
+usr/share/doc/libXi/*.html usr/share/doc/libxi-dev
+usr/share/doc/libXi/*.txt  usr/share/doc/libxi-dev
+usr/share/man/man3
--- libxi-1.8.2.orig/debian/libxi6-udeb.install
+++ libxi-1.8.2/debian/libxi6-udeb.install
@@ -0,0 +1 @@
+usr/lib/*/libXi.so.6* usr/lib
--- libxi-1.8.2.orig/debian/libxi6.install
+++ libxi-1.8.2/debian/libxi6.install
@@ -0,0 +1 @@
+usr/lib/*/libXi.so.6*
--- libxi-1.8.2.orig/debian/libxi6.symbols
+++ libxi-1.8.2/debian/libxi6.symbols
@@ -0,0 +1,89 @@
+libXi.so.6 libxi6 #MINVER#
+ XAllowDeviceEvents@Base 0
+ XChangeDeviceControl@Base 0
+ XChangeDeviceDontPropagateList@Base 0
+ XChangeDeviceKeyMapping@Base 0
+ XChangeDeviceProperty@Base 2:1.2.0
+ XChangeFeedbackControl@Base 0
+ XChangeKeyboardDevice@Base 0
+ XChangePointerDevice@Base 0
+ XCloseDevice@Base 0
+ XDeleteDeviceProperty@Base 2:1.2.0
+ XDeviceBell@Base 0
+ XFreeDeviceControl@Base 0
+ XFreeDeviceList@Base 0
+ XFreeDeviceMotionEvents@Base 0
+ XFreeDeviceState@Base 0
+ XFreeFeedbackList@Base 0
+ XGetDeviceButtonMapping@Base 0
+ XGetDeviceControl@Base 0
+ XGetDeviceDontPropagateList@Base 0
+ XGetDeviceFocus@Base 0
+ XGetDeviceKeyMapping@Base 0
+ XGetDeviceModifierMapping@Base 0
+ XGetDeviceMotionEvents@Base 0
+ XGetDeviceProperty@Base 2:1.2.0
+ XGetExtensionVersion@Base 0
+ XGetFeedbackControl@Base 0
+ XGetSelectedExtensionEvents@Base 0
+ XGrabDevice@Base 0
+ XIGrabPinchGestureBegin@Base 2:1.8
+ XIGrabSwipeGestureBegin@Base 2:1.8
+ XIAllowEvents@Base 2:1.2.99.4
+ XIAllowTouchEvents@Base 2:1.5.99.2
+ XIBarrierReleasePointer@Base 2:1.6.99.1
+ XIBarrierReleasePointers@Base 2:1.6.99.1
+ XIChangeHierarchy@Base 2:1.2.99.4
+ XIChangeProperty@Base 2:1.2.99.4
+ XIDefineCursor@Base 2:1.2.99.4
+ XIDeleteProperty@Base 2:1.2.99.4
+ XIFreeDeviceInfo@Base 2:1.2.99.4
+ XIGetClientPointer@Base 2:1.2.99.4
+ XIGetFocus@Base 2:1.2.99.4
+ XIGetProperty@Base 2:1.2.99.4
+ XIGetSelectedEvents@Base 2:1.2.99.4
+ XIGrabButton@Base 2:1.2.99.4
+ XIGrabDevice@Base 2:1.2.99.4
+ XIGrabEnter@Base 2:1.2.99.4
+ XIGrabFocusIn@Base 2:1.2.99.4
+ XIGrabKeycode@Base 2:1.2.99.4
+ XIGrabTouchBegin@Base 2:1.5.99.2
+ XIListProperties@Base 2:1.2.99.4
+ XIQueryDevice@Base 2:1.2.99.4
+ XIQueryPointer@Base 2:1.2.99.4
+ XIQueryVersion@Base 2:1.2.99.4
+ XISelectEvents@Base 2:1.2.99.4
+ XISetClientPointer@Base 2:1.2.99.4
+ XISetFocus@Base 2:1.2.99.4
+ XIUndefineCursor@Base 2:1.2.99.4
+ XIUngrabButton@Base 2:1.2.99.4
+ XIUngrabDevice@Base 2:1.2.99.4
+ XIUngrabEnter@Base 2:1.2.99.4
+ XIUngrabFocusIn@Base 2:1.2.99.4
+ XIUngrabKeycode@Base 2:1.2.99.4
+ XIUngrabTouchBegin@Base 2:1.5.99.2
+ XIWarpPointer@Base 2:1.2.99.4
+ XGrabDeviceButton@Base 0
+ XGrabDeviceKey@Base 0
+ XListDeviceProperties@Base 2:1.2.0
+ XListInputDevices@Base 0
+ XOpenDevice@Base 0
+ XQueryDeviceState@Base 0
+ XSelectExtensionEvent@Base 0
+ XSendExtensionEvent@Base 0
+ XSetDeviceButtonMapping@Base 0
+ XSetDeviceFocus@Base 0
+ XSetDeviceMode@Base 0
+ XSetDeviceModifierMapping@Base 0
+ XSetDeviceValuators@Base 0
+ XUngrabDevice@Base 0
+ XUngrabDeviceButton@Base 0
+ XUngrabDeviceKey@Base 0
+ XIUngrabPinchGestureBegin@Base 2:1.8
+ XIUngrabSwipeGestureBegin@Base 2:1.8
+ _XiGetDevicePresenceNotifyEvent@Base 2:1.1.2
+ _xibadclass@Base 0
+ _xibaddevice@Base 0
+ _xibadevent@Base 0
+ _xibadmode@Base 0
+ _xidevicebusy@Base 0
--- libxi-1.8.2.orig/debian/patches/0001-Documentation-Ids.patch
+++ libxi-1.8.2/debian/patches/0001-Documentation-Ids.patch
@@ -0,0 +1,37 @@
+
+Add explicit ids to the documentation chapters and sections so they are
+identical no matter which architecture (or day or asciidoc version) the
+packages are built on. This is necessary for multi-arch compatibility.
+
+Author: Francois Gouget <fgouget@codeweavers.com>
+
+diff -u libXi-1.7.4.orig/specs/inputlib.xml libXi-1.7.4/specs/inputlib.xml
+--- libXi-1.7.4.orig/specs/inputlib.xml	2011-12-20 01:28:44.000000000 +0100
++++ libXi-1.7.4/specs/inputlib.xml	2015-02-10 16:22:13.817236191 +0100
+@@ -26,7 +26,7 @@
+      <holder>Ardent Computer</holder>
+    </copyright>
+ 
+-<legalnotice>
++<legalnotice id="Legal_Notice">
+ <para>
+ Permission to use, copy, modify, and distribute this documentation for any purpose and without
+ fee is hereby granted, provided that the above copyright notice and this permission notice appear
+@@ -36,7 +36,7 @@
+ </para>
+ </legalnotice>
+ 
+-<legalnotice>
++<legalnotice id="Multi_Licensing">
+ <para role="multiLicensing">
+ Copyright &copy; 1989, 1990, 1991, 1992 X Consortium
+ </para>
+diff -u libXi-1.7.4.orig/specs/library.xml libXi-1.7.4/specs/library.xml
+--- libXi-1.7.4.orig/specs/library.xml	2011-12-20 01:28:44.000000000 +0100
++++ libXi-1.7.4/specs/library.xml	2015-02-10 16:17:31.518591542 +0100
+@@ -1,4 +1,4 @@
+-<chapter><title>Input Extension</title>
++<chapter id='Input_Extension'><title>Input Extension</title>
+ <sect1 id='Overview'>
+ <title>Overview</title>
+ <!-- .XS -->
--- libxi-1.8.2.orig/debian/patches/series
+++ libxi-1.8.2/debian/patches/series
@@ -0,0 +1 @@
+0001-Documentation-Ids.patch
--- libxi-1.8.2.orig/debian/rules
+++ libxi-1.8.2/debian/rules
@@ -0,0 +1,35 @@
+#!/usr/bin/make -f
+# debian/rules for the Debian libxi package.
+# Copyright © 2004 Scott James Remnant <scott@netsplit.com>
+# Copyright © 2005 Daniel Stone <daniel@fooishbar.org>
+# Copyright © 2005 David Nusinow <dnusinow@debian.org>
+
+# Uncomment this to turn on verbose mode.
+#export DH_VERBOSE=1
+
+# set this to the name of the main shlib's binary package
+PACKAGE = libxi6
+
+%:
+	dh $@ --with quilt --builddirectory=build/
+
+override_dh_auto_configure:
+	dh_auto_configure -- \
+		--with-xmlto \
+		--without-fop
+
+override_dh_install:
+	find $(CURDIR)/debian/tmp -name "*.la" -type f -delete
+	find $(CURDIR)/debian/tmp -name "*.db" -type f -delete
+	find $(CURDIR)/debian/tmp -name "*.xml" -type f -delete
+	dh_install
+
+override_dh_missing:
+	dh_missing --fail-missing
+
+override_dh_strip:
+	dh_strip -p$(PACKAGE) --dbgsym-migration="$(PACKAGE)-dbg (<< 2:1.7.8-1~)"
+	dh_strip -N$(PACKAGE)
+
+override_dh_makeshlibs:
+	dh_makeshlibs -V'libxi6 (>= 2:1.6.99.1)' --add-udeb=$(PACKAGE)-udeb -- -c4
--- libxi-1.8.2.orig/debian/upstream/metadata
+++ libxi-1.8.2/debian/upstream/metadata
@@ -0,0 +1,5 @@
+---
+Bug-Database: https://gitlab.freedesktop.org/xorg/lib/libxi/-/issues
+Bug-Submit: https://gitlab.freedesktop.org/xorg/lib/libxi/-/issues
+Repository: https://gitlab.freedesktop.org/xorg/lib/libxi.git
+Repository-Browse: https://gitlab.freedesktop.org/xorg/lib/libxi
--- libxi-1.8.2.orig/debian/upstream/signing-key.asc
+++ libxi-1.8.2/debian/upstream/signing-key.asc
@@ -0,0 +1,209 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+
+mQGiBD8b0wYRBACph9kRJmP+4+JGsCgFlFoy4vFO0DCG+jmkQN0n1wdInt/N/UtA
+sZToO72AUmfmYizA+IEbzBrx0UnUo3w3BDmHxUWf/akZiPUz9AA/YFY4xC3MY2OK
+VN2Jz6YSce4zJ5jd2ZRobHm4HuIf/8yqSCcsv7FNfrLaTNIFRs5gYYsqZwCgwmkp
+RSLRc8WAnHrTWNQDaEFM2rUEAKTjrTjMN8+KGd0BxNX7HiTSqQP++nXNwAYs1oWB
+Yt82YHj9SvRCqCzD1pzJQivYnlNoWDza1VeMnfdAvkdia8z4lYbO/RunXZJvra3Z
+VDm+izq+uwUAyvFuEYnNz09VSqwXKT6+XW0Xtz2vHq52r6DS6mK8cGJHZ5OhrRjq
+UEYxA/9STh+QfA98xtNoRcf52E/46r7IpCj440oRVc9lMfxQZrLGQNqp7sPdIhGQ
+CCo2NUII5hkhdAG71kpbfSXU4Sh32p1cU1KYCAkDFfb49bKuAs+Pff8v6FGZxTdd
+AinPZr4BbsYJatk818aTCnu0+s7L8jL5GPfeyuyEMKwzVBx2mLQpUGV0ZXIgSHV0
+dGVyZXIgKFdoby1UKSA8b2ZmaWNlQHdoby10Lm5ldD6IWQQTEQIAGQUCPxvTBgQL
+BwMCAxUCAwMWAgECHgECF4AACgkQ4jt+cLRn8L/0RACfWo3KTMUg+uPRqA6RXxk0
+4CWjXaMAoJeIxOpZLB3RBltPnSi7PyVQIkHFuQENBD8b0wgQBACTnqOYOWYVR8O1
+D73J6nbdAeZCbXrUkyXIuyqBOdKmX/0QJmSs7Wfsa+hPfDhj6ai0Gs2C8Qg/0Pzk
+86b4p9DLkf0M6RaYjUtCJBpS59xrsV6nz6xZzQa4RRdf1YJmw2tia1MMXzxbwQU2
+bKpYEm8NsGaBURMGd02EvsMN2wI2uwADBQP/e9MjVr/85XDzAlUBN8HwYW5szTyP
+8ZVcQOqOmNebkTWGLvkPrBdXmxpzrWjxPolO1WcWQjUL0bN/rbdqefT65iVHJHJZ
+/cpTtgGlCPhL5JTA50ltd0I13CABYWLFmswonXES/6tcglf4rr3Nri2sOrY5HggP
+ipEzOo5vdKOow/qIRgQYEQIABgUCPxvTCAAKCRDiO35wtGfwv68jAKCDvL2gkrg1
+4NfV7rNc057K1nL2GgCeKApWRgGVzaOkAp0P5tQulaDD6zM=
+=7uBX
+-----END PGP PUBLIC KEY BLOCK-----
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+
+mQINBFeKY50BEADAX0lod3IVceb/IWJn3kTAcO2P7PWlcBiyUDaq5b2kFkliKleZ
+ec4LoCHakQBlkRBMPNwOOxvADNk3tLQjBDpbYr6lQIrN+AxMGkXBhJ82T3bsDvlj
+3Z1wRJ1zVA7eMIktsk0FAoJxV1y7e3sBKcP0eTlXqXvR2djhi+FW+ueJDAJIFSkb
+uFirgwtX5t8nt8jCmIl75KNUKOakoENY3hLWtr16W8fO1JGkEhghI2mXcz664KTd
+MPZp6JH0/8UHTHzmATOCTqNxoDtMTi2l5059Lh/nhmso9moTYqyKmaJP2rnZUr62
+97sRMG4WcxaYfWpPyO3MCmDyGeh4sW0OC06PpED3i9xMzf/kMkMdY4ZIFcLRcPtf
+LIJhw+lc/GE1Rqe961IB5xCgnZezB7ZIL+ZlOAMwKGkq7lLbcZr2QZn84lpABKF0
+AvxECoJ4etmIcdbDVmsw18AhA3u9sr98hS5IXDyeos3Xwz6Abml8aPrhqhkKvo+J
+Kcq9FNYHg0RRlos0TqocjDzGnUjEYrmIopLcwIu2SnsNSJTygZGtqrpT+2sGEqvm
+k6Oyk95QCa580zqldvxe3CG0vrAfPvoG7irllM68TS4JcqqDHTq6eupUv9ZdIzXf
+eyTHa5cytGahgVtUcui1lzqcCBkqwN8TKl+0wCcEnxRasHJy3A2Gp+AG3wARAQAB
+tCJNYXR0aGlldSBIZXJyYiA8bWF0dGhpZXVAaGVycmIuZXU+iQI+BBMBAgAoBQJX
+imOdAhsDBQkDwmcABgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgAAKCRBoc5PuN9Eo
++PF1EACldzZPNYaC9H5E9sMn9pMsJTucBYVUy74Aw6MWAiAzRpxb9DmySmC2oEYW
+JJkwDTwv6M0Na0ed6zD79GKtAalORz2GppZpS7uoINClElWoM5TCYph6linyv9Wj
+OTlcbpX0Jqw0tdHNI2UOEjvBP3vW9kVYpEhfnHET8Ncp55j1hzoqxOhGIBE/67zc
+cLAenONAvA3YN3tHTGaOaFv+vuCFRJx9FpKbGHmdUPd3MtLqtaA4EQvDvDEholEI
+eWrjmdXJibSet6Amc5AIdFaQevZiADjjMh8MINw/6OEy9OB4s+z1RzgOrHgLiIZm
+dlP6WrNjXQwl2gmNPhctGaSHM+j2+3gckNGlI4LQYxNtKvI4iv/CoHDYmwgrcrZO
+TwFHfqt0LwqjpsU203Hw609oWYcxLeGZdITBjDz20UcfsmKQDqrBq3P1FuC5GBW3
+5bEa3wAhyE+/WKhJ94bXiHmpKsp50va3bEe17uQcYd8+E8L53aR7XP87qaHx//Mu
++OQa5Wc2d1OFHf1Mi62nbzr7pws/Mf7OSf/tnhRthuwtlfYnsUVo8usUKL/xStqo
+Ul4kc/Q81AlyaZfr7dbxsQWm2q3ksLaMaAxnk0p+kMXVzXZ9GKNOgUOJdbahORs5
+RU2f44xzfNavb63u3McADtaXskl+KHB4uDbGbGESVhm5PULk37QnTWF0dGhpZXUg
+SGVycmIgPG1hdHRoaWV1LmhlcnJiQGxhYXMuZnI+iQI+BBMBAgAoBQJXlJ63AhsD
+BQkDwmcABgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgAAKCRBoc5PuN9Eo+PKID/wM
+II+2d11clp1X7eZgkxkAHUhI2W3NSesuFnjkkQRKQoVMokDdeSOkBhMJuWoFfbZk
+jYs2VHU9029rDqcoDSqGwo2IffvrXXJ4SjOTjlvXS1lr/H2VdWRbq8ImnDwSsoiD
+dWB3dZyqzf7ABKZ7ccA+NMSs6NxeEN/0+0sTJ386Zp480ByNX0uPqYSq5lX/VEke
+nI8r02u2ZfuykhGkT0sM013VprfYLa+6HvF+QT9KfP220mqRbonaDkYvCxwjCMzd
+rUmvyqw3VsooUpg/W/PmDNeShSuOxebaGnFyGTNvTarElCBdynFD01dqOecOqfY8
+gy+PJ1aF1qjmf+RQD/SZq+gvgyXqyBhJy7zgJnzzNWzDlUIw0ZOLyZxzFR7lRV79
+2mrGgczlQr5rLAgBy2pgwsCmP7nFx50r4ft2juugnQixoOBU/YfhBplM76EROaCc
+MTs5nPEqzJ9p4SNkPcK8AroR2Ka3+f7t+XOoHpx/XhJOBYlPaUmoFkWKr0Y8BWWh
+1nJxyFKrSNbwUgam8ypZzwzbI1vDiX8Ol6NpEeOLwzFNT0pyTdC9UN93M1VIyKWC
+1vaeMogUREKT6SmDjRn3fISktZ0IGVf2AnFMhtgZ46TJO4BZgDdZAjTkZc/lP0yF
+Nl6MpGwnaymmL50ckT77OdlfIcXFwvNPFwWlFPlcyrkCDQRXimOdARAA4otssvZm
+sKg+g0bVyJHhn/YOHLYMih+Xf07xJHyalH0UCGnGdHZwl0B97G950SwQ7yVXtGa9
+CAPe97clE6dPD6jaumQ13BHavXM+ThgjCe8V56ayYcdzqFkxlCx0Uocoa63G0/cE
+TiOqeqhNZs8JY+D7l83jCa4lU/1pLusbkCpCQ7d5/FFLz7QSihzJWp+UTsjbNik5
+spaseEMGFRKUcB3SZ/l1dTgc0wBQ1hlvLX+h4/sG0iUs1pVpo5ORC+bUfWRokl96
+uj5QZz5rY21FaNSP1rB1HKHNkwhxifBCHQMhYGTXvD7GH+JNyF2TdRmo7eBCfAPJ
+aP3mX9t2SkCipdSsUs+Uuyib9MLA71ApW90AGiRm6HtOCxR0c3+qQRNIdFVm8mnM
+hCxXRexf6Z2wZdXXy6uY0LVRgI0o31NPJPk8l2Hnb/kHGxjyUFzEWh65J/eA368d
+4m8uF+Rr7WWlpQjwgWHU12kGThEVFFBFh2gmeIjYZdDDVhCi2mQ6lGSV2Pt7pZYL
+/PPChWLBqrVBkIUQ0GV22nRYvGdaIv2LVPu8PggbPs/wwh35nJ3rUQyJF55CFV5y
+WIWAWXfRYTKG9jkt+ncjZLEBxDO26zzO/MjIVPZxGyYryXEOgr6xp38xbyX9FpjL
+KBaIueLWEyphVjBb1uUpDGx+UDYe9vbJjPUAEQEAAYkCJQQYAQIADwUCV4pjnQIb
+DAUJA8JnAAAKCRBoc5PuN9Eo+D8dEACa60Q3ta6BWyHG0SOgfYGHE15LodACVHNI
+N6Ou+JtmLarMW/AvPclNC25mxZV0ywLbun4CnJ9qYbt/Kx7djn48mrNa0rKN8Q+V
+K5RvQA1kD890yzwu5jH6r5BQ8VBcfsPvsvatgbquzFn+NNiH9U4xRf/9BSY2Zk3G
+yA15xG0T9zoklOMg8MWbeRaJPkDELyaHPWerbO7rebynePENSFPz3o3g+K9WcCM2
+xkEL571SmT4z3Mp/p0pwemWBCP2WoKCnSjAGiiHpCFru3SlZhRIvNJyK5jeS/IU6
+d5qeTBse6TXzp6Q4xkzACIN66P5SG/YY3/ONbfs6wB3lIkvVC9n7jEXjMK1T0fK8
+9DBDjzvAkJcKLLuIljjkMhRWSCED74sn+MlaWm0xMeo276EnaVILNcrHecSr8+eX
+pVXSWEJ1+ErzZladJC+CrqUm0QljPV8Smtmk9MvOLHZ4qL4bI4Hu7MywuGNrLSol
+qO0pAT1AjaYTRuH2MhZ6mJe/EtSl0EHXEkcDteE4jbYj3lwVhA1c/So0CdayImmD
+/0tdqUfekw4va8PpbQ0wroL0XUvf3wl6HOhFhahWSqqb1fVr2slVttkaMb8M4MPt
+Ka2m4qiiuGYivPIAVapSEA4DYc+krVqVXV/yDd3T7XcNtnClVo+rmOn5WiGq24am
+79+hF4bWyw==
+=WW1Z
+-----END PGP PUBLIC KEY BLOCK-----
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+
+mQINBE6HVy4BEADACp0EU6HZ4KyFx/qfhzNarCfnlyEoCFY08k516UaHrUOrYWPp
+ukoahcceA/M3H/xM0CGIn6uiuG/Cq7+qODAZNBsr6haIbDaqSUt+953b5qCSbD65
+LBR8TXvW+9KkXPhXTKi/osYBdmsbFLeVLqU5Kd4QJqWKRLtuo0ENbFkQPVypEJk8
+Ozg2zZ2yeSQAy0pgeFh8lezI7A23yj229kFq0EEfeqHpgifIzR2hNIhS5pTSOt8V
+RDapO3FpOmxPPUMsaJ1KATD92+SgbZW8evW7ffz4QUiQiFsfTSOTCaTFu8qpu6Fb
+a9u/u6mTrJQGRdqDcFp3iWjEUOVr0gUdLSr3zey152PBRaC26/eLqH8PFgCerBkn
+o9vso0Vr+Kh63OOQeDHATZGy7tMHbWW5AEXVkTpNoSRYr48pd6u9Z1TfWVcovNAZ
+tWiFVKKxniTa4MZY2czOSyh2YahCbEt3P0DoNihy3YHhTvW1k0Os2x5yCsfpGzp3
+U8x1apfQqAyRNIa9SptLpQ7xF+lv52D9kp3XdkWXw1BFY+nmm/FqoC4tKU8AmbuB
+n3SX/sYjq3Z6aLoBOmZ849G0Zp1xEYHCbfWBxvqhIc6dlPc3Y9uYV01+FlTzX9Mh
+THa8p6oABrXbWRJpkOvaVbdDhXON+02Jlvawy3T3rwVkuEfEZu8akv7miwARAQAB
+tCBNYXR0IFR1cm5lciA8bWF0dHN0ODhAZ21haWwuY29tPokCWAQTAQgAQgIbAwYL
+CQgHAwIGFQgCCQoLBBYCAwECHgECF4ACGQEWIQQ7tjnlb4YfouhlBWkP3Wgtl0yn
+KgUCXEbE5wUJEZEGlAAKCRAP3Wgtl0ynKjzMD/9DYhJ2OEosC6ZDo98Co3hk+Eqy
++egLbU5JVfueC5r3hVjvD4OORvx2bgOAt1bYJZkFgruoA3UmPkGYICROVGwwijZz
+XU7uttZl9fEZZYNYUlawtixEyz5104hW7EfJ1df2NGvyqNu32gRH/hqhTeHugHa4
+MEfQQbbQjUSHqgUzKXBcZ3BKU9Ndmh8/4fSMdAO70rYTNet3JN74ey4ztHeJbRjv
+cqKjbMowO8qlQh7aIAo4hnXzwtygyVvrzHRoMZ3yd35t/6N3XBIw0m2RuPKegIFd
+M5fntLcrfmHk92v9CWEOVAmcm4XFFV4uYQtQZcnxys6fMakksaRkIwH804gaKwaj
+L0eILnQ8xGW/0xlzKW4mSJnk2Mtn8dURbT4ngDSradYii3vIFsfbP3q8t/exlZak
+9/o5RYnQpkkJbiRJveLvqmKexMRwn7ALi3O5NYKTlHcP+EMGvkmeOeMtMAYb3G+d
+3ihOT+CLdiI6YTLNVw7h1zvNcxBkGsNuxNzAhuMa5TEADqzGwP41+e67LfX6aYzK
+NZyR8OOJfh3fJgiBN6oluHXtb0dEyk4x8czycpQlwQpdBgj+djUM2it6pgku2bdt
+/Q5azSUTjKLA2NSbSch3kN/5sawCt6V6O6MGnUaGDbmOZ4agEGkoW1awSIyo3FbM
+01vkHu+VqMuz+Wk4XLQhTWF0dCBUdXJuZXIgPG1hdHRzdDg4QGdlbnRvby5vcmc+
+iQJVBBMBCAA/AhsDBgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgBYhBDu2OeVvhh+i
+6GUFaQ/daC2XTKcqBQJcRsTrBQkRkQaUAAoJEA/daC2XTKcqayYQAIAdo715gbvX
+ARFdXIPrLsP3brniTLZ0AV5g/ysLvmcTurGSARUbbBDjReRVJwOq0/FRbElIZuR+
+wSSGUbH/f4UvyKzwDMHWHxyROAWDI1hxPN/AId2CV4YcjptcHPi3odDYuG8NZo1n
+NecGhSXkJYfLLe1D5yyj7Tmia28uXfDJhE9gDKG99lV6lPhTAV9Hr83g+QI5W4l0
+Utnczo2aiH8nis5RP7R6qjaqod23x4s9MJUZqXl+U8Gv/XGUtQrHM0rGz3gsIRRL
+qvZN0F2MCQQk241Ukswv7fIHsyEzmn6CruSSjSZ8Y1sbN/bJgbd/lTybgO4faUjd
+BrFTDhnBnFNiplAvya247MfFzZKOq55jdE3MLzTJfenddy3SzaXOma8PA/BUXYGa
+3D1xg+mZjB+0MkEP4LLC91s0dq2XmWE0BxSMCiSilwbjsrrWV+Qsq9yKElBPnF7v
+/JaNL+AWizDhA/XIFRBTcbBRCAayPE2CNmY6/hO3UtoG5W8vGoZ0JYMCafV2U13E
+Nf9uqtq6mQWhUPnHzkMTulMQbIa7zsC4goD86C5Y3PUQr2wPeD+JbYM1zyzeFng7
+4oFDBWnjwMmUYC2xKB02amBpBMVMHyrmGje1ODrLZ0Tl3JNpZttxLOutoyStxrf5
+UT/a3k7Ua5m8xehkXQqyFccrRtT4Rj09tCNNYXR0IFR1cm5lciA8bWF0dC50dXJu
+ZXJAaW50ZWwuY29tPokCVgQTAQgAQAIbAwcLCQgHAwIBBhUIAgkKCwQWAgMBAh4B
+AheAFiEEO7Y55W+GH6LoZQVpD91oLZdMpyoFAlxGxOsFCRGRBpQACgkQD91oLZdM
+pyoB4BAAnIQO8cdtpjn0PZr2pcV2nlJrMyXXZ4zwSvCjgfgdWYiaUqP5ZCA433FC
+s1k2+oSxErMQ8tBfvGssY4vooM3tdKuJKR9TvJNasckpnUJ4w7Mh0VemYRiIBBKQ
+a1LttNj7bERMb4a1R+JKJz38CQUEoCcohJYVlbotiJ9OZDpEj/lGmja5DiwZx23g
+ORKeg4gomsdaz9CTPnlDPXsQSUrjRj6hNuC9cteflXOwjx/18GCpxzpStLfft32o
+SfghrodXdW4RmH4sQvzj8xCOtuB5Bkge8rRYqiM8C/GtfkXr3/RsKvkxXHjnmEhR
+ybK7wl0UGY86NanSFsHlobRGOGA1W07TUM3sBbKJxNK8NqNWkgOg1VP6+bUXpMYJ
+zqUiw0JqB97AyqDWmtA5k2FmFZ5lucmyHXCrMPEjAJUmogB5FL0Dv+avms+aOeOc
+25Z+5BoLRv+GLqkCyguMW5TikGhQCsp7vs0DloFCW3skxhosj8XG8gKLD+KBCEmU
+sDSaEdff161tIocemxVILbvnuzd4iHkPftumx9xzlnXI7heR0ls9Daa2gyQ2Qy94
+TS49Npm0kastyy1faeB9fFklyYaM2F4kvBowjlBEwT2E3zUPXtMtPmzGgAAOowYj
+SM+SQVgWu8u1h8vv8ieZMNrv02cBzyKqRjpklq5Ss0iVG/Enwry0Jk1hdHQgVHVy
+bmVyIDxtYXR0c3Q4OEBmcmVlZGVza3RvcC5vcmc+iQJWBBMBCABAAhsDBwsJCAcD
+AgEGFQgCCQoLBBYCAwECHgECF4AWIQQ7tjnlb4YfouhlBWkP3Wgtl0ynKgUCXEbE
+6wUJEZEGlAAKCRAP3Wgtl0ynKmnyD/9oesnUKT4uoX7cbMTBizNy1RD9KzGUE+uV
+ZQXHL/mFajlAssE0BRRxNmBObs71gnJGCTowe+KpceDP4ctO5eariubClSRutOGB
+mMxdwwcN/BVWN2hN+EMK55RUt62RviInwQp2S/F+jgjOT4Cxgvm4ZowiLgzy+8q/
+WG9Snvm7Shi3oLB6d0/XVv0BVyrzrWo30UviuEhDmqeojVfzkli7E08E77HsEfsH
+XYUmSe5scF3ko3jKFBrUA9SLZyllOf4SQvnugnG8QDF+8soOWa/1F+MN4rnLsyp9
+Ck7GoGiG3hFtYc09Bs1LR7yRP8G5/03GnkPrDhTxL0c8z5HyDOh/jvUB7lAjbBMl
+nl3i8lN7EUYgtHlex8Xuzy1ZDbMILfHFJAPQIGutNKfuTlqro/50+NRuWbpMAt9L
+jPyQSUCsM6v0gBBLb5Mj9M8fmv34H3U3URjNi3qc7Hw7vXPqBeOoE3CUKSukhXXj
+OysmyUokheRpW2/aa+usB3NTSf5k1bYQu0jA6dtIH6POvnO9ufPJ8KML1ETMp/lB
+eqcKsK+13Dine9IQWy7Dw81FitXaFhi22MbuUOswEiHlWiCI8Wwf6FCV+h4xTE+D
+EIu3Uij3hvDvecyK3n2b5q4RmtbFw1CdMYUEgrQcPFQXZ77XPp7dZj6vxB5pXeOe
+pHG+iEApDLgzBFtW0zgWCSsGAQQB2kcPAQEHQFnWW4TjF4yhukllwue8epbmWfr1
+oEGvFAzfpKJXSHokiQMUBBgBCAAmAhsCFiEEO7Y55W+GH6LoZQVpD91oLZdMpyoF
+Al0mQmYFCQWlkAUA4sAWIAQZFggAfRYhBLOWm08O+X1yHmOOvZyCWmYF1Au+BQJb
+VtM4XxSAAAAAAC4AKGlzc3Vlci1mcHJAbm90YXRpb25zLm9wZW5wZ3AuZmlmdGho
+b3JzZW1hbi5uZXRCMzk2OUI0RjBFRjk3RDcyMUU2MzhFQkQ5QzgyNUE2NjA1RDQw
+QkJFAAoJEJyCWmYF1Au+/F4A/Ro3rYgJbK5T3/dt/svCRMCpawu9pEGGQamcxVv7
+BJ6MAP47H/FLryJ+LH8srz7j5SFalYek/04C/svxSgBtIacrDQkQD91oLZdMpypP
+3hAAot/JFcn8IIsnayJLvtBKP2kMca3Dk91GpI4otptL9yI4D9WF/yX0CnzL/T7v
+lI3ijuhidi/xzIzpgJz3T4F4GJSWn/6Eohoq5qfn2kwl1sA6BjzfltGAVV/MgGv1
+IRm2zSrz/1hXy46pty8RZIWc6iA8lYOgbmxhw15HEGv3gFJH35PUy3Jdb4ikeKZZ
+wxCGtbFA7VDy3f8sj8LcV74GEo/b3dsSp8esnIlhaLssd7CWhsMnRh2pAlQrBSnj
+zmHDNAd1Vhk7ZAaHK7jpZDfWCifFNu5fQvfd4ex8o4Upasf5ophkBCgmB5lDDlzj
+gWCM5t+jn++J94Yw5s07otQFMATDQySx0K+lO5iVYVWxvx4H01wsQIfupa7O1y+F
+LcGg5vz9xKo6+/o8VC+OcbgMDJioUV5q+N/08wMD1d+QAbzL/O1RZSuL/h6LTSn1
+yL8DqYIbAMJ8Hz1Yrxvi2KP+aqlnatYEoI/9zaQUGuy+ru8aizDHCN7cqx6SEyNd
+6bnFFwrvil6AIGWkT1vtveDGTIVNGWZJjGmrmwamf9idmQT9qYydU+y0pgRvM0n2
+DKCqVlMtwHZihgySwB0DldLIO5dAhocfC5kfENECl+qkInSicUdNP+BQT8rtrKUZ
+Zw63NEC2PdHrHIBeeEvCXK8Jp7pt2N/EPx9u9Qlw6VVyC9u5Ag0ETodXLgEQAKvl
+wepeqGbkmKOTHchHY1cnO95BHnlOBbiflU+abvKSzR4zm/QIMccn7E0hGsAvrDSn
+dtD2vSsVHGbCDIkl0WNrr+z8aIP5FrOkDyLIO8rCNr+CxKt4cl7vpxQ0CP/Erq/M
+6jeGvBwFfbyQcM1M/P6xVJAk1AAj0ts38c2DUOz11FQmMDy0Zwb832mfXO6lMznK
+kWdAl9Qql1vO4cH5XWn45V97nqnsg1fIK8sTov3Dr29RiW5VKuPuQgT1pIzvPCsN
+9Of/1Eb8nwgHSP9x2CqUHSsomBQbvZoJIwbeWW2cziy2/jKlbsBnG/9d4X2KyS9j
+OIgnPULf0NoEk6qghqZeh1Eccegbq6VdKwz4TI3lcX9pEnU9nyAeIAcvERLbNWwW
+IAMd6JcriJ/SBklBUFEn9on4Qf7ex63KoOxDjj5ze2TYuy7dqZrcIq0qmosMV8Oc
+aY1bz7eFQdnFT6Hg1q+yOfKptnnt0AJNdgRL/72hMU5a3CrF1stXMgtlLfz28i81
++UI9wmRBFNuvFffTtS8M6sRybbbq1WHJ0nC5XSdOfjTRRjHLoyNoJydVRMLacU8R
+SpdRP7r+xYjDpaUmImuzi+ZYF/Ym/Pt7RiKgrjtXqjyS7F9dkmE7Vt3xjPn45lwf
+kkYum+16FQTPIhNWbmqnFs3Wb7pN6M/gc8VbXzkPABEBAAGJAjwEGAEIACYCGwwW
+IQQ7tjnlb4YfouhlBWkP3Wgtl0ynKgUCXEbFIQUJEZEGlQAKCRAP3Wgtl0ynKrOk
+EACFDCLFyUZLCRdjc4Wxb3B/JqoXC7sXRN/VUp34W5Mc4xFqldJ9OPPEle1QLtKq
+hKMjewurr3NUQwmieYkhGU3aeEB/fHw5O71RWGXuqYdprYtsMNf1q7VuKCwQ8JVZ
+5IAceAxuY5Iv7CKWnBUnJ1Od0fquAp9AAueetXma3oywqyEeDZCJX5sLzzsU/AAE
+JSolBg409QPzUcYl5AVFmJfw8hFBGteFtRtGam9BsVF1pUH9sgnC0dfKabcUP/AC
+5JQq7mbGm9Ap2aVW8b5Gx9aIFmj30U6ABCOvpn8X33XHvLPByymqIixKWyJBFqhL
+Bb2MdXVMhaA2AI6q4aYZsDxXTVlB0BATOHOYQoGUWUm0ru1dkF3ZydUZYiJSe5AW
+DyoTIrdQI4+IRScp1i0zriwp5Bs7ikL2i2X2hFjKv3xJaPGupNStcoDrGm47VMDH
+qUf+OmA/LrKoeCyk/tw2BNEBFboE1gPRQsjwGdKCvmDs7uVYNY1DpIrThcdOa0FG
+pMD8SQALp+UcZe0FsdKeccVJ/1sz5DlOriDUd40flMfS9ombg4nbkEzia6Git0Gb
+S8mFiaoRuQtPyTXOXxaJuX7QffkvW5huvbG7ROfRoyoxR90Uliv3geM+iStKsXLE
+PDo+uYb7XEUMqKxpSKeeImIdYT2oIYQMfaF6Kx1jlutV07g4BFtzwQwSCisGAQQB
+l1UBBQEBB0Byc8gV5gBy7ESPa7yI6vEI0trYdHpjKZXrHTGA+TWvaAMBCAeJAjwE
+GAEIACYWIQQ7tjnlb4YfouhlBWkP3Wgtl0ynKgUCW3PBDAIbDAUJBaOagAAKCRAP
+3Wgtl0ynKu33D/9uP4GsgvQR6UOBi9Xs3Hs6eJcvEoh+FrBCu/49rI8RG3k18YQY
+LbmdG3t1UnEd+SZP7ARl5jz2XNe6B3cA8GUD3OH0eYzKqi4YxmUi1zwbwao8buKo
+J+dh7covzyx6VG9c4QH2fDgq0VhamB0Top8B5y4ZRrQkuGuXCYx6fmma+NjV6PoC
+5FcosgwBT6XVtLZx+y95EYYCBzY5zNbkvkeBjam4I4xo0ijIjTEWYjB77275/kcH
+rJrJQFbEgHZ0nVmY+0V061BIRKOJOcBuVko3lWjN5hbIZjQqWMUrWNhnf/i6bZzZ
+OaSNDBURQlx+7EzpFYKF2g2NBV8yS5RSow/Yd2WIKkqvvE4seHmLcJZ/30aTHXOC
+0P4jiNGTCmoqo+1uGxJJ8GXMNfY4S1iycS10H8tIBJ0UHZTGP4Slk5GdzVqdtiZ1
+3pBK7+HcFBUrr/PE51iGQyfqA3g+Ur6XTG4daLQqiKyGNqrBRLBM2ZbK7+1tui8f
+kDLTT9c5M1Nq/g3qKbYfWIt9dRK+BfzPPizrupIzbZzZ6i5dL/xLnA+BPgLwPySF
+iDRvKBFfoWF6HDnRJFx87J2rq7878gfUWPQowj4V/gFE6Xvt7GZHN0IHc/f4sz2V
+sPnZULvcd/Rv0rq7QloCOPVJxUDz/Xvm6CWBhR9rhQdLTxRlyMWofCQ82Q==
+=VZ/9
+-----END PGP PUBLIC KEY BLOCK-----
--- libxi-1.8.2.orig/debian/watch
+++ libxi-1.8.2/debian/watch
@@ -0,0 +1,3 @@
+version=3
+opts=pgpsigurlmangle=s/$/.sig/ \
+https://xorg.freedesktop.org/releases/individual/lib/ libXi-(.*)\.tar\.gz
