--- libxext-1.3.4.orig/ChangeLog
+++ libxext-1.3.4/ChangeLog
@@ -1,121 +1,3 @@
-commit ebb167f34a3514783966775fb12573c4ed209625
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Sat Mar 16 10:37:14 2019 -0700
-
-    libXext 1.3.4
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit 97a5180793138498c046a7aa2ad061732e182645
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Sat Mar 16 10:36:21 2019 -0700
-
-    Add description of libXext to README.md
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit 86b2e70648531f1af11e0541f69468212c3f2190
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Sat Mar 16 10:25:49 2019 -0700
-
-    Update configure.ac bug URL for gitlab migration
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit 7ecd11d573a087e075036e0479c9e3f58ee9fa5d
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Mon Nov 19 21:53:21 2018 -0800
-
-    Update README for gitlab migration
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit a07b4bb8290d0c1bba7bcecd5bb6896fbe1b169c
-Author: Mihail Konev <k.mvc@ya.ru>
-Date:   Thu Jan 26 13:52:49 2017 +1000
-
-    autogen: add default patch prefix
-    
-    Signed-off-by: Mihail Konev <k.mvc@ya.ru>
-
-commit df966fe1336c52d0e3a6800c31a608fdc0229cff
-Author: Emil Velikov <emil.l.velikov@gmail.com>
-Date:   Mon Mar 9 12:00:52 2015 +0000
-
-    autogen.sh: use quoted string variables
-    
-    Place quotes around the $srcdir, $ORIGDIR and $0 variables to prevent
-    fall-outs, when they contain space.
-    
-    Signed-off-by: Emil Velikov <emil.l.velikov@gmail.com>
-    Reviewed-by: Peter Hutterer <peter.hutterer@who-t.net>
-    Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
-
-commit cdbb04f29f48c3fd47c3db59e985e009457b62ef
-Author: Peter Hutterer <peter.hutterer@who-t.net>
-Date:   Tue Jan 24 10:32:07 2017 +1000
-
-    autogen.sh: use exec instead of waiting for configure to finish
-    
-    Syncs the invocation of configure with the one from the server.
-    
-    Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
-    Reviewed-by: Emil Velikov <emil.velikov@collabora.com>
-
-commit 490a25e6f8a4d2482af4364c700b68ad11a4d10b
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Thu Feb 25 23:36:50 2016 -0800
-
-    Use __builtin_popcountl if available to replace Ones() in XSecurity.c
-    
-    If the compiler knows of a better algorithm for counting the number of
-    bits set in a word for the target CPU, let it use that, instead of the
-    classic algorithm optimized for PDP-6.
-    
-    Tested for the range of values used in XSecurity.c and verified results
-    are the same from both:
-    
-    for (unsigned long i = 0; i <= XSecurityAllAuthorizationAttributes; i++) {
-        printf("ones: %d\tpopcnt: %d\n", Ones(i), __builtin_popcountl(i));
-    }
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit 54ba591149f3a92c5a4a66e428ae2a33d1027053
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Thu Feb 25 15:34:45 2016 -0800
-
-    Fix typos in man pages
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit 82df003f36a405711ac41f2eb4e862f0878659e9
-Author: Alan Coopersmith <alan.coopersmith@oracle.com>
-Date:   Thu Feb 25 14:12:14 2016 -0800
-
-    Assert that values buffer has enough room for provided values
-    
-    Catch if anyone ever defines more types again and forgets to increase
-    the size of the value buffer to match.
-    
-    v2: assert on the full set of possible values, regardless of which the
-        current caller passed in this call - more likely to be spotted during
-        testing, less likely to not be found until called in production.
-    
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
-commit 0744837f525d8ba103e807af7c44ad2bf5cbd6ca
-Author: Rob Wu <rob@robwu.nl>
-Date:   Thu Feb 25 18:05:50 2016 +0100
-
-    XSecurityGenerateAuthorization: Allocate enough space in values buffer
-    
-    https://bugs.freedesktop.org/show_bug.cgi?id=94292
-    
-    Reviewed-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-    Reviewed-by: Seth Arnold <seth.arnold@canonical.com>
-    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
-
 commit efdcbb7634501e1117d422636a0a75d7ea84b16b
 Author: Peter Hutterer <peter.hutterer@who-t.net>
 Date:   Thu Jul 24 01:30:38 2014 +1000
@@ -1369,9 +1251,9 @@
     Don't export extra symbols
     
     Turns the following functions static
-            _xgeFindDisplay
-            _xgeCheckExtInit
-            _xgeEventToWire
+    	_xgeFindDisplay
+    	_xgeCheckExtInit
+    	_xgeEventToWire
     
     Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
 
--- libxext-1.3.4.orig/autogen.sh
+++ libxext-1.3.4/autogen.sh
@@ -0,0 +1,17 @@
+#! /bin/sh
+
+srcdir=`dirname "$0"`
+test -z "$srcdir" && srcdir=.
+
+ORIGDIR=`pwd`
+cd "$srcdir"
+
+autoreconf -v --install || exit 1
+cd "$ORIGDIR" || exit $?
+
+git config --local --get format.subjectPrefix >/dev/null 2>&1 ||
+    git config --local format.subjectPrefix "PATCH libXext"
+
+if test -z "$NOCONFIGURE"; then
+    exec "$srcdir"/configure "$@"
+fi
--- libxext-1.3.4.orig/debian/README.source
+++ libxext-1.3.4/debian/README.source
@@ -0,0 +1,49 @@
+------------------------------------------------------
+Quick Guide To Patching This Package For The Impatient
+------------------------------------------------------
+
+1. Make sure you have quilt installed
+2. Unpack the package as usual with "dpkg-source -x"
+3. Run the "patch" target in debian/rules
+4. Create a new patch with "quilt new" (see quilt(1))
+5. Edit all the files you want to include in the patch with "quilt edit" 
+   (see quilt(1)).
+6. Write the patch with "quilt refresh" (see quilt(1))
+7. Run the "clean" target in debian/rules
+
+Alternatively, instead of using quilt directly, you can drop the patch in to 
+debian/patches and add the name of the patch to debian/patches/series.
+
+------------------------------------
+Guide To The X Strike Force Packages
+------------------------------------
+
+The X Strike Force team maintains X packages in git repositories on
+salsa.debian.org in the xorg-team group. Most upstream packages
+are actually maintained in git repositories as well, so they often
+just need to be pulled into salsa.debian.org in a "upstream-*" branch.
+Otherwise, the upstream sources are manually installed in the Debian
+git repository.
+
+The .orig.tar.gz upstream source file could be generated this
+"upstream-*" branch in the Debian git repository but it is actually
+copied from upstream tarballs directly.
+
+Due to X.org being highly modular, packaging all X.org applications
+as their own independent packages would have created too many Debian
+packages. For this reason, some X.org applications have been grouped
+into larger packages: xutils, xutils-dev, x11-apps, x11-session-utils,
+x11-utils, x11-xfs-utils, x11-xkb-utils, x11-xserver-utils.
+Most packages, including the X.org server itself and all libraries
+and drivers are, however maintained independently.
+
+The Debian packaging is added by creating the "debian-*" git branch
+which contains the aforementioned "upstream-*" branch plus the debian/
+repository files.
+When a patch has to be applied to the Debian package, two solutions
+are involved:
+* If the patch is available in one of the upstream branches, it
+  may be git'cherry-picked into the Debian repository. In this
+  case, it appears directly in the .diff.gz.
+* Otherwise, the patch is added to debian/patches/ which is managed
+  with quilt as documented in /usr/share/doc/quilt/README.source.
--- libxext-1.3.4.orig/debian/changelog
+++ libxext-1.3.4/debian/changelog
@@ -0,0 +1,311 @@
+libxext (2:1.3.4-1) unstable; urgency=medium
+
+  [ Sven Joachim ]
+  * Move xmlto, xorg-sgml-doctools, w3m and xsltproc to
+    Build-Depends-Indep (Closes: #887046).
+  * Update Vcs-* URLs to point at salsa.debian.org.
+  * Bump debhelper compat level to 10.
+  * Drop the libxext6-dbg package in favor of an automatic -dbgsym package.
+  * Convert debian/rules to dh and drop xsfbs.
+  * Use https in debian/watch.
+  * Remove dpkg-dev from Build-Depends.
+  * Bump Standards-Version to 4.1.3, no further changes needed.
+
+  [ Timo Aaltonen ]
+  * New upstream release.
+  * control: Migrate to x11proto-dev.
+  * rules: Use dh_missing.
+  * control: Migrate to debhelper-compat, bump to 12.
+  * Update README.source for salsa.
+  * Add upstream metadata, drop git repo from d/watch.
+  * control: Bump policy to 4.6.0.
+
+ -- Timo Aaltonen <tjaalton@debian.org>  Thu, 16 Sep 2021 20:01:44 +0300
+
+libxext (2:1.3.3-1) unstable; urgency=medium
+
+  * Let uscan verify tarball signatures.
+  * New upstream release.
+  * Bump libx11-dev build-dep to 1.6.
+  * Remove Cyril from Uploaders.
+
+ -- Julien Cristau <jcristau@debian.org>  Sun, 05 Oct 2014 17:34:09 +0200
+
+libxext (2:1.3.2-1) unstable; urgency=low
+
+  * New upstream release.
+  * Bump debhelper compat level to 7.
+  * Use dpkg-buildflags.
+  * Disable silent rules.
+  * Use dh_prep instead of dh_clean -k.
+
+ -- Julien Cristau <jcristau@debian.org>  Mon, 24 Jun 2013 16:14:00 +0200
+
+libxext (2:1.3.1-2+deb7u1) wheezy-security; urgency=high
+
+  * integer overflows calculating memory needs for replies [CVE-2013-1982]
+
+ -- Julien Cristau <jcristau@debian.org>  Tue, 14 May 2013 00:46:19 +0200
+
+libxext (2:1.3.1-2) unstable; urgency=low
+
+  * Split docs out to a separate libxext-doc package.  libxext-dev is now
+    Multi-Arch: same, and libxext-doc Multi-Arch: foreign.
+  * Adjust debian/rules so buildds don't have to build the specs.
+
+ -- Julien Cristau <jcristau@debian.org>  Sun, 15 Apr 2012 21:07:20 +0200
+
+libxext (2:1.3.1-1) unstable; urgency=low
+
+  * New upstream release.
+  * Bump xorg-sgml-doctools build-dep per configure.ac.
+  * Adjust rules for new location of .db files, and set docdir in configure
+    call.
+  * Don't require (fake)root for debian/rules clean.
+
+ -- Julien Cristau <jcristau@debian.org>  Sun, 15 Apr 2012 20:46:06 +0200
+
+libxext (2:1.3.0-3) unstable; urgency=low
+
+  * Team upload.
+  * Don't put debug symbols for the udeb in libxdmcp6-dbg.
+  * Add build-indep and build-arch debian/rules targets.
+
+ -- Julien Cristau <jcristau@debian.org>  Sun, 12 Jun 2011 00:10:43 +0200
+
+libxext (2:1.3.0-2) unstable; urgency=low
+
+  * Team upload.
+
+  [ Steve Langasek ]
+  * Build for multiarch.
+
+  [ Julien Cristau ]
+  * Bump Standards-Version to 3.9.2.
+
+ -- Julien Cristau <jcristau@debian.org>  Sat, 11 Jun 2011 15:47:31 +0200
+
+libxext (2:1.3.0-1) unstable; urgency=low
+
+  * New upstream release.
+  * Update debian/copyright from upstream COPYING.
+  * Bump x11proto-xext-dev build-dep.
+  * Bump xorg-sgml-doctools build-dep, add xsltproc.
+  * Bump xutils-dev build-dep.
+  * Use xorg-sgml-doctool's copy of xorg.css.
+  * Don't install *.db yet, keep that for later.
+  * Update symbols file for new *Fence symbols.
+  * Bump shlibs accordingly.
+  * Wrap Depends.
+  * Get rid of the Replaces on x11proto-xext-dev, that was long ago.
+
+ -- Cyril Brulebois <kibi@debian.org>  Sun, 08 May 2011 21:15:49 +0200
+
+libxext (2:1.2.0-2) unstable; urgency=low
+
+  * Remove Brice Goglin and David Nusinow from Uploaders.  Thanks for your
+    work!
+  * Bump Standards-Version to 3.9.1 (no changes).
+  * Upload to unstable.
+
+ -- Julien Cristau <jcristau@debian.org>  Mon, 07 Feb 2011 22:19:09 +0100
+
+libxext (2:1.2.0-1) experimental; urgency=low
+
+  * New upstream release:
+    - include/X11/extensions/lbx{buf,bufstr,image}.h disappeared,
+      functions are found in liblbxutil.
+  * Update debian/copyright from upstream COPYING.
+  * Bump xutils-dev build-dep for new macros.
+  * Pass --with-xmlto and --without-fop to configure.
+  * Add xmlto, xorg-sgml-doctools, and w3m build-dep.
+  * Switch dh_install from --list-missing to --fail-missing for added
+    safety.
+  * Get rid of *.xml files before running dh_install.
+  * Exclude libXext.la in dh_install call.
+  * Ship *.html, *.txt, *.css in libxext-dev.
+
+ -- Cyril Brulebois <kibi@debian.org>  Fri, 14 Jan 2011 05:39:31 +0100
+
+libxext (2:1.1.2-1) unstable; urgency=low
+
+  * New upstream release.
+  * Update debian/copyright from upstream COPYING.
+  * Bump xutils-dev build-dep for xorg-macros 1.4.
+  * Drop patch included upstream, disable patch system.
+  * Bump Standards-Version to 3.9.0.
+
+ -- Julien Cristau <jcristau@debian.org>  Sun, 11 Jul 2010 14:15:51 +0100
+
+libxext (2:1.1.1-3) unstable; urgency=low
+
+  [ Julien Cristau ]
+  * Rename the build directory to not include DEB_BUILD_GNU_TYPE for no
+    good reason.  Thanks, Colin Watson!
+
+  [ Cyril Brulebois ]
+  * Add udeb needed for the graphical installer: libxext6-udeb.
+  * Bump the B-D on libx11-dev to ensure libxext6-udeb gets a dependency
+    on libx11-6-udeb.
+  * Bump Standards-Version from 3.8.3 to 3.8.4 (no changes needed).
+  * Add myself to Uploaders.
+
+ -- Cyril Brulebois <kibi@debian.org>  Thu, 11 Mar 2010 03:54:58 +0100
+
+libxext (2:1.1.1-2) unstable; urgency=low
+
+  * Upload to unstable.
+
+ -- Julien Cristau <jcristau@debian.org>  Wed, 06 Jan 2010 13:03:11 +0000
+
+libxext (2:1.1.1-1) experimental; urgency=low
+
+  [ Timo Aaltonen ]
+  * New upstream release (closes: #555999).
+  * Bump the build-dep on xutils-dev (>= 1:7.5~1).
+
+  [ Julien Cristau ]
+  * Add a symbols file for libxext6.
+  * Don't export xgeExtRegister.
+
+ -- Julien Cristau <jcristau@debian.org>  Sat, 28 Nov 2009 15:46:39 +0100
+
+libxext (2:1.0.99.4-1) experimental; urgency=low
+
+  [ Julien Cristau ]
+  * Drop the -1 debian revisions from build-deps.
+  * Drop the XS- prefix from Vcs-* control fields.
+  * libxext6{,-dbg} don't need to depend on x11-common.
+  * Use ${binary:Version} instead of the deprecated ${Source-Version}.
+  * Run autoreconf at build time.
+  * Parse space-separated DEB_BUILD_OPTIONS, handle parallel=N.
+  * Drop Pre-Depends on x11-common.  This was needed for upgrades from sarge.
+  * Bump Standards-Version to 3.8.3
+  * New upstream release.
+  * Bump build-dependencies following configure.ac, and drop libxau-dev
+    (unneeded).
+  * Update libxext-dev.install to install headers.
+  * libxext-dev Depends on new x11proto-xext-dev, and Replaces old one.
+
+  [ Brice Goglin ]
+  * Add a link to www.X.org and a reference to the upstream module
+    in the long description.
+  * Add upstream URL to debian/copyright.
+  * Use updated xsfbs, closes: #538585.
+  * Move -dbg package to section debug.
+
+  [ Timo Aaltonen ]
+  * Add README.source from xsfbs.
+
+ -- Julien Cristau <jcristau@debian.org>  Tue, 29 Sep 2009 11:52:49 +0200
+
+libxext (2:1.0.4-1) unstable; urgency=low
+
+  [ Timo Aaltonen ]
+  * Bump the epoch so that this can be synced to Ubuntu in the future.
+
+  [ Brice Goglin ]
+  * New upstream release.
+  * Update debian/copyright to the new upstream COPYING.
+  * Add myself to Uploaders, remove Fabio and Branden with their permission.
+
+ -- Brice Goglin <bgoglin@debian.org>  Sun, 02 Mar 2008 16:25:56 +0100
+
+libxext (1:1.0.3-2) unstable; urgency=low
+
+  * Put binary packages in the correct sections.
+  * Add XS-Vcs-Browser.
+  * Upload to unstable.
+
+ -- Julien Cristau <jcristau@debian.org>  Wed, 11 Apr 2007 14:45:45 +0200
+
+libxext (1:1.0.3-1) experimental; urgency=low
+
+  * New upstream release.
+    + includes XShm(3) manpage (closes: #128929).
+    + drop manpage sections patch, applied upstream.
+  * Fix typo in package description, thanks to Rakesh 'arky' Ambati
+    (closes: #398487).
+  * Drop obsolete CVS info from the descriptions, and add XS-Vcs-Git.
+  * Install the upstream ChangeLog
+  * Use dh_installman to replace ".so" links with symlinks.
+
+ -- Julien Cristau <jcristau@debian.org>  Fri,  9 Feb 2007 12:14:36 +0100
+
+libxext (1:1.0.1-2) unstable; urgency=low
+
+  [ Andres Salomon ]
+  * Test for obj-$(DEB_BUILD_GNU_TYPE) before creating it during build;
+    idempotency fix.
+
+  [ Drew Parsons ]
+  * dbg package has priority extra.
+
+  [ David Nusinow ]
+  * Make a new manpage section patch to dynamically generate the manpage
+    sections using the libmandir variable that other manpages in the package
+    use
+  * Autoreconf for manpage sections
+
+ -- David Nusinow <dnusinow@debian.org>  Sat, 26 Aug 2006 18:16:06 +0000
+
+libxext (1:1.0.1-1) experimental; urgency=low
+
+  * New upstream release
+  * Run dh_install with --list-missing
+  * Bump debhelper compat to 5
+  * Update manpage location patch to be a lot simpler
+  * Don't try to install headers, as they aren't included
+  * Remove extra dependency on x11-common in libxext-dev
+  * Version libxext-dev's pre-dep on x11-common to be >= 1:7.0.0 to match the
+    rest of Debian and shut up lintian
+  * Re-wrap description of libxext-dev so that it's not too long on any one
+    line
+  * Update standards version to 3.7.2.0
+
+ -- David Nusinow <dnusinow@debian.org>  Mon,  3 Jul 2006 15:41:05 -0400
+
+libxext (1:1.0.0-4) unstable; urgency=low
+
+  * Reorder makeshlib command in rules file so that ldconfig is run
+    properly. Thanks Drew Parsons and Steve Langasek.
+  * Add quilt to build-depends
+
+ -- David Nusinow <dnusinow@debian.org>  Tue, 18 Apr 2006 23:40:53 -0400
+
+libxext (1:1.0.0-3) unstable; urgency=low
+
+  * Upload to unstable
+
+ -- David Nusinow <dnusinow@debian.org>  Thu, 23 Mar 2006 22:44:36 -0500
+
+libxext (1:1.0.0-2) experimental; urgency=low
+
+  * Backport manpage location fix from upstream
+
+ -- David Nusinow <dnusinow@debian.org>  Thu,  9 Mar 2006 23:16:00 -0500
+
+libxext (1:1.0.0-1) experimental; urgency=low
+
+  * First upload to Debian
+
+ -- David Nusinow <dnusinow@debian.org>  Thu, 29 Dec 2005 20:51:38 -0500
+
+libxext (1:6.4.3-3) breezy; urgency=low
+
+  * Bump Build-Depends on x11proto-core-dev and libx11-dev to today's CVS
+    versions, to stop _XOPEN_SOURCE leakage.
+
+ -- Daniel Stone <daniel.stone@ubuntu.com>  Fri, 22 Jul 2005 23:33:51 +1000
+
+libxext (1:6.4.3-2) breezy; urgency=low
+
+  * Add missing build-dep on libxau-dev.
+
+ -- Adam Conrad <adconrad@ubuntu.com>  Fri, 17 Jun 2005 08:42:42 +0000
+
+libxext (1:6.4.3-1) breezy; urgency=low
+
+  * First libxext release.
+
+ -- Daniel Stone <daniel.stone@ubuntu.com>  Mon, 16 May 2005 22:10:17 +1000
--- libxext-1.3.4.orig/debian/control
+++ libxext-1.3.4/debian/control
@@ -0,0 +1,161 @@
+Source: libxext
+Section: x11
+Priority: optional
+Maintainer: Debian X Strike Force <debian-x@lists.debian.org>
+Build-Depends:
+ debhelper-compat (= 12),
+ libx11-dev (>= 2:1.6),
+ x11proto-dev,
+ pkg-config,
+ xutils-dev (>= 1:7.6+2),
+Build-Depends-Indep:
+# specs
+ xmlto (>= 0.0.22),
+ xorg-sgml-doctools (>= 1:1.8),
+ w3m,
+ xsltproc,
+Standards-Version: 4.6.0
+Vcs-Git: https://salsa.debian.org/xorg-team/lib/libxext.git
+Vcs-Browser: https://salsa.debian.org/xorg-team/lib/libxext
+
+Package: libxext6
+Section: libs
+Architecture: any
+Depends:
+ ${shlibs:Depends},
+ ${misc:Depends},
+Pre-Depends: ${misc:Pre-Depends}
+Multi-Arch: same
+Description: X11 miscellaneous extension library
+ libXext provides an X Window System client interface to several extensions to
+ the X protocol.
+ .
+ The supported protocol extensions are:
+  - DOUBLE-BUFFER (DBE), the Double Buffer extension;
+  - DPMS, the VESA Display Power Management System extension;
+  - Extended-Visual-Information (EVI), an extension for gathering extra
+    information about the X server's visuals;
+  - LBX, the Low Bandwidth X extension;
+  - MIT-SHM, the MIT X client/server shared memory extension;
+  - MIT-SUNDRY-NONSTANDARD, a miscellaneous extension by MIT;
+  - Multi-Buffering, the multi-buffering and stereo display extension;
+  - SECURITY, the X security extension;
+  - SHAPE, the non-rectangular shaped window extension;
+  - SYNC, the X synchronization extension;
+  - TOG-CUP, the Open Group's Colormap Utilization extension;
+  - XC-APPGROUP, the X Consortium's Application Group extension;
+  - XC-MISC, the X Consortium's resource ID querying extension;
+  - XTEST, the X test extension (this is one of two client-side
+    implementations; the other is in the libXtst library, provided by the
+    libxtst6 package);
+ .
+ libXext also provides a small set of utility functions to aid authors of
+ client APIs for X protocol extensions.
+ .
+ More information about X.Org can be found at:
+ <URL:http://www.X.org>
+ .
+ This module can be found at
+ git://anongit.freedesktop.org/git/xorg/lib/libXext
+
+Package: libxext6-udeb
+Package-Type: udeb
+Section: debian-installer
+Architecture: any
+Depends:
+ ${shlibs:Depends},
+ ${misc:Depends},
+Description: X11 miscellaneous extension library
+ This is a udeb, or a microdeb, for the debian-installer.
+
+Package: libxext-dev
+Section: libdevel
+Architecture: any
+Multi-Arch: same
+Depends:
+ ${shlibs:Depends},
+ ${misc:Depends},
+ libxext6 (= ${binary:Version}),
+ x11proto-dev,
+ libx11-dev,
+Suggests:
+ libxext-doc,
+Description: X11 miscellaneous extensions library (development headers)
+ libXext provides an X Window System client interface to several extensions to
+ the X protocol.
+ .
+ The supported protocol extensions are:
+  - DOUBLE-BUFFER (DBE), the Double Buffer extension;
+  - DPMS, the VESA Display Power Management System extension;
+  - Extended-Visual-Information (EVI), an extension for gathering extra
+    information about the X server's visuals;
+  - LBX, the Low Bandwidth X extension;
+  - MIT-SHM, the MIT X client/server shared memory extension;
+  - MIT-SUNDRY-NONSTANDARD, a miscellaneous extension by MIT;
+  - Multi-Buffering, the multi-buffering and stereo display extension;
+  - SECURITY, the X security extension;
+  - SHAPE, the non-rectangular shaped window extension;
+  - SYNC, the X synchronization extension;
+  - TOG-CUP, the Open Group's Colormap Utilization extension;
+  - XC-APPGROUP, the X Consortium's Application Group extension;
+  - XC-MISC, the X Consortium's resource ID querying extension;
+  - XTEST, the X test extension (this is one of two client-side
+    implementations; the other is in the libXtst library, provided by the
+    libxtst6 package);
+ .
+ libXext also provides a small set of utility functions to aid authors of
+ client APIs for X protocol extensions.
+ .
+ This package contains the development headers for the library found in
+ libxext6. Non-developers likely have little use for this package.
+ .
+ More information about X.Org can be found at:
+ <URL:http://www.X.org>
+ .
+ This module can be found at
+ git://anongit.freedesktop.org/git/xorg/lib/libXext
+
+Package: libxext-doc
+Section: doc
+Architecture: all
+Multi-Arch: foreign
+Depends:
+ ${misc:Depends},
+Breaks:
+ libxext-dev (<< 2:1.3.1-2),
+Replaces:
+ libxext-dev (<< 2:1.3.1-2),
+Description: X11 miscellaneous extensions library (documentation)
+ libXext provides an X Window System client interface to several extensions to
+ the X protocol.
+ .
+ The supported protocol extensions are:
+  - DOUBLE-BUFFER (DBE), the Double Buffer extension;
+  - DPMS, the VESA Display Power Management System extension;
+  - Extended-Visual-Information (EVI), an extension for gathering extra
+    information about the X server's visuals;
+  - LBX, the Low Bandwidth X extension;
+  - MIT-SHM, the MIT X client/server shared memory extension;
+  - MIT-SUNDRY-NONSTANDARD, a miscellaneous extension by MIT;
+  - Multi-Buffering, the multi-buffering and stereo display extension;
+  - SECURITY, the X security extension;
+  - SHAPE, the non-rectangular shaped window extension;
+  - SYNC, the X synchronization extension;
+  - TOG-CUP, the Open Group's Colormap Utilization extension;
+  - XC-APPGROUP, the X Consortium's Application Group extension;
+  - XC-MISC, the X Consortium's resource ID querying extension;
+  - XTEST, the X test extension (this is one of two client-side
+    implementations; the other is in the libXtst library, provided by the
+    libxtst6 package);
+ .
+ libXext also provides a small set of utility functions to aid authors of
+ client APIs for X protocol extensions.
+ .
+ This package contains the documentation for the included protocol extension
+ libraries. Non-developers likely have little use for this package.
+ .
+ More information about X.Org can be found at:
+ <URL:http://www.X.org>
+ .
+ This module can be found at
+ git://anongit.freedesktop.org/git/xorg/lib/libXext
--- libxext-1.3.4.orig/debian/copyright
+++ libxext-1.3.4/debian/copyright
@@ -0,0 +1,199 @@
+This package was downloaded from
+http://xorg.freedesktop.org/releases/individual/lib/
+
+Copyright 1986, 1987, 1988, 1989, 1994, 1998  The Open Group
+
+Permission to use, copy, modify, distribute, and sell this software and its
+documentation for any purpose is hereby granted without fee, provided that
+the above copyright notice appear in all copies and that both that
+copyright notice and this permission notice appear in supporting
+documentation.
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+OPEN GROUP BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN
+AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+Except as contained in this notice, the name of The Open Group shall not be
+used in advertising or otherwise to promote the sale, use or other dealings
+in this Software without prior written authorization from The Open Group.
+
+Copyright (c) 1996 Digital Equipment Corporation, Maynard, Massachusetts.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software.
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
+DIGITAL EQUIPMENT CORPORATION BE LIABLE FOR ANY CLAIM, DAMAGES, INCLUDING, 
+BUT NOT LIMITED TO CONSEQUENTIAL OR INCIDENTAL DAMAGES, OR OTHER LIABILITY, 
+WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR 
+IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+Except as contained in this notice, the name of Digital Equipment Corporation 
+shall not be used in advertising or otherwise to promote the sale, use or other
+dealings in this Software without prior written authorization from Digital 
+Equipment Corporation.
+
+Copyright (c) 1997 by Silicon Graphics Computer Systems, Inc.
+Permission to use, copy, modify, and distribute this
+software and its documentation for any purpose and without
+fee is hereby granted, provided that the above copyright
+notice appear in all copies and that both that copyright
+notice and this permission notice appear in supporting
+documentation, and that the name of Silicon Graphics not be
+used in advertising or publicity pertaining to distribution
+of the software without specific prior written permission.
+Silicon Graphics makes no representation about the suitability
+of this software for any purpose. It is provided "as is"
+without any express or implied warranty.
+SILICON GRAPHICS DISCLAIMS ALL WARRANTIES WITH REGARD TO THIS
+SOFTWARE, INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY
+AND FITNESS FOR A PARTICULAR PURPOSE. IN NO EVENT SHALL SILICON
+GRAPHICS BE LIABLE FOR ANY SPECIAL, INDIRECT OR CONSEQUENTIAL
+DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE,
+DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE
+OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION  WITH
+THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+Copyright 1992 Network Computing Devices
+
+Permission to use, copy, modify, distribute, and sell this software and its
+documentation for any purpose is hereby granted without fee, provided that
+the above copyright notice appear in all copies and that both that
+copyright notice and this permission notice appear in supporting
+documentation, and that the name of NCD. not be used in advertising or
+publicity pertaining to distribution of the software without specific,
+written prior permission.  NCD. makes no representations about the
+suitability of this software for any purpose.  It is provided "as is"
+without express or implied warranty.
+
+NCD. DISCLAIMS ALL WARRANTIES WITH REGARD TO THIS SOFTWARE, INCLUDING ALL
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS, IN NO EVENT SHALL NCD.
+BE LIABLE FOR ANY SPECIAL, INDIRECT OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION
+OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN 
+CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+Copyright 1991,1993 by Digital Equipment Corporation, Maynard, Massachusetts,
+and Olivetti Research Limited, Cambridge, England.
+
+                        All Rights Reserved
+
+Permission to use, copy, modify, and distribute this software and its
+documentation for any purpose and without fee is hereby granted,
+provided that the above copyright notice appear in all copies and that
+both that copyright notice and this permission notice appear in
+supporting documentation, and that the names of Digital or Olivetti
+not be used in advertising or publicity pertaining to distribution of the
+software without specific, written prior permission.
+
+DIGITAL AND OLIVETTI DISCLAIM ALL WARRANTIES WITH REGARD TO THIS
+SOFTWARE, INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY AND
+FITNESS, IN NO EVENT SHALL THEY BE LIABLE FOR ANY SPECIAL, INDIRECT OR
+CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF
+USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR
+OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
+PERFORMANCE OF THIS SOFTWARE.
+
+Copyright 1986, 1987, 1988 by Hewlett-Packard Corporation
+
+Permission to use, copy, modify, and distribute this
+software and its documentation for any purpose and without
+fee is hereby granted, provided that the above copyright
+notice appear in all copies and that both that copyright
+notice and this permission notice appear in supporting
+documentation, and that the name of Hewlett-Packard not be used in
+advertising or publicity pertaining to distribution of the
+software without specific, written prior permission.
+
+Hewlett-Packard makes no representations about the 
+suitability of this software for any purpose.  It is provided 
+"as is" without express or implied warranty.
+
+This software is not subject to any license of the American
+Telephone and Telegraph Company or of the Regents of the
+University of California.
+
+Copyright (c) 1994, 1995  Hewlett-Packard Company
+
+Permission is hereby granted, free of charge, to any person obtaining
+a copy of this software and associated documentation files (the
+"Software"), to deal in the Software without restriction, including
+without limitation the rights to use, copy, modify, merge, publish,
+distribute, sublicense, and/or sell copies of the Software, and to
+permit persons to whom the Software is furnished to do so, subject to
+the following conditions:
+
+The above copyright notice and this permission notice shall be included
+in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+IN NO EVENT SHALL HEWLETT-PACKARD COMPANY BE LIABLE FOR ANY CLAIM,
+DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
+OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR
+THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+Except as contained in this notice, the name of the Hewlett-Packard
+Company shall not be used in advertising or otherwise to promote the
+sale, use or other dealings in this Software without prior written
+authorization from the Hewlett-Packard Company.
+
+Copyright Digital Equipment Corporation, 1996
+
+Permission to use, copy, modify, distribute, and sell this
+documentation for any purpose is hereby granted without fee,
+provided that the above copyright notice and this permission
+notice appear in all copies.  Digital Equipment Corporation
+makes no representations about the suitability for any purpose
+of the information in this document.  This documentation is
+provided ``as is'' without express or implied warranty.
+
+Copyright (c) 1999, 2005, 2006, Oracle and/or its affiliates. All rights reserved.
+
+Permission is hereby granted, free of charge, to any person obtaining a
+copy of this software and associated documentation files (the "Software"),
+to deal in the Software without restriction, including without limitation
+the rights to use, copy, modify, merge, publish, distribute, sublicense,
+and/or sell copies of the Software, and to permit persons to whom the
+Software is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice (including the next
+paragraph) shall be included in all copies or substantial portions of the
+Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
+THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+DEALINGS IN THE SOFTWARE.
+
+Copyright (c) 1989 X Consortium, Inc. and Digital Equipment Corporation.
+Copyright (c) 1992 X Consortium, Inc. and Intergraph Corporation.
+Copyright (c) 1993 X Consortium, Inc. and Silicon Graphics, Inc.
+Copyright (c) 1994, 1995 X Consortium, Inc. and Hewlett-Packard Company.
+
+Permission to use, copy, modify, and distribute this documentation for
+any purpose and without fee is hereby granted, provided that the above
+copyright notice and this permission notice appear in all copies.
+Digital Equipment Corporation, Intergraph Corporation, Silicon
+Graphics, Hewlett-Packard, and the X Consortium make no
+representations about the suitability for any purpose of the
+information in this document.  This documentation is provided ``as is''
+without express or implied warranty.
--- libxext-1.3.4.orig/debian/libxext-dev.install
+++ libxext-1.3.4/debian/libxext-dev.install
@@ -0,0 +1,5 @@
+usr/share/man/man3/*
+usr/lib/*/libXext.a
+usr/lib/*/libXext.so
+usr/lib/*/pkgconfig/xext.pc
+usr/include/X11/extensions/
--- libxext-1.3.4.orig/debian/libxext-doc.install
+++ libxext-1.3.4/debian/libxext-doc.install
@@ -0,0 +1 @@
+usr/share/doc/libxext-dev
--- libxext-1.3.4.orig/debian/libxext6-udeb.install
+++ libxext-1.3.4/debian/libxext6-udeb.install
@@ -0,0 +1 @@
+usr/lib/*/libXext.so.6* usr/lib
--- libxext-1.3.4.orig/debian/libxext6.install
+++ libxext-1.3.4/debian/libxext6.install
@@ -0,0 +1 @@
+usr/lib/*/libXext.so.6*
--- libxext-1.3.4.orig/debian/libxext6.symbols
+++ libxext-1.3.4/debian/libxext6.symbols
@@ -0,0 +1,136 @@
+libXext.so.6 libxext6 #MINVER#
+ DPMSCapable@Base 0
+ DPMSDisable@Base 0
+ DPMSEnable@Base 0
+ DPMSForceLevel@Base 0
+ DPMSGetTimeouts@Base 0
+ DPMSGetVersion@Base 0
+ DPMSInfo@Base 0
+ DPMSQueryExtension@Base 0
+ DPMSSetTimeouts@Base 0
+ XGEQueryExtension@Base 2:1.1
+ XGEQueryVersion@Base 2:1.1
+ XLbxGetEventBase@Base 0
+ XLbxQueryExtension@Base 0
+ XLbxQueryVersion@Base 0
+ XMITMiscGetBugMode@Base 0
+ XMITMiscQueryExtension@Base 0
+ XMITMiscSetBugMode@Base 0
+ XMissingExtension@Base 0
+ XSecurityAllocXauth@Base 0
+ XSecurityFreeXauth@Base 0
+ XSecurityGenerateAuthorization@Base 0
+ XSecurityQueryExtension@Base 0
+ XSecurityRevokeAuthorization@Base 0
+ XSetExtensionErrorHandler@Base 0
+ XShapeCombineMask@Base 0
+ XShapeCombineRectangles@Base 0
+ XShapeCombineRegion@Base 0
+ XShapeCombineShape@Base 0
+ XShapeGetRectangles@Base 0
+ XShapeInputSelected@Base 0
+ XShapeOffsetShape@Base 0
+ XShapeQueryExtension@Base 0
+ XShapeQueryExtents@Base 0
+ XShapeQueryVersion@Base 0
+ XShapeSelectInput@Base 0
+ XShmAttach@Base 0
+ XShmCreateImage@Base 0
+ XShmCreatePixmap@Base 0
+ XShmDetach@Base 0
+ XShmGetEventBase@Base 0
+ XShmGetImage@Base 0
+ XShmPixmapFormat@Base 0
+ XShmPutImage@Base 0
+ XShmQueryExtension@Base 0
+ XShmQueryVersion@Base 0
+ XSyncAwait@Base 0
+ XSyncAwaitFence@Base 2:1.3.0
+ XSyncChangeAlarm@Base 0
+ XSyncChangeCounter@Base 0
+ XSyncCreateAlarm@Base 0
+ XSyncCreateCounter@Base 0
+ XSyncCreateFence@Base 2:1.3.0
+ XSyncDestroyAlarm@Base 0
+ XSyncDestroyCounter@Base 0
+ XSyncDestroyFence@Base 2:1.3.0
+ XSyncFreeSystemCounterList@Base 0
+ XSyncGetPriority@Base 0
+ XSyncInitialize@Base 0
+ XSyncIntToValue@Base 0
+ XSyncIntsToValue@Base 0
+ XSyncListSystemCounters@Base 0
+ XSyncMaxValue@Base 0
+ XSyncMinValue@Base 0
+ XSyncQueryAlarm@Base 0
+ XSyncQueryCounter@Base 0
+ XSyncQueryExtension@Base 0
+ XSyncQueryFence@Base 2:1.3.0
+ XSyncResetFence@Base 2:1.3.0
+ XSyncSetCounter@Base 0
+ XSyncSetPriority@Base 0
+ XSyncTriggerFence@Base 2:1.3.0
+ XSyncValueAdd@Base 0
+ XSyncValueEqual@Base 0
+ XSyncValueGreaterOrEqual@Base 0
+ XSyncValueGreaterThan@Base 0
+ XSyncValueHigh32@Base 0
+ XSyncValueIsNegative@Base 0
+ XSyncValueIsPositive@Base 0
+ XSyncValueIsZero@Base 0
+ XSyncValueLessOrEqual@Base 0
+ XSyncValueLessThan@Base 0
+ XSyncValueLow32@Base 0
+ XSyncValueSubtract@Base 0
+ XTestFakeAckType@Base 0
+ XTestFakeInput@Base 0
+ XTestFlush@Base 0
+ XTestGetInput@Base 0
+ XTestInputActionType@Base 0
+ XTestMovePointer@Base 0
+ XTestPressButton@Base 0
+ XTestPressKey@Base 0
+ XTestQueryInputSize@Base 0
+ XTestReset@Base 0
+ XTestStopInput@Base 0
+ XagCreateAssociation@Base 0
+ XagCreateEmbeddedApplicationGroup@Base 0
+ XagCreateNonembeddedApplicationGroup@Base 0
+ XagDestroyApplicationGroup@Base 0
+ XagDestroyAssociation@Base 0
+ XagGetApplicationGroupAttributes@Base 0
+ XagQueryApplicationGroup@Base 0
+ XagQueryVersion@Base 0
+ XcupGetReservedColormapEntries@Base 0
+ XcupQueryVersion@Base 0
+ XcupStoreColors@Base 0
+ XdbeAllocateBackBufferName@Base 0
+ XdbeBeginIdiom@Base 0
+ XdbeDeallocateBackBufferName@Base 0
+ XdbeEndIdiom@Base 0
+ XdbeFreeVisualInfo@Base 0
+ XdbeGetBackBufferAttributes@Base 0
+ XdbeGetVisualInfo@Base 0
+ XdbeQueryExtension@Base 0
+ XdbeSwapBuffers@Base 0
+ XeviGetVisualInfo@Base 0
+ XeviQueryExtension@Base 0
+ XeviQueryVersion@Base 0
+ XextAddDisplay@Base 0
+ XextCreateExtension@Base 0
+ XextDestroyExtension@Base 0
+ XextFindDisplay@Base 0
+ XextRemoveDisplay@Base 0
+ XmbufChangeBufferAttributes@Base 0
+ XmbufChangeWindowAttributes@Base 0
+ XmbufClearBufferArea@Base 0
+ XmbufCreateBuffers@Base 0
+ XmbufCreateStereoWindow@Base 0
+ XmbufDestroyBuffers@Base 0
+ XmbufDisplayBuffers@Base 0
+ XmbufGetBufferAttributes@Base 0
+ XmbufGetScreenInfo@Base 0
+ XmbufGetVersion@Base 0
+ XmbufGetWindowAttributes@Base 0
+ XmbufQueryExtension@Base 0
+ _XExtensionErrorFunction@Base 0
--- libxext-1.3.4.orig/debian/rules
+++ libxext-1.3.4/debian/rules
@@ -0,0 +1,37 @@
+#!/usr/bin/make -f
+# debian/rules for the Debian libxext package.
+# Copyright © 2004 Scott James Remnant <scott@netsplit.com>
+# Copyright © 2005 Daniel Stone <daniel@fooishbar.org>
+# Copyright © 2005 David Nusinow <dnusinow@debian.org>
+
+# Uncomment this to turn on verbose mode.
+#export DH_VERBOSE=1
+
+# set this to the name of the main shlib's binary package
+PACKAGE = libxext6
+
+export DEB_CFLAGS_MAINT_APPEND=-Wall
+
+%:
+	dh $@ --builddirectory=build/
+
+docflags = --docdir=/usr/share/doc/libxext-dev
+
+override_dh_auto_configure-arch: docflags += --disable-specs
+override_dh_auto_configure-indep: docflags += --enable-specs --with-xmlto --without-fop
+
+override_dh_auto_configure-arch override_dh_auto_configure-indep:
+	dh_auto_configure -- $(docflags)
+
+override_dh_install:
+	# Kill *.xml, we don't need them:
+	find debian/tmp -name '*.xml' -delete
+	find debian/tmp -name '*.db' -delete
+	find debian/tmp -name '*.la' -delete
+	dh_install
+
+override_dh_missing:
+	dh_missing --fail-missing
+
+override_dh_makeshlibs:
+	dh_makeshlibs -V"$(PACKAGE) (>= 2:1.3.0)" --add-udeb=$(PACKAGE)-udeb -- -c4
--- libxext-1.3.4.orig/debian/upstream/metadata
+++ libxext-1.3.4/debian/upstream/metadata
@@ -0,0 +1,5 @@
+---
+Bug-Database: https://gitlab.freedesktop.org/xorg/lib/libxext/-/issues
+Bug-Submit: https://gitlab.freedesktop.org/xorg/lib/libxext/-/issues
+Repository: https://gitlab.freedesktop.org/xorg/lib/libxext.git
+Repository-Browse: https://gitlab.freedesktop.org/xorg/lib/libxext
--- libxext-1.3.4.orig/debian/upstream/signing-key.asc
+++ libxext-1.3.4/debian/upstream/signing-key.asc
@@ -0,0 +1,24 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+Version: GnuPG v1
+
+mQGiBD8b0wYRBACph9kRJmP+4+JGsCgFlFoy4vFO0DCG+jmkQN0n1wdInt/N/UtA
+sZToO72AUmfmYizA+IEbzBrx0UnUo3w3BDmHxUWf/akZiPUz9AA/YFY4xC3MY2OK
+VN2Jz6YSce4zJ5jd2ZRobHm4HuIf/8yqSCcsv7FNfrLaTNIFRs5gYYsqZwCgwmkp
+RSLRc8WAnHrTWNQDaEFM2rUEAKTjrTjMN8+KGd0BxNX7HiTSqQP++nXNwAYs1oWB
+Yt82YHj9SvRCqCzD1pzJQivYnlNoWDza1VeMnfdAvkdia8z4lYbO/RunXZJvra3Z
+VDm+izq+uwUAyvFuEYnNz09VSqwXKT6+XW0Xtz2vHq52r6DS6mK8cGJHZ5OhrRjq
+UEYxA/9STh+QfA98xtNoRcf52E/46r7IpCj440oRVc9lMfxQZrLGQNqp7sPdIhGQ
+CCo2NUII5hkhdAG71kpbfSXU4Sh32p1cU1KYCAkDFfb49bKuAs+Pff8v6FGZxTdd
+AinPZr4BbsYJatk818aTCnu0+s7L8jL5GPfeyuyEMKwzVBx2mLQpUGV0ZXIgSHV0
+dGVyZXIgKFdoby1UKSA8b2ZmaWNlQHdoby10Lm5ldD6IWQQTEQIAGQUCPxvTBgQL
+BwMCAxUCAwMWAgECHgECF4AACgkQ4jt+cLRn8L/0RACfWo3KTMUg+uPRqA6RXxk0
+4CWjXaMAoJeIxOpZLB3RBltPnSi7PyVQIkHFuQENBD8b0wgQBACTnqOYOWYVR8O1
+D73J6nbdAeZCbXrUkyXIuyqBOdKmX/0QJmSs7Wfsa+hPfDhj6ai0Gs2C8Qg/0Pzk
+86b4p9DLkf0M6RaYjUtCJBpS59xrsV6nz6xZzQa4RRdf1YJmw2tia1MMXzxbwQU2
+bKpYEm8NsGaBURMGd02EvsMN2wI2uwADBQP/e9MjVr/85XDzAlUBN8HwYW5szTyP
+8ZVcQOqOmNebkTWGLvkPrBdXmxpzrWjxPolO1WcWQjUL0bN/rbdqefT65iVHJHJZ
+/cpTtgGlCPhL5JTA50ltd0I13CABYWLFmswonXES/6tcglf4rr3Nri2sOrY5HggP
+ipEzOo5vdKOow/qIRgQYEQIABgUCPxvTCAAKCRDiO35wtGfwv68jAKCDvL2gkrg1
+4NfV7rNc057K1nL2GgCeKApWRgGVzaOkAp0P5tQulaDD6zM=
+=7uBX
+-----END PGP PUBLIC KEY BLOCK-----
--- libxext-1.3.4.orig/debian/watch
+++ libxext-1.3.4/debian/watch
@@ -0,0 +1,3 @@
+version=3
+opts=pgpsigurlmangle=s/$/.sig/ \
+https://xorg.freedesktop.org/releases/individual/lib/ libXext-(.*)\.tar\.gz
