--- libxdmcp-1.1.5.orig/.gitlab-ci.yml
+++ libxdmcp-1.1.5/.gitlab-ci.yml
@@ -0,0 +1,99 @@
+# vim: set expandtab shiftwidth=2 tabstop=8 textwidth=0 filetype=yaml:
+#
+# This CI uses the freedesktop.org ci-templates.
+# Please see the ci-templates documentation for details:
+# https://freedesktop.pages.freedesktop.org/ci-templates/
+
+.templates_sha: &template_sha 25e8e9f020880aa35fffe3d6b259b933f0f9449f # see https://docs.gitlab.com/ee/ci/yaml/#includefile
+
+
+include:
+  # Debian container builder template
+  - project: 'freedesktop/ci-templates'
+    ref: *template_sha
+    file: '/templates/debian.yml'
+  - project: 'freedesktop/ci-templates'
+    ref: *template_sha
+    file: '/templates/ci-fairy.yml'
+  - template: Security/SAST.gitlab-ci.yml
+
+
+stages:
+  - prep             # prep work like rebuilding the container images if there is a change
+  - build            # for actually building and testing things in a container
+  - test
+  - deploy
+
+
+variables:
+  FDO_UPSTREAM_REPO: 'xorg/lib/libXdmcp'
+  FDO_DISTRIBUTION_VERSION: 'stable'
+  # The tag should be updated each time the list of packages is updated.
+  # Changing a tag forces the associated image to be rebuilt.
+  # Note: the tag has no meaning, we use a date format purely for readability
+  FDO_DISTRIBUTION_TAG:  '2023-09-23.0'
+  FDO_DISTRIBUTION_PACKAGES: 'git gcc pkgconf autoconf automake libtool make xutils-dev x11proto-dev libx11-dev xmlto xorg-sgml-doctools w3m xsltproc fop xz-utils'
+
+
+#
+# Verify that commit messages are as expected
+#
+check-commits:
+  extends:
+    - .fdo.ci-fairy
+  stage: prep
+  script:
+    - ci-fairy check-commits --junit-xml=results.xml
+  except:
+    - master@xorg/lib/libXdmcp
+  variables:
+    GIT_DEPTH: 100
+  artifacts:
+    reports:
+      junit: results.xml
+
+#
+# Verify that the merge request has the allow-collaboration checkbox ticked
+#
+check-merge-request:
+  extends:
+    - .fdo.ci-fairy
+  stage: deploy
+  script:
+    - ci-fairy check-merge-request --require-allow-collaboration --junit-xml=results.xml
+  artifacts:
+    when: on_failure
+    reports:
+      junit: results.xml
+  allow_failure: true
+
+
+#
+# Build a container with the given tag and the packages pre-installed.
+# This only happens if/when the tag changes, otherwise the existing image is
+# re-used.
+#
+container-prep:
+  extends:
+    - .fdo.container-build@debian
+  stage: prep
+  variables:
+    GIT_STRATEGY: none
+
+
+#
+# The default build, runs on the image built above.
+#
+build:
+  stage: build
+  extends:
+    - .fdo.distribution-image@debian
+  script:
+    - autoreconf -ivf
+    - mkdir _builddir
+    - pushd _builddir > /dev/null
+    - ../configure --disable-silent-rules --enable-docs --enable-unit-tests
+    - make
+    - make check
+    - make distcheck
+    - popd > /dev/null
--- libxdmcp-1.1.5.orig/debian/README.source
+++ libxdmcp-1.1.5/debian/README.source
@@ -0,0 +1,49 @@
+------------------------------------------------------
+Quick Guide To Patching This Package For The Impatient
+------------------------------------------------------
+
+1. Make sure you have quilt installed
+2. Unpack the package as usual with "dpkg-source -x"
+3. Run the "patch" target in debian/rules
+4. Create a new patch with "quilt new" (see quilt(1))
+5. Edit all the files you want to include in the patch with "quilt edit" 
+   (see quilt(1)).
+6. Write the patch with "quilt refresh" (see quilt(1))
+7. Run the "clean" target in debian/rules
+
+Alternatively, instead of using quilt directly, you can drop the patch in to 
+debian/patches and add the name of the patch to debian/patches/series.
+
+------------------------------------
+Guide To The X Strike Force Packages
+------------------------------------
+
+The X Strike Force team maintains X packages in git repositories on
+git.debian.org in the pkg-xorg subdirectory. Most upstream packages
+are actually maintained in git repositories as well, so they often
+just need to be pulled into git.debian.org in a "upstream-*" branch.
+Otherwise, the upstream sources are manually installed in the Debian
+git repository.
+
+The .orig.tar.gz upstream source file could be generated this
+"upstream-*" branch in the Debian git repository but it is actually
+copied from upstream tarballs directly.
+
+Due to X.org being highly modular, packaging all X.org applications
+as their own independent packages would have created too many Debian
+packages. For this reason, some X.org applications have been grouped
+into larger packages: xutils, xutils-dev, x11-apps, x11-session-utils,
+x11-utils, x11-xfs-utils, x11-xkb-utils, x11-xserver-utils.
+Most packages, including the X.org server itself and all libraries
+and drivers are, however maintained independently.
+
+The Debian packaging is added by creating the "debian-*" git branch
+which contains the aforementioned "upstream-*" branch plus the debian/
+repository files.
+When a patch has to be applied to the Debian package, two solutions
+are involved:
+* If the patch is available in one of the upstream branches, it
+  may be git'cherry-picked into the Debian repository. In this
+  case, it appears directly in the .diff.gz.
+* Otherwise, the patch is added to debian/patches/ which is managed
+  with quilt as documented in /usr/share/doc/quilt/README.source.
--- libxdmcp-1.1.5.orig/debian/changelog
+++ libxdmcp-1.1.5/debian/changelog
@@ -0,0 +1,227 @@
+libxdmcp (1:1.1.5-1) unstable; urgency=medium
+
+  [ Timo Aaltonen ]
+  * control: Migrate to x11proto-dev.
+
+  [ Julien Cristau ]
+  * control: update debian and upstream git URLs
+  * New upstream release.
+  * Replace libxdmcp6-dbg with automatic dbgsym package.
+  * Add debian/source/format.
+
+ -- Julien Cristau <jcristau@debian.org>  Tue, 19 Nov 2024 12:52:56 +0100
+
+libxdmcp (1:1.1.2-3) unstable; urgency=medium
+
+  * Merge 1:1.1.2-1.1 NMU, lost by mistake in 1:1.1.2-2.
+
+ -- Julien Cristau <jcristau@debian.org>  Thu, 02 Mar 2017 15:27:19 +0100
+
+libxdmcp (1:1.1.2-2) unstable; urgency=medium
+
+  * CVE-2017-2625: Build-depend on libbsd-dev for arc4random_buf.
+    Closes: #856399.
+
+ -- Emilio Pozuelo Monfort <pochu@debian.org>  Tue, 28 Feb 2017 22:47:22 +0100
+
+libxdmcp (1:1.1.2-1.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Remove &fullrelver; usage from doc/xdmcp.xml to make libxdmcp-dev M-A:same
+    safe when built with different versions of xorg-sgml-doctools. (closes:
+    #761628)
+  * Make usr/share/doc/libxdmcp-dev/xdmcp.txt.gz reproducible in the presence
+    of locales by exporting LC_ALL (closes: #783223).
+
+ -- Helmut Grohne <helmut@subdivi.de>  Wed, 11 Nov 2015 20:09:26 +0100
+
+libxdmcp (1:1.1.2-1) unstable; urgency=medium
+
+  * Let uscan verify tarball signatures.
+  * New upstream release.
+  * Switch to dh with the autoreconf and quilt addons.
+  * Disable silent build rules.
+  * Remove Cyril from Uploaders.
+
+ -- Julien Cristau <jcristau@debian.org>  Sun, 26 Apr 2015 17:16:08 +0200
+
+libxdmcp (1:1.1.1-1) unstable; urgency=low
+
+  * New upstream release.
+  * Don't require (fake)root for debian/rules clean.
+  * Exclude .db files from dh_install.
+  * Drop workaround from debian/rules for an upstream bug fixed in March 2006.
+
+ -- Julien Cristau <jcristau@debian.org>  Sat, 14 Apr 2012 18:37:46 +0200
+
+libxdmcp (1:1.1.0-4) unstable; urgency=low
+
+  * Drop /usr/share/doc/libxdmcp-dev/xdmcp.html from the package; the
+    autogenerated contents with random node numbers mean that this is not
+    multiarch-friendly, and the same information is available in
+    xdmcp.txt.gz.
+  * Mark libxdmcp-dev Multi-Arch: same.
+
+ -- Steve Langasek <vorlon@debian.org>  Tue, 08 Nov 2011 20:30:37 -0800
+
+libxdmcp (1:1.1.0-3) unstable; urgency=low
+
+  * Team upload.
+  * Don't put the debug symbols for the udeb in libxdmcp6-dbg.
+  * Add build-{arch,indep} debian/rules targets.
+
+ -- Julien Cristau <jcristau@debian.org>  Sun, 12 Jun 2011 00:06:39 +0200
+
+libxdmcp (1:1.1.0-2) unstable; urgency=low
+
+  * Team upload.
+
+  [ Steve Langasek ]
+  * Build for multiarch.
+
+  [ Julien Cristau ]
+  * Remove David Nusinow from Uploaders.
+  * Standards-Version bumped to 3.9.2.
+
+ -- Julien Cristau <jcristau@debian.org>  Tue, 07 Jun 2011 20:09:40 +0200
+
+libxdmcp (1:1.1.0-1) unstable; urgency=low
+
+  * New upstream release.
+  * Bump xutils-dev build-dep for new macros.
+  * Add xmlto, xorg-sgml-doctools, and w3m build-dep for the doc.
+  * Pass --with-xmlto and --without-fop, we want html and txt only.
+  * Ship *.html, *.css, and *.txt from the doc directory.
+  * Kill *.xml in the doc directory, no point in shipping them.
+  * Switch from --list-missing to --fail-missing for additional safety.
+  * Exclude the .la file accordingly.
+
+ -- Cyril Brulebois <kibi@debian.org>  Sat, 06 Nov 2010 14:53:01 +0100
+
+libxdmcp (1:1.0.3-2) unstable; urgency=low
+
+  [ Julien Cristau ]
+  * Rename the build directory to not include DEB_BUILD_GNU_TYPE for no
+    good reason.  Thanks, Colin Watson!
+  * Remove myself from Uploaders
+
+  [ Cyril Brulebois ]
+  * Add udeb needed for the graphical installer: libxdmcp6-udeb.
+  * Add myself to Uploaders.
+  * Bump Standards-Version from 3.8.3 to 3.8.4 (no changes needed).
+
+ -- Cyril Brulebois <kibi@debian.org>  Tue, 09 Mar 2010 02:30:26 +0100
+
+libxdmcp (1:1.0.3-1) unstable; urgency=low
+
+  [ Brice Goglin ]
+  * Add a link to www.X.org and a reference to the upstream module
+    in the long description.
+  * Add upstream URL to debian/copyright.
+
+  [ Timo Aaltonen ]
+  * New upstream release (closes: #555996).
+  * Run autoreconf on build. Add build-deps on automake, libtool
+    and xutils-dev.
+  * Parse space-separated DEB_BUILD_OPTIONS, and handle parallel=N.
+  * Bump Standards-Version to 3.8.3.
+  * Move -dbg package to section debug.
+  * Drop pre-dependency on x11-common from libxdmcp-dev. This was needed
+    for upgrades from sarge.
+
+ -- Julien Cristau <jcristau@debian.org>  Wed, 25 Nov 2009 16:28:22 +0100
+
+libxdmcp (1:1.0.2-3) unstable; urgency=low
+
+  * Put libxdmcp-dev in libdevel where it belongs.
+  * Remove Fabio and Branden from Uploaders with their permission.
+  * Add myself to Uploaders.
+  * Bump Standards-Version to 3.7.3.
+  * Drop the XS- prefix from Vcs-* control fields.
+  * libxdmcp6{,-dbg} don't need to depend on x11-common.
+  * Use ${binary:Version} instead of the deprecated ${Source-Version}.
+
+ -- Julien Cristau <jcristau@debian.org>  Sun, 18 May 2008 14:42:59 +0200
+
+libxdmcp (1:1.0.2-2) unstable; urgency=low
+
+  * Add "Section" header to binary packages in debian/control, with the values
+    from the override.
+  * Upload to unstable.
+  * Add XS-Vcs-Browser to debian/control.
+
+ -- Julien Cristau <jcristau@debian.org>  Wed, 11 Apr 2007 12:36:41 +0200
+
+libxdmcp (1:1.0.2-1) experimental; urgency=low
+
+  * New upstream release.
+  * Drop obsolete CVS info from the descriptions, and add XS-Vcs-Git.
+  * Install the upstream changelog.
+
+ -- Julien Cristau <jcristau@debian.org>  Tue, 13 Feb 2007 10:19:24 +0100
+
+libxdmcp (1:1.0.1-2) unstable; urgency=low
+
+  [ Andres Salomon ]
+  * Test for obj-$(DEB_BUILD_GNU_TYPE) before creating it during build;
+    idempotency fix.
+
+  [ Drew Parsons ]
+  * dbg package has priority extra.
+
+ -- David Nusinow <dnusinow@debian.org>  Tue, 29 Aug 2006 20:12:48 +0000
+
+libxdmcp (1:1.0.1-1) experimental; urgency=low
+
+  * New upstream release
+  * Version all x11-common dependencies to 1:7.0.0 to match the rest of Debian
+    and shut Debian up
+  * Remove extra dep on x11-common in the -dev package
+  * Bump standards version to 3.7.2.0
+  * Reformat -dev description so that it's not too long on any one line
+  * Bump debhelper compat to 5
+  * Run dh_install with --list-missing
+  * Don't try and install manpages for the -dev package, there aren't any
+
+ -- David Nusinow <dnusinow@debian.org>  Sun,  2 Jul 2006 22:13:38 -0400
+
+libxdmcp (1:1.0.0-4) unstable; urgency=low
+
+  * Reorder makeshlib command in rules file so that ldconfig is run
+    properly. Thanks Drew Parsons and Steve Langasek.
+
+ -- David Nusinow <dnusinow@debian.org>  Tue, 18 Apr 2006 21:49:58 -0400
+
+libxdmcp (1:1.0.0-3) unstable; urgency=low
+
+  * Upload to unstable
+
+ -- David Nusinow <dnusinow@debian.org>  Thu, 23 Mar 2006 22:44:34 -0500
+
+libxdmcp (1:1.0.0-2) experimental; urgency=low
+
+  * Version x11-common dependencies. Thanks Kurt Roeckx. (closes: #351779)
+
+ -- David Nusinow <dnusinow@debian.org>  Sun, 19 Mar 2006 23:35:55 -0500
+
+libxdmcp (1:1.0.0-1) experimental; urgency=low
+
+  * First upload to Debian
+  * In debian/rules copy Wraphelp.c to the build directory to allow des code
+    to build. This is an ugly hack and we'll remove it when we can. Thanks
+    Eugene Konev.
+
+ -- David Nusinow <dnusinow@debian.org>  Fri,  6 Jan 2006 00:48:12 -0500
+
+libxdmcp (1:0.1.3-2) breezy; urgency=low
+
+  * Add build dependencies: pkg-config, x11proto-core-dev
+  * Change Build-Depends-Indep to Build-Depends.
+
+ -- Matthias Klose <doko@ubuntu.com>  Mon, 23 May 2005 20:02:59 +0000
+
+libxdmcp (1:0.1.3-1) breezy; urgency=low
+
+  * First libxdmcp release.
+
+ -- Daniel Stone <daniel.stone@ubuntu.com>  Mon, 16 May 2005 22:10:17 +1000
--- libxdmcp-1.1.5.orig/debian/compat
+++ libxdmcp-1.1.5/debian/compat
@@ -0,0 +1 @@
+9
--- libxdmcp-1.1.5.orig/debian/control
+++ libxdmcp-1.1.5/debian/control
@@ -0,0 +1,63 @@
+Source: libxdmcp
+Section: x11
+Priority: optional
+Maintainer: Debian X Strike Force <debian-x@lists.debian.org>
+Build-Depends:
+ debhelper (>= 9),
+ dh-autoreconf,
+ quilt,
+ pkg-config,
+ x11proto-dev,
+ automake,
+ libtool,
+ libbsd-dev,
+ xutils-dev (>= 1:7.5+4),
+# doc:
+ xmlto,
+ xorg-sgml-doctools (>= 1:1.5),
+ w3m,
+Standards-Version: 3.9.2
+Vcs-Git: https://salsa.debian.org/xorg-team/lib/libxdmcp.git
+Vcs-Browser: https://salsa.debian.org/xorg-team/lib/libxdmcp
+
+Package: libxdmcp6
+Architecture: any
+Section: libs
+Depends: ${shlibs:Depends}, ${misc:Depends}
+Pre-Depends: ${misc:Pre-Depends}
+Multi-Arch: same
+Description: X11 Display Manager Control Protocol library
+ This package provides the main interface to the X11 display manager control
+ protocol library, which allows for remote logins to display managers.
+ .
+ More information about X.Org can be found at:
+ <URL:http://www.X.org>
+ .
+ This module can be found at
+ https://gitlab.freedesktop.org/xorg/lib/libXdmcp
+
+Package: libxdmcp6-udeb
+XC-Package-Type: udeb
+Architecture: any
+Section: debian-installer
+Depends: ${shlibs:Depends}, ${misc:Depends}
+Description: X11 Display Manager Control Protocol library
+ This is a udeb, or a microdeb, for the debian-installer.
+
+Package: libxdmcp-dev
+Architecture: any
+Multi-Arch: same
+Section: libdevel
+Depends: ${shlibs:Depends}, ${misc:Depends}, libxdmcp6 (= ${binary:Version}), x11proto-dev
+Description: X11 authorisation library (development headers)
+ This package provides the main interface to the X11 display manager control
+ protocol library, which allows for remote logins to display managers.
+ .
+ This package contains the development headers for the library found in 
+ libxdmcp6. Non-developers likely have little use for this package.
+ .
+ More information about X.Org can be found at:
+ <URL:http://www.X.org>
+ .
+ This module can be found at
+ https://gitlab.freedesktop.org/xorg/lib/libXdmcp
--- libxdmcp-1.1.5.orig/debian/copyright
+++ libxdmcp-1.1.5/debian/copyright
@@ -0,0 +1,26 @@
+This package was downloaded from
+http://xorg.freedesktop.org/releases/individual/lib/
+
+Copyright 1989, 1998  The Open Group
+
+Permission to use, copy, modify, distribute, and sell this software and its
+documentation for any purpose is hereby granted without fee, provided that
+the above copyright notice appear in all copies and that both that
+copyright notice and this permission notice appear in supporting
+documentation.
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+OPEN GROUP BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN
+AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+Except as contained in this notice, the name of The Open Group shall not be
+used in advertising or otherwise to promote the sale, use or other dealings
+in this Software without prior written authorization from The Open Group.
+
+Author:  Keith Packard, MIT X Consortium
--- libxdmcp-1.1.5.orig/debian/libxdmcp-dev.install
+++ libxdmcp-1.1.5/debian/libxdmcp-dev.install
@@ -0,0 +1,5 @@
+usr/include/X11/*
+usr/lib/*/libXdmcp.a
+usr/lib/*/libXdmcp.so
+usr/lib/*/pkgconfig/xdmcp.pc
+usr/share/doc/libXdmcp/*.txt  usr/share/doc/libxdmcp-dev
--- libxdmcp-1.1.5.orig/debian/libxdmcp6-udeb.install
+++ libxdmcp-1.1.5/debian/libxdmcp6-udeb.install
@@ -0,0 +1 @@
+usr/lib/*/libXdmcp.so.6* usr/lib
--- libxdmcp-1.1.5.orig/debian/libxdmcp6.install
+++ libxdmcp-1.1.5/debian/libxdmcp6.install
@@ -0,0 +1 @@
+usr/lib/*/libXdmcp.so.6*
--- libxdmcp-1.1.5.orig/debian/rules
+++ libxdmcp-1.1.5/debian/rules
@@ -0,0 +1,40 @@
+#!/usr/bin/make -f
+# debian/rules for the Debian libxdmcp package.
+# Copyright © 2004 Scott James Remnant <scott@netsplit.com>
+# Copyright © 2005 Daniel Stone <daniel@fooishbar.org>
+# Copyright © 2005 David Nusinow <dnusinow@debian.org>
+
+# Uncomment this to turn on verbose mode.
+#export DH_VERBOSE=1
+
+# set this to the name of the main shlib's binary package
+PACKAGE = libxdmcp6
+
+# generation of usr/share/doc/libxdmcp-dev/xdmcp.txt.gz is locale dependent
+export LC_ALL=C.UTF-8
+
+.PHONY: build
+build:
+	dh build --with quilt,autoreconf --builddirectory=build/ --parallel
+
+%:
+	dh $@ --with quilt,autoreconf --builddirectory=build/ --parallel
+
+override_dh_auto_configure:
+	dh_auto_configure -- \
+		--disable-silent-rules \
+		--with-xmlto \
+		--without-fop
+
+override_dh_auto_install:
+	dh_auto_install
+	find debian/tmp/usr/share/doc/libXdmcp -name '*.xml' -delete
+
+override_dh_install:
+	dh_install --fail-missing --exclude=libXdmcp.la --exclude .html --exclude .css --exclude .db
+
+override_dh_strip:
+	dh_strip --dbgsym-migration='libxdmcp6-dbg (<< 1:1.1.5)'
+
+override_dh_makeshlibs:
+	dh_makeshlibs --add-udeb=$(PACKAGE)-udeb
--- libxdmcp-1.1.5.orig/debian/source/format
+++ libxdmcp-1.1.5/debian/source/format
@@ -0,0 +1 @@
+1.0
--- libxdmcp-1.1.5.orig/debian/upstream/signing-key.asc
+++ libxdmcp-1.1.5/debian/upstream/signing-key.asc
@@ -0,0 +1,60 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+Version: GnuPG v1
+
+mQGiBEab+moRBACDH5yKqS3wcc5bdxY7PBNuwKvF5TKMfagmSvuRDtZjjIIWaA/n
+Z1KboV9Gq5g7kP7+Kfu+Qgd8u65eVsWwmPW10fXvj3aCU53glx2EdGdrHcgiyH2g
+EQfPiyBw+trIppWFRV0IDXSLMA1FNC92t2nSG/VFHaPTVwcgkIRSfcXDvwCglGdE
+a6f4uLqoNHP+m4yYnzapFuMD/R4+2AJDAvEWKDdYCGZzlawjAmmWyXrmT7/C/mx9
+8qUR473l4buXjHgDkkXXlHqdzil1vK85PhrKzNJDCCmlHUJNz+QwiAMOLwpD+kwV
+Pb57RG7y+a5JQ5+jtVw4RlUxZIk/wj2An9YBO3A5vR7PdjM32ZJCN2+aM4dYfNzQ
+xQKTA/47icvBaBVTl9rztjg2pd2Aqpc1P/GsIYLGj7XjnnJvGAENBHSH1QjpZMJG
+CTS9oJ+B0/wrIr+pA+MdFgYAb6ojMQJOO6UChjWWSGjMFcs/CeXhxlLBido3DtAE
+TbNTwO6OEfAvdosvTdhJFnwvZlJ+zZGGy5CrF2Fd9PUe9tmASbQoQWxhbiBDb29w
+ZXJzbWl0aCA8YWxhbmNAZnJlZWRlc2t0b3Aub3JnPohoBBMRAgAoAhsDBgsJCAcD
+AgYVCAIJCgsEFgIDAQIeAQIXgAUCUXnRYgUJFEPYeAAKCRCi+54IHy0TDonxAKCP
+cAgXNojuujUg5Wqi6v0RBFVSUgCggq1SsVEdq9NDWvXvkeGyNaBivSK0K0FsYW4g
+Q29vcGVyc21pdGggPGFsYW4uY29vcGVyc21pdGhAc3VuLmNvbT6IZgQTEQIAJgIb
+AwYLCQgHAwIEFQIIAwQWAgMBAh4BAheABQJRedFiBQkUQ9h4AAoJEKL7nggfLRMO
+6sUAn0jl3h9rY4OJ13Lu7nsKclyhDpOqAKCFgTmaDGRuDRxloLg9jftrn7a7vrQu
+QWxhbiBDb29wZXJzbWl0aCA8YWxhbi5jb29wZXJzbWl0aEBvcmFjbGUuY29tPohr
+BBMRAgArAhsDBgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgAIZAQUCUXnRYgUJFEPY
+eAAKCRCi+54IHy0TDtBZAJ9IgVVNoFIPRjTsNjcSFaLznuDRJgCcC/WgV312IrxS
+Q8PRAyEgozSB9Ke5Ag0ERpv6bxAIAJp5aUlho5rUhpS6ik7spsAQFPRuycPKMNu0
+J4F0v/OoPz085soV8ytLj4HqCGk2Zamh1jSgliZwuk9m7V7Wgxx+nBJawpWDX/eK
+LObErfDwQ4dfOFvjbXLQMmNnQNaUGIWLPP3l8GuBOHMq60Bu+TPgh627vUntL5RE
+QEQqTXIzWC6U10QsDblLwIvdOVSdGF5xl/N1myXzSKvrsZwWtoFc8G9v9hcCjhtN
+1sm9b7Ojc51iZXvcetcvPy5RA6AUW3yEExaedUdLnvIF9sjFYIfJWFVYh2AgavnG
+re6fF+NV2v3zfx3wRT7H9//m4YIDYJmgZgyQccXegTwfGBIq3osAAwYH/1FiMUMM
+ES5Ilz2nDqId+DCWECAU6wgvIFRcXrZWxDxB+ZrnmTCXoAD0xedpfOkRHp8XTVc/
+9MU+wQ+lZRx2OQ6MJW0XGuFvHm94KZF/8HzWA2Ah7U4n0+3sLpk6zWceZq2zZNF0
+yVTjwD98+xNK1Q9sP8aOKdtg8yMH3hisKR6rdW+mfX5q0Q8Gol2hZsFH/qyIhnPz
+hXDknuOh8E5iMkzrejVXUEn++Yzj23XjP59SObLznVkyxI+kBI9qvVEPfFBDybjH
+WqLcgRcCpXAzjizEi+/d31iDa2ErJHV4R42obecFqiPnoDtiX3IiP7z9fmxM4aWP
+ZZRqvq+1ht5wkn+ISQQYEQIACQUCRpv6bwIbDAAKCRCi+54IHy0TDoLoAKCHYRpw
+/XfyEunw1YL/uMZzl78qIQCdFVcXNbqD83qVhW4Ly7hyDL8o0aK5Ag0EUXnVIQEQ
+AKHpjOmY056n0tsZoW9q5egsMcl5tKC8uimrhO05nnq+5/60/YedC++V9c9b/3/X
+7O28LyBkAtBgD0xJZSDQ0DhTzKAp6AzjQtBvI68uinGwxSjT+oQpPMxqhA1I0kzo
+EDCdEqV+HsVOAEdbAi/tP9bbdTDzwVc8MWDriamBUqc53Rb00Mffy9435UgTS4gA
+hMwANhy6XZmOMBhITOzxFJUEDTDJtLbE0b1jPRQS7NHQgak1inmuvPMc3wAuoEcS
+CSt1xupbYsBoXOjK5wC/eE1LIdZoRyW2OkT140DqDZ8zfRID860hnirnYgb09TPN
+tj93pudUAUt6T9+tcLN4/rxhxHOwse66KGHO4bQ1rZ6mfco6SYd9V60cL6hC2eMe
+cyxZliMu17lj7EX8lxUH+omIgHc7HGoyUR6V+WB60cxWj5v05zdeLeZ2aLBcPFhx
+lfDESm8f4ezdJSDS1QZmC0P5h3RJfhhfmdBr8kHzr7111D1/O71Av1VV5FyJ9YxU
+Sxp4IPuzK7JbbgVHcA6PvXrDzWUslmZgPADpKH4hTmG/NdCqhEXcufvY6s5yNksB
+8X3ReNvuSSyfGnRz3kvtyK0XzC7KRX2PquLI6A8KJprHwZGqEB1NDG8b2iaYnghO
+jyfIYEVQF3nGfaBwv4lrCPEoZSUaK8f/NQZjNU8NQyTnABEBAAGJAm0EGBEKAA8F
+AlF51SECGwIFCQlmAYACKQkQovueCB8tEw7BXSAEGQEKAAYFAlF51SEACgkQz98U
+iCjGQqfW5g//dOdJHt23cdMyz5VADaE7u+L0E+eX9GtHF4J649eXsui59EtbHh2n
+XdGhd5SqQ8FDi9GCEKaQ4S31n/YBLEBCkj7R0IMikW2o78/JxDovB8+aL606hgma
+fNVx1aIshIglrl8Xlu3sjeAvG48W6YjdL2mfrIDHjIVwOZsMihbOJvST6Q3upHdn
+mjDtM5HCQmI5NEXDWYj6IZuhJnnrDWwNsyYV4KPoUBxAcqIyCeZbVssuWWnHPXX8
+VavVq98vpVynfGzGYpJbDj19C/utMjKGI5dcvbVaucA7X/oktxrxS6SBDhuIaAE9
+4ZHlbxqfyHfETI/La2Z/ALDAtYdhJR2gSkTHyKSW1QqYlulSfB//lnna44mmTuRO
+NbDNgb0FGSvtsBMZ80iHDqPgUfS60kxCfFrsSGfTFU+X4QAzpTtUJEcr+J4HULDe
+MfwOgghVfmKxFXWfud8xDaCXuywLTtVgMCZp4P7MAyuJlaxsFTu+c1Vly94grk4U
+MtALLMqCXSosA490gLTSdg3HSwxt2Q/LJdy427ZIMvjGXIruns8U/OmL9dVgWu3b
+JHsL68Skx8Ts63qTN9QXM/PB+8VwOaC7PJ+g6t40DleOmdsS8cN31yf5KB8rsL4u
+n4u1yrMJfpnSblPMu5wJi3kjoA+Dd5ZFqx9nTi4wBjfVYGCPsleq59K8kQCYx1Cn
+lZcq630ITy9dB/aHCQry2gCbBwZ2Rsf9kr05S8uLhlwW3vRSvRs=
+=tc6G
+-----END PGP PUBLIC KEY BLOCK-----
--- libxdmcp-1.1.5.orig/debian/watch
+++ libxdmcp-1.1.5/debian/watch
@@ -0,0 +1,4 @@
+#git=git://anongit.freedesktop.org/xorg/lib/libXdmcp
+version=3
+opts=pgpsigurlmangle=s/$/.sig/ \
+http://xorg.freedesktop.org/releases/individual/lib/ libXdmcp-(.*)\.tar\.gz
